<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha M. Tran" />


<title>KEGG Analysis</title>

<script src="site_libs/header-attrs-2.24/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.29/datatables.js"></script>
<link href="site_libs/dt-core-1.13.4/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.4/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.4/js/jquery.dataTables.min.js"></script>
<script src="site_libs/jszip-1.13.4/jszip.min.js"></script>
<script src="site_libs/pdfmake-1.13.4/pdfmake.js"></script>
<script src="site_libs/pdfmake-1.13.4/vfs_fonts.js"></script>
<link href="site_libs/dt-ext-buttons-1.13.4/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="site_libs/dt-ext-buttons-1.13.4/js/dataTables.buttons.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.13.4/js/buttons.html5.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.13.4/js/buttons.colVis.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.13.4/js/buttons.print.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Mouse endometial transcriptome 2023</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="setUp.html">Pre-processing</a>
</li>
<li>
  <a href="deAnalysis.html">DGE Analysis</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Functional Enrichment Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="go.html">Gene Ontology</a>
    </li>
    <li>
      <a href="kegg.html">KEGG</a>
    </li>
    <li>
      <a href="reactome.html">Reactome</a>
    </li>
    <li>
      <a href="extraFigures.html">IPA</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">KEGG Analysis</h1>
<h4 class="author">Ha M. Tran</h4>
<h4 class="date">22/08/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-09-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong>
<code>Mouse_endometrial_transcriptome_2023/1_analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the
code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135Mouseendometrialtranscriptome2023tree18c64635cc1b08f42233ef4416cc5f1869e6e3e6targetblank18c6463a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/tree/18c64635cc1b08f42233ef4416cc5f1869e6e3e6" target="_blank">18c6463</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135Mouseendometrialtranscriptome2023tree18c64635cc1b08f42233ef4416cc5f1869e6e3e6targetblank18c6463a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/tree/18c64635cc1b08f42233ef4416cc5f1869e6e3e6" target="_blank">18c6463</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  .gitignore
    Untracked:  0_data/RDS_objects/DT.rds

Unstaged changes:
    Modified:   0_data/RDS_objects/dge.rds
    Modified:   0_data/RDS_objects/enrichGO.rds
    Modified:   0_data/RDS_objects/enrichGO_sig.rds
    Modified:   0_data/RDS_objects/fc.rds
    Modified:   0_data/RDS_objects/lfc.rds
    Modified:   0_data/RDS_objects/lmTreat.rds
    Modified:   0_data/RDS_objects/lmTreat_all.rds
    Modified:   0_data/RDS_objects/lmTreat_sig.rds
    Modified:   0_data/RDS_objects/pub.rds
    Deleted:    1_analysis/mmu04060.pv.png
    Deleted:    1_analysis/mmu04151.pv.png
    Deleted:    1_analysis/mmu04270.pv.png
    Deleted:    1_analysis/mmu04510.pv.png
    Deleted:    1_analysis/mmu04640.pv.png
    Deleted:    1_analysis/mmu04670.pv.png
    Modified:   2_plots/de/pval_1.05.svg
    Modified:   2_plots/de/pval_1.1.svg
    Modified:   2_plots/de/pval_1.5.svg
    Modified:   2_plots/go/bp_dot_1.05.svg
    Modified:   2_plots/go/bp_dot_1.5.svg
    Modified:   2_plots/go/cc_dot_1.05.svg
    Modified:   2_plots/go/cc_dot_1.5.svg
    Modified:   2_plots/go/mf_dot_1.5.svg
    Modified:   2_plots/go/upset_1.05.svg
    Modified:   2_plots/go/upset_1.1.svg
    Modified:   2_plots/go/upset_1.5.svg
    Modified:   2_plots/ipa/Cell-To-Cell Signaling.svg
    Modified:   2_plots/ipa/diseaseAndFunction.svg
    Modified:   2_plots/ipa/pathways.svg
    Modified:   2_plots/kegg/kegg_dot_1.05.svg
    Modified:   2_plots/kegg/kegg_dot_1.1.svg
    Modified:   2_plots/kegg/kegg_dot_1.5.svg
    Modified:   2_plots/kegg/upset_kegg_1.05.svg
    Modified:   2_plots/kegg/upset_kegg_1.1.svg
    Modified:   2_plots/kegg/upset_kegg_1.5.svg
    Modified:   2_plots/qc/PCA_IntvsCont.svg
    Modified:   2_plots/qc/counts_after_filtering_3_3.svg
    Modified:   2_plots/qc/counts_before_after_filtering_3_3.svg
    Modified:   2_plots/qc/counts_before_filtering.svg
    Modified:   2_plots/qc/library_size.svg
    Modified:   2_plots/reactome/react_dot_1.05.svg
    Modified:   2_plots/reactome/react_dot_1.1.svg
    Modified:   2_plots/reactome/react_dot_1.5.svg
    Modified:   2_plots/reactome/upset_react_1.05.svg
    Modified:   2_plots/reactome/upset_react_1.1.svg
    Modified:   2_plots/reactome/upset_react_1.5.svg
    Modified:   3_output/enrichGO_sig.xlsx
    Modified:   3_output/enrichKEGG_all.xlsx
    Modified:   3_output/enrichKEGG_sig.xlsx
    Modified:   3_output/lmTreat_all.xlsx
    Modified:   3_output/lmTreat_fc1.5_voom2_all_fdr.xlsx
    Modified:   3_output/lmTreat_sig.xlsx
    Modified:   3_output/reactome_all.xlsx
    Modified:   3_output/reactome_sig.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>1_analysis/kegg.Rmd</code>) and HTML
(<code>docs/kegg.html</code>) files. If you’ve configured a remote Git
repository (see <code>?wflow_git_remote</code>), click on the hyperlinks
in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/18c64635cc1b08f42233ef4416cc5f1869e6e3e6/1_analysis/kegg.Rmd" target="_blank">18c6463</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-09-16
</td>
<td>
Added KEGG copyright permission and changed kable
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/bddaca35450da776c2b4dbed41babd0a9c7f5a63/1_analysis/kegg.Rmd" target="_blank">bddaca3</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-23
</td>
<td>
fixed kegg table and made minor adjustments
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/b44640abac9ec4e34694489147942349d93fa0c2/docs/kegg.html" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/eb10bef914f0d0b9b1743ea9b235859ce87feba4/1_analysis/kegg.Rmd" target="_blank">eb10bef</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-22
</td>
<td>
workflowr::wflow_publish(here::here("1_analysis/*.Rmd"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/8c9178d0ddead7d974f55328edb95ebe40c0a3dd/docs/kegg.html" target="_blank">8c9178d</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
added png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/d578f46c648558988d8eac977c170c39e29213ab/docs/kegg.html" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/1_analysis/kegg.Rmd" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
adding for pathview
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/159f3529790db3ae75472a460696c82121d932a3/docs/kegg.html" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
adding for pathview
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/1_analysis/kegg.Rmd" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
<td>
test png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/kegg.html" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
<td>
test png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/691cf3499cc98849bde703fde17b25047c072b71/docs/kegg.html" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/7f6bab239f8f9682dab208d1c314dd72dbbd21bc/1_analysis/kegg.Rmd" target="_blank">7f6bab2</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
workflowr::wflow_publish(here::here("1_analysis/*.Rmd"))
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b6cf1906d2fd775f953bd1dd6b835234e260209c/1_analysis/kegg.Rmd" target="_blank">b6cf190</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-19
</td>
<td>
quick commit
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/1_analysis/kegg.Rmd" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/kegg.html" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)
library(KEGGREST)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(viridis)
library(cowplot)
library(pheatmap)
library(DT)

# Custom ggplot
library(ggplotify)
library(ggpubr)
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Mm.eg.db)
library(enrichplot)
library(pathview)

theme_set(theme_minimal())
pub &lt;- readRDS(here::here(&quot;0_data/RDS_objects/pub.rds&quot;))
DT &lt;- readRDS(here::here(&quot;0_data/RDS_objects/DT.rds&quot;))</code></pre>
<div id="import-dgelist-data" class="section level3">
<h3>Import DGElist Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and
gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/RDS_objects/dge.rds&quot;))
fc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/fc.rds&quot;))
lfc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lfc.rds&quot;))
lmTreat &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat.rds&quot;))
lmTreat_sig &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat_sig.rds&quot;))</code></pre>
</div>
</div>
<div id="kegg-analysis" class="section level1 tabset">
<h1 class="tabset">KEGG Analysis</h1>
<p>KEGG enrichment analysis is performed with the significant DE genes
that have absolute FC &gt; 1.5 ( genes from Limma). Top 30 most
significant KEGG are displayed. All enriched KEGG pathways are
exported.</p>
<p><em>KEGG pathway images reproduced by permission from Kanehisa
Laboratories, September 2023</em></p>
<pre class="r"><code># chosing the pathways of interest
kegg_id &lt;- c(&quot;mmu04670&quot;, &quot;mmu04640&quot;, &quot;mmu04270&quot;, &quot;mmu04151&quot;, &quot;mmu04510&quot;, &quot;mmu04060&quot;)
kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)</code></pre>
<pre class="r"><code>enrichKEGG &lt;- list()
enrichKEGG_all &lt;- list()
enrichKEGG_sig &lt;- list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  # find enriched KEGG pathways
  enrichKEGG[[x]] &lt;- clusterProfiler::enrichKEGG(
    gene = lmTreat_sig[[x]]$entrezid,
    keyType = &quot;kegg&quot;,
    organism = &quot;mmu&quot;,
    pvalueCutoff = 0.05,
    pAdjustMethod = &quot;none&quot;
  )

  enrichKEGG[[x]] &lt;- enrichKEGG[[x]] %&gt;% 
    clusterProfiler::setReadable(OrgDb = org.Mm.eg.db, keyType = &quot;ENTREZID&quot;)
   
  enrichKEGG_all[[x]] &lt;- enrichKEGG[[x]]@result

  # filter the significant and print top 30
  enrichKEGG_sig[[x]] &lt;- enrichKEGG_all[[x]] %&gt;%
    dplyr::filter(pvalue &lt;= 0.05) %&gt;%
    separate(col = BgRatio, sep = &quot;/&quot;, into = c(&quot;Total&quot;, &quot;Universe&quot;)) %&gt;%
    dplyr::mutate(
      logPval = -log(pvalue, 10),
      GeneRatio = Count / as.numeric(Total)
    ) %&gt;%
    dplyr::select(c(&quot;Description&quot;, &quot;GeneRatio&quot;, &quot;pvalue&quot;, &quot;logPval&quot;, &quot;p.adjust&quot;, &quot;qvalue&quot;, &quot;geneID&quot;, &quot;Count&quot;))
  
  # # at the beginnning of a word (after 35 characters), add a newline. shorten the y axis for dot plot 
  # enrichKEGG_sig[[x]]$Description &lt;- sub(pattern = &quot;(.{1,35})(?:$| )&quot;, 
  #                                      replacement = &quot;\\1\n&quot;, 
  #                                      x = enrichKEGG_sig[[x]]$Description)
  # 
  # # remove the additional newline at the end of the string
  # enrichKEGG_sig[[x]]$Description &lt;- sub(pattern = &quot;\n$&quot;, 
  #                                      replacement = &quot;&quot;, 
  #                                      x = enrichKEGG_sig[[x]]$Description)
}</code></pre>
<div id="fc1.05" class="section level2 tabset">
<h2 class="tabset">FC=1.05</h2>
<pre class="r"><code>p=1</code></pre>
<div id="table" class="section level3">
<h3>Table</h3>
<pre class="r"><code>enrichKEGG_sig[[p]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;Significantly enriched KEGG pathways&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8d32cfd9db97395c4a5f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8d32cfd9db97395c4a5f">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Significantly enriched KEGG pathways<\/caption>","data":[["mmu04713","mmu04514","mmu04724","mmu04371","mmu05414","mmu04340","mmu04015","mmu05032","mmu04022","mmu05412","mmu04020","mmu04512","mmu04924","mmu04926","mmu04725","mmu04360","mmu04510","mmu04014","mmu04728","mmu04974","mmu05410","mmu04151","mmu04640","mmu04970","mmu04062","mmu04727","mmu00514","mmu04921","mmu04330","mmu04072","mmu04670","mmu04310","mmu04261","mmu04010","mmu04611","mmu04658","mmu05205","mmu05217","mmu04024","mmu04726","mmu04730","mmu04916","mmu04933","mmu04672","mmu02010","mmu04925","mmu04742","mmu04540","mmu00533","mmu05323","mmu04270","mmu00260","mmu00230","mmu04390","mmu04723","mmu04927","mmu04659","mmu04929","mmu05100","mmu04260","mmu04350","mmu05416","mmu00601","mmu04810","mmu04923","mmu00061","mmu00565","mmu00062","mmu04961","mmu01212","mmu04928"],["Circadian entrainment - Mus musculus (house mouse)","Cell adhesion molecules - Mus musculus (house mouse)","Glutamatergic synapse - Mus musculus (house mouse)","Apelin signaling pathway - Mus musculus (house mouse)","Dilated cardiomyopathy - Mus musculus (house mouse)","Hedgehog signaling pathway - Mus musculus (house mouse)","Rap1 signaling pathway - Mus musculus (house mouse)","Morphine addiction - Mus musculus (house mouse)","cGMP-PKG signaling pathway - Mus musculus (house mouse)","Arrhythmogenic right ventricular cardiomyopathy - Mus musculus (house mouse)","Calcium signaling pathway - Mus musculus (house mouse)","ECM-receptor interaction - Mus musculus (house mouse)","Renin secretion - Mus musculus (house mouse)","Relaxin signaling pathway - Mus musculus (house mouse)","Cholinergic synapse - Mus musculus (house mouse)","Axon guidance - Mus musculus (house mouse)","Focal adhesion - Mus musculus (house mouse)","Ras signaling pathway - Mus musculus (house mouse)","Dopaminergic synapse - Mus musculus (house mouse)","Protein digestion and absorption - Mus musculus (house mouse)","Hypertrophic cardiomyopathy - Mus musculus (house mouse)","PI3K-Akt signaling pathway - Mus musculus (house mouse)","Hematopoietic cell lineage - Mus musculus (house mouse)","Salivary secretion - Mus musculus (house mouse)","Chemokine signaling pathway - Mus musculus (house mouse)","GABAergic synapse - Mus musculus (house mouse)","Other types of O-glycan biosynthesis - Mus musculus (house mouse)","Oxytocin signaling pathway - Mus musculus (house mouse)","Notch signaling pathway - Mus musculus (house mouse)","Phospholipase D signaling pathway - Mus musculus (house mouse)","Leukocyte transendothelial migration - Mus musculus (house mouse)","Wnt signaling pathway - Mus musculus (house mouse)","Adrenergic signaling in cardiomyocytes - Mus musculus (house mouse)","MAPK signaling pathway - Mus musculus (house mouse)","Platelet activation - Mus musculus (house mouse)","Th1 and Th2 cell differentiation - Mus musculus (house mouse)","Proteoglycans in cancer - Mus musculus (house mouse)","Basal cell carcinoma - Mus musculus (house mouse)","cAMP signaling pathway - Mus musculus (house mouse)","Serotonergic synapse - Mus musculus (house mouse)","Long-term depression - Mus musculus (house mouse)","Melanogenesis - Mus musculus (house mouse)","AGE-RAGE signaling pathway in diabetic complications - Mus musculus (house mouse)","Intestinal immune network for IgA production - Mus musculus (house mouse)","ABC transporters - Mus musculus (house mouse)","Aldosterone synthesis and secretion - Mus musculus (house mouse)","Taste transduction - Mus musculus (house mouse)","Gap junction - Mus musculus (house mouse)","Glycosaminoglycan biosynthesis - keratan sulfate - Mus musculus (house mouse)","Rheumatoid arthritis - Mus musculus (house mouse)","Vascular smooth muscle contraction - Mus musculus (house mouse)","Glycine, serine and threonine metabolism - Mus musculus (house mouse)","Purine metabolism - Mus musculus (house mouse)","Hippo signaling pathway - Mus musculus (house mouse)","Retrograde endocannabinoid signaling - Mus musculus (house mouse)","Cortisol synthesis and secretion - Mus musculus (house mouse)","Th17 cell differentiation - Mus musculus (house mouse)","GnRH secretion - Mus musculus (house mouse)","Bacterial invasion of epithelial cells - Mus musculus (house mouse)","Cardiac muscle contraction - Mus musculus (house mouse)","TGF-beta signaling pathway - Mus musculus (house mouse)","Viral myocarditis - Mus musculus (house mouse)","Glycosphingolipid biosynthesis - lacto and neolacto series - Mus musculus (house mouse)","Regulation of actin cytoskeleton - Mus musculus (house mouse)","Regulation of lipolysis in adipocytes - Mus musculus (house mouse)","Fatty acid biosynthesis - Mus musculus (house mouse)","Ether lipid metabolism - Mus musculus (house mouse)","Fatty acid elongation - Mus musculus (house mouse)","Endocrine and other factor-regulated calcium reabsorption - Mus musculus (house mouse)","Fatty acid metabolism - Mus musculus (house mouse)","Parathyroid hormone synthesis, secretion and action - Mus musculus (house mouse)"],["0.245","0.186","0.211","0.19","0.213","0.259","0.154","0.209","0.162","0.221","0.142","0.202","0.211","0.171","0.179","0.149","0.143","0.136","0.163","0.176","0.187","0.118","0.181","0.186","0.141","0.18","0.233","0.144","0.19","0.141","0.151","0.131","0.135","0.113","0.144","0.159","0.122","0.175","0.116","0.13","0.167","0.14","0.139","0.186","0.173","0.137","0.141","0.14","0.286","0.138","0.118","0.175","0.119","0.115","0.115","0.139","0.124","0.143","0.132","0.126","0.118","0.125","0.185","0.0996","0.14","0.211","0.146","0.172","0.131","0.129","0.111"],["1.17e-08","2.39e-08","2.59e-07","7.06e-07","2.16e-06","3.17e-06","3.27e-06","5.21e-06","7.11e-06","7.39e-06","7.83e-06","1.46e-05","2.58e-05","2.96e-05","3.42e-05","4.65e-05","5.54e-05","6.03e-05","6.07e-05","6.62e-05","7.31e-05","0.00011","0.000111","0.000125","0.00013","0.000189","0.000363","0.000391","0.000688","0.000689","0.000704","0.00105","0.00126","0.00126","0.00127","0.00164","0.00191","0.00237","0.00313","0.00516","0.00522","0.00546","0.00597","0.00612","0.00613","0.00652","0.00674","0.00998","0.0108","0.0109","0.013","0.0141","0.0142","0.0143","0.0167","0.0184","0.0194","0.0208","0.026","0.0262","0.0274","0.0283","0.0289","0.0292","0.0314","0.0322","0.0356","0.0382","0.0446","0.0484","0.0499"],["7.93","7.62","6.59","6.15","5.66","5.5","5.49","5.28","5.15","5.13","5.11","4.83","4.59","4.53","4.47","4.33","4.26","4.22","4.22","4.18","4.14","3.96","3.95","3.9","3.89","3.72","3.44","3.41","3.16","3.16","3.15","2.98","2.9","2.9","2.9","2.79","2.72","2.62","2.5","2.29","2.28","2.26","2.22","2.21","2.21","2.19","2.17","2","1.97","1.96","1.89","1.85","1.85","1.84","1.78","1.74","1.71","1.68","1.59","1.58","1.56","1.55","1.54","1.53","1.5","1.49","1.45","1.42","1.35","1.32","1.3"],["1.17e-08","2.39e-08","2.59e-07","7.06e-07","2.16e-06","3.17e-06","3.27e-06","5.21e-06","7.11e-06","7.39e-06","7.83e-06","1.46e-05","2.58e-05","2.96e-05","3.42e-05","4.65e-05","5.54e-05","6.03e-05","6.07e-05","6.62e-05","7.31e-05","0.00011","0.000111","0.000125","0.00013","0.000189","0.000363","0.000391","0.000688","0.000689","0.000704","0.00105","0.00126","0.00126","0.00127","0.00164","0.00191","0.00237","0.00313","0.00516","0.00522","0.00546","0.00597","0.00612","0.00613","0.00652","0.00674","0.00998","0.0108","0.0109","0.013","0.0141","0.0142","0.0143","0.0167","0.0184","0.0194","0.0208","0.026","0.0262","0.0274","0.0283","0.0289","0.0292","0.0314","0.0322","0.0356","0.0382","0.0446","0.0484","0.0499"],["2.93e-06","2.99e-06","2.16e-05","4.42e-05","0.000108","0.000117","0.000117","0.000163","0.000178","0.000178","0.000178","0.000306","0.000497","0.00053","0.000572","0.000728","8e-04","8e-04","8e-04","0.000829","0.000872","0.00121","0.00121","0.0013","0.00131","0.00182","0.00336","0.0035","0.00569","0.00569","0.00569","0.00823","0.00906","0.00906","0.00906","0.0114","0.0129","0.0157","0.0201","0.0319","0.0319","0.0326","0.0341","0.0341","0.0341","0.0355","0.0359","0.0521","0.0547","0.0547","0.0638","0.0664","0.0664","0.0664","0.0761","0.0823","0.0854","0.09","0.11","0.11","0.113","0.114","0.114","0.114","0.121","0.122","0.133","0.141","0.162","0.173","0.176"],["Kcnj3/Gria4/Cacna1c/Ryr1/Gnai1/Gucy1a2/Grin2b/Adcy3/Gngt2/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Gng11/Gng2/Adcyap1r1/Rps6ka5/Plcb2/Ryr3/Cacna1h/Gnb5/Gng7/Gnb4","Itga4/Lrrc4c/Ptprd/Nlgn1/Cdh2/Itgb7/Lrrc4b/Icam1/Jam3/Cldn8/Slitrk3/Ptprm/Cdh5/Cd226/Itgal/Esam/H2-Ab1/Nlgn3/Nfasc/H2-Ob/Sele/Pecam1/H2-Eb1/Neo1/Cldn5/H2-Aa/Ntng1/Itga8/Spn/Cldn1/Cd4/Vcan/Nlgn2/Jam2","Slc1a3/Kcnj3/Gria4/Cacna1c/Grik1/Gnai1/Grin2b/Adcy3/Shank2/Gngt2/Plcb1/Adcy4/Adcy7/Gng11/Grik2/Gng2/Plcb2/Slc38a1/Dlg4/Gnb5/Gng7/Gnb4/Cacna1a/Pld2","Ryr1/Slc8a1/Nos3/Gnai1/Mef2c/Adcy3/Gngt2/Pik3r6/Plcb1/Slc8a3/Adcy4/Adcy7/Gng11/Aplnr/Agtr1a/Gng2/Myl3/Plcb2/Ryr3/Akt3/Ccn2/Gnb5/Gng7/Apln/Gnb4/Lipe","Cacna1c/Sgcd/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Adcy3/Itga1/Slc8a3/Adcy4/Adcy7/Lama2/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4","Gli1/Ihh/Hhip/Evc2/Ptch2/Evc/Kif7/Ccnd2/Gli3/Lrp2/Gpr161/Arrb1/Arrb2/Ptch1/Iqce","Hgf/Angpt1/Kdr/Igf1/Evl/Rasgrp3/Gnai1/Pfn2/Lpar4/Grin2b/Prkd1/Adcy3/Tek/Plcb1/Rasgrp2/Kit/Adcy4/Adcy7/Itgal/Vegfc/Magi2/Pard6g/Kitl/Arap3/Tln2/Plcb2/Akt3/Apbb1ip/Lat/Pdgfrb/Lcp2/F2r/Sipa1","Kcnj3/Gnai1/Pde1b/Adcy3/Gngt2/Adcy4/Adcy7/Gabra4/Gng11/Pde1a/Gng2/Arrb1/Arrb2/Gnb5/Gng7/Gnb4/Gabbr1/Pde8a/Cacna1a","Cacna1c/Adra1a/Slc8a1/Nos3/Gnai1/Ednrb/Mef2c/Gucy1a2/Adcy3/Pik3r6/Plcb1/Slc8a3/Trpc6/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Bdkrb2/Nfatc4/Plcb2/Pde5a/Akt3/Atp2b4/Atp1a2/Irs1","Cacna1c/Lef1/Sgcd/Itga4/Slc8a1/Cdh2/Itgb7/Cacna2d1/Itga1/Slc8a3/Lama2/Itga7/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4","Hgf/Cacna1c/Kdr/Ryr1/Adra1a/Slc8a1/Nos3/P2rx2/Ednrb/Grin2b/Pde1b/Adcy3/Cacna1e/Ptgfr/Plcb1/Slc8a3/Adcy4/Adcy7/Vegfc/Cd38/Agtr1a/Pde1a/Ednra/Bdkrb2/Htr2a/Nfatc4/Plcb2/Asph/Ryr3/Cacna1h/Ptger3/Atp2b4/Pdgfrb/Cacna1a/F2r/Camk1","Itga4/Col4a6/Thbs3/Npnt/Cd44/Lamc3/Lama4/Itgb7/Fn1/Itga1/Col9a2/Frem1/Lama2/Itga7/Sv2c/Itga8/Itgb4/Col4a5","Cacna1c/Gnai1/Gucy1a2/Pde1b/Edn3/Plcb1/Gucy1b1/Gucy1a1/Agtr1a/Pde1a/Kcnma1/Ednra/Adcyap1r1/Plcb2/Clca2/Ptger2","Col4a6/Nos3/Gnai1/Ednrb/Mmp2/Shc3/Adcy3/Gngt2/Plcb1/Adcy4/Adcy7/Gng11/Vegfc/Gng2/Arrb1/Plcb2/Arrb2/Akt3/Gnb5/Gng7/Gnb4/Col4a5","Kcnj3/Cacna1c/Gnai1/Kcnq5/Adcy3/Gngt2/Pik3r6/Plcb1/Adcy4/Adcy7/Gng11/Gng2/Plcb2/Fyn/Akt3/Gnb5/Gng7/Gnb4/Kcnq4/Cacna1a","Epha8/Efnb3/Sema5a/Lrrc4c/Dcc/Plxnb1/Sema5b/Nrp1/Sema7a/Gnai1/Cxcl12/Trpc6/Gdf7/Epha3/Sema3g/Pard6g/Nfatc4/Epha7/Neo1/Ptch1/Fyn/Unc5c/Ngef/Sema6a/Ntng1/Bmp7/Slit3","Hgf/Itga4/Kdr/Igf1/Col4a6/Thbs3/Lamc3/Lama4/Ccnd2/Itgb7/Fn1/Shc3/Pip5k1b/Itga1/Col9a2/Lama2/Vegfc/Itga7/Parvg/Parvb/Flnc/Tln2/Fyn/Akt3/Pdgfrb/Cav1/Itga8/Itgb4/Col4a5","Hgf/Angpt1/Pla2g10/Kdr/Igf1/Ets1/Rasgrp3/Shc3/Rasa3/Grin2b/Tek/Gngt2/Rasgrp2/Flt3/Kit/Gng11/Vegfc/Igf2/Gng2/Kitl/Pla1a/Akt3/Gnb5/Gng7/Ets2/Syngap1/Zap70/Gnb4/Lat/Pdgfrb/Pld2/Rasal3","Kcnj3/Gria4/Cacna1c/Maob/Drd4/Gnai1/Scn1a/Grin2b/Maoa/Gngt2/Plcb1/Gng11/Gng2/Arrb1/Plcb2/Arrb2/Akt3/Gnb5/Gng7/Gnb4/Cacna1a/Kif5a","Col11a1/Col26a1/Col14a1/Col4a6/Slc8a1/Pga5/Col16a1/Slc36a4/Slc8a3/Col9a2/Col13a1/Mme/Col5a2/Col15a1/Atp1a2/Slc7a7/Col23a1/Col18a1/Col4a5","Cacna1c/Sgcd/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Itga1/Slc8a3/Lama2/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4","Hgf/Angpt1/Itga4/Kdr/Igf1/Col4a6/Thbs3/Nos3/Lamc3/Lama4/Ccnd2/Itgb7/Fn1/Areg/Lpar4/Tek/Gngt2/Pik3r6/Itga1/Col9a2/Flt3/Kit/Gng11/Lama2/Vegfc/Magi2/Igf2/Gng2/Itga7/Kitl/Akt3/Gnb5/Gng7/Gnb4/Pdgfrb/Il2rg/Itga8/Itgb4/F2r/Il7r/Myc/Col4a5/Irs1","Itga4/Cd44/Il11ra1/Cd3g/Itga1/Flt3/Cd1d1/Kit/Mme/H2-Ab1/Cd38/H2-Ob/Kitl/H2-Eb1/H2-Aa/Il7r/Cd4","Adra1a/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Lyz2/Kcnma1/Plcb2/Ryr3/Lyz1/Atp2b4/Atp1a2","Cxcl15/Gnai1/Shc3/Itk/Adcy3/Cxcr3/Cxcl12/Gngt2/Pik3r6/Plcb1/Dock2/Rasgrp2/Adcy4/Adcy7/Gng11/Gng2/Xcl1/Arrb1/Plcb2/Arrb2/Stat5b/Xcr1/Fgr/Akt3/Gnb5/Gng7/Gnb4","Cacna1c/Abat/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gabra4/Gng11/Gng2/Slc38a1/Gnb5/Gng7/Gnb4/Gabbr1/Cacna1a","St6gal2/Gxylt2/B4galt2/Galnt16/Lfng/Eogt/Galntl6/Pomt2/Mfng/Galnt18","Kcnj3/Cacna1c/Ryr1/Nos3/Gnai1/Cacna2d1/Mef2c/Gucy1a2/Adcy3/Pik3r6/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Nfatc4/Cacng7/Plcb2/Ryr3/Cacnb4/Camk1","Aph1b/Lfng/Jag2/Heyl/Dtx3/Tle1/Mfng/Psen1/Maml3/Tle2/Hey2/Numbl","Shc3/Pip5k1b/Lpar4/Adcy3/Ptgfr/Pik3r6/Plcb1/Plpp3/Kit/Adcy4/Adcy7/Dgkb/Agtr1a/Kitl/Plcb2/Fyn/Akt3/Plpp1/Pdgfrb/Pld2/F2r","Itga4/Gnai1/Mmp2/Icam1/Itk/Jam3/Cxcl12/Cldn8/Cdh5/Thy1/Msn/Itgal/Esam/Pecam1/Cldn5/Cldn1/Sipa1/Jam2","Sfrp2/Wnt6/Lef1/Apcdd1/Wnt11/Nkd1/Ccnd2/Plcb1/Peg12/Wnt9a/Znrf3/Prickle2/Daam2/Prickle1/Nfatc4/Plcb2/Tle1/Fzd2/Psen1/Nkd2/Tle2/Ror1/Myc","Cacna1c/Adra1a/Slc8a1/Scn4b/Gnai1/Cacna2d1/Adcy3/Pik3r6/Plcb1/Slc8a3/Adcy4/Adcy7/Agtr1a/Myl3/Rps6ka5/Cacng7/Plcb2/Akt3/Atp2b4/Atp1a2/Cacnb4","Hgf/Cacna1c/Angpt1/Kdr/Igf1/Rasgrp3/Cacna2d1/Areg/Mef2c/Tek/Mapk8ip1/Cacna1e/Rasgrp2/Rps6ka2/Flt3/Kit/Vegfc/Igf2/Kitl/Rps6ka5/Arrb1/Cacng7/Flnc/Arrb2/Irak4/Cacna1h/Akt3/Pdgfrb/Map4k2/Cacna1a/Mknk2/Cacnb4/Myc/Ptpn7","Nos3/Gnai1/Gucy1a2/Adcy3/Pik3r6/Plcb1/Rasgrp2/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Tln2/Plcb2/Fyn/Akt3/Apbb1ip/Lcp2/F2r","Cd3g/Jag2/H2-Ab1/H2-Ob/H2-Eb1/Stat5b/Nfkbie/H2-Aa/Zap70/Lat/Il12rb1/Il2rg/Maml3/Cd4","Ihh/Hgf/Wnt6/Wnt11/Hoxd10/Kdr/Igf1/Hcls1/Cd44/Gpc3/Fn1/Mmp2/Ank2/Wnt9a/Msn/Igf2/Twist2/Flnc/Ptch1/Fzd2/Akt3/Cav1/Lum/Myc/Twist1","Gli1/Wnt6/Hhip/Lef1/Wnt11/Ptch2/Kif7/Gli3/Wnt9a/Ptch1/Fzd2","Gli1/Gria4/Cacna1c/Hhip/Vipr2/Gnai1/Grin2b/Adcy3/Edn3/Gli3/Htr1b/Adcy4/Adcy7/Ednra/Adcyap1r1/Arap3/Ptch1/Akt3/Ptger3/Atp2b4/Atp1a2/Gabbr1/Lipe/Pld2/F2r/Ptger2","Kcnj3/Cacna1c/Maob/Kcnd2/Gnai1/Maoa/Gngt2/Plcb1/Htr1b/Gng11/Gng2/Htr2a/Plcb2/Gnb5/Gng7/Gnb4/Cacna1a","Ryr1/Igf1/Gnai1/Gucy1a2/Plcb1/Gucy1b1/Gucy1a1/Plcb2/Gnaz/Cacna1a","Wnt6/Lef1/Wnt11/Gnai1/Ednrb/Adcy3/Plcb1/Wnt9a/Kit/Adcy4/Adcy7/Kitl/Plcb2/Fzd2","Col4a6/Nos3/Fn1/Mmp2/Icam1/Plcb1/Vegfc/Agtr1a/Nox4/Sele/Plcb2/Stat5b/Akt3/Col4a5","Itga4/Itgb7/Cxcl12/Tnfsf13b/H2-Ab1/H2-Ob/H2-Eb1/H2-Aa","Abcg2/Abca5/Abcg1/Abca8b/Abcd2/Abcb1a/Abcc5/Abcg3/Abcg4","Kcnk3/Cacna1c/Prkd1/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Atp2b4/Atp1a2/Lipe/Camk1","Cacna1c/P2rx2/Pde1b/Plcb1/Htr1b/Scn2a/Adcy4/Gabra4/Pde1a/Plcb2/Scn3a/Gabbr1/Cacna1a","Gnai1/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Tubb4a/Htr2a/Plcb2/Pdgfrb","Chst2/B4galt2/St3gal2/Chst1","Angpt1/Icam1/Tek/Cxcl12/Tnfsf13b/Itgal/H2-Ab1/H2-Ob/Atp6v0e2/H2-Eb1/H2-Aa/Atp6v1b2","Cacna1c/Pla2g10/Adra1a/Gucy1a2/Adcy3/Edn3/Plcb1/Calca/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Plcb2/Calcrl","Maob/Cbs/Maoa/Gamt/Cth/Phgdh/Sardh","Nme4/Gucy1a2/Pde1b/Adcy3/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Enpp1/Pde1a/Hddc3/Lacc1/Pde5a/Impdh1/Pde8a/Papss2","Wnt6/Lef1/Wnt11/Nkd1/Ccnd2/Areg/Gdf7/Wnt9a/Dlg2/Pard6g/Dlg4/Fzd2/Ccn2/Dlg5/Lats2/Nkd2/Bmp7/Myc","Kcnj3/Gria4/Cacna1c/Gnai1/Adcy3/Gngt2/Plcb1/Adcy4/Adcy7/Gabra4/Gng11/Gng2/Plcb2/Gnb5/Gng7/Gnb4/Cacna1a","Kcnk3/Cacna1c/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Pde8a","Cd3g/H2-Ab1/Il27ra/H2-Ob/H2-Eb1/Stat5b/Nfkbie/H2-Aa/Zap70/Lat/Il12rb1/Il2rg/Cd4","Kcnj3/Cacna1c/Plcb1/Arrb1/Plcb2/Arrb2/Cacna1h/Akt3/Gabbr1","Hcls1/Fn1/Shc3/Septin3/Arhgap10/Arhgef26/Elmo2/Cav1/Actr3b/Septin6","Cacna1c/Slc8a1/Cacna2d1/Slc9a7/Slc8a3/Myl3/Cacng7/Asph/Cox4i2/Atp1a2/Cacnb4","Nog/Fst/Id3/Id4/Gdf7/Tgif2/Emp3/Smad9/Neo1/Nrros/Zfyve9/Bmp7/Myc","Sgcd/Icam1/Lama2/Itgal/H2-Ab1/H2-Ob/H2-Eb1/Fyn/H2-Aa/Sgcb/Cav1","B4galt2/B3galnt1/Ggta1/B3gnt5/St3gal4","Itga4/Itgb7/Fn1/Pip5k1b/Pfn2/Lpar4/Cxcl12/Itga1/Msn/Abi2/Itgal/Itga7/Scin/Nckap1l/Bdkrb2/Fgd1/Akt3/Pdgfrb/Cyfip2/Actr3b/Itga8/Itgb4/F2r","Gnai1/Adcy3/Adcy4/Adcy7/Akt3/Ptger3/Lipe/Irs1","Acsl6/Acacb/Acsl1/Olah","Pla2g10/Plpp3/Lpcat1/Plpp1/Pld2/Enpp2/Pafah2","Elovl4/Them5/Echs1/Acot7/Hacd1","Slc8a1/Plcb1/Slc8a3/Pth1r/Bdkrb2/Plcb2/Atp2b4/Atp1a2","Acsl6/Fads2/Fads1/Elovl4/Echs1/Cpt1c/Acsl1/Hacd1","Mmp16/Gnai1/Mef2c/Adcy3/Plcb1/Pth1r/Adcy4/Adcy7/Arrb1/Plcb2/Arrb2/Pld2"],["24","34","24","26","20","15","33","19","28","17","36","18","16","22","20","27","29","32","22","19","17","43","17","16","27","16","10","22","12","21","18","23","21","34","18","14","25","11","26","17","10","14","14","8","9","14","13","12","4","12","17","7","16","18","17","10","13","9","10","11","13","11","5","23","8","4","7","5","8","8","12"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>pvalue<\/th>\n      <th>logPval<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable(caption = &quot;Significantly enriched KEGG pathways&quot;) %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="dot-plot" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>kegg_dot &lt;- list()
upset=list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()

  # dot plot, save
  kegg_dot[[x]] &lt;- ggplot(enrichKEGG_sig[[x]][1:15, ]) +
    geom_point(aes(x = GeneRatio, y = reorder(Description, GeneRatio), colour = logPval, size = Count)) +
    scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits = c(0, NA)) +
    scale_size(range = c(1.5,5)) +
    ggtitle(&quot;KEGG Pathways&quot;) +
    ylab(label = &quot;&quot;) +
    xlab(label = &quot;Gene Ratio&quot;) +
    labs(color = expression(&quot;-log&quot;[10] * &quot;Pvalue&quot;), size = &quot;Gene Counts&quot;)
  ggsave(filename = paste0(&quot;kegg_dot_&quot;, fc[i], &quot;.svg&quot;), plot = kegg_dot[[x]] + pub, path = here::here(&quot;2_plots/kegg/&quot;), 
         width = 250, height = 130, units = &quot;mm&quot;)
  
  upset[[x]] &lt;- upsetplot(x = enrichKEGG[[x]], 10)
  ggsave(filename = paste0(&quot;upset_kegg_&quot;, fc[i], &quot;.svg&quot;), plot = upset[[x]], path = here::here(&quot;2_plots/kegg/&quot;), 
         width = 170, height = 130, units = &quot;mm&quot;)
}

kegg_dot[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/kegg_dot-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-kegg_dot-1">
Past versions of kegg_dot-1.png
</button>
</p>
<div id="fig-kegg_dot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/kegg_dot-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/kegg_dot-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/figure/kegg.Rmd/kegg_dot-1.png" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/kegg_dot-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/kegg_dot-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="upset-plot" class="section level3">
<h3>Upset Plot</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/upset-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-upset-1">
Past versions of upset-1.png
</button>
</p>
<div id="fig-upset-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/upset-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/upset-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/upset-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="fc1.1" class="section level2 tabset">
<h2 class="tabset">FC=1.1</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="table-1" class="section level3">
<h3>Table</h3>
<pre class="r"><code>enrichKEGG_sig[[p]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;Significantly enriched KEGG pathways&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-d701554a25665d09d09f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d701554a25665d09d09f">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Significantly enriched KEGG pathways<\/caption>","data":[["mmu04713","mmu04371","mmu04724","mmu05414","mmu04514","mmu05412","mmu04022","mmu05032","mmu04340","mmu04725","mmu04020","mmu04512","mmu04926","mmu04015","mmu04062","mmu05410","mmu04924","mmu04970","mmu04728","mmu04510","mmu04360","mmu04974","mmu04727","mmu04014","mmu04921","mmu04261","mmu04640","mmu04151","mmu05217","mmu04670","mmu04726","mmu04916","mmu04730","mmu04611","mmu04072","mmu04742","mmu00514","mmu04672","mmu05205","mmu04270","mmu04010","mmu04658","mmu04723","mmu04024","mmu04933","mmu04925","mmu04310","mmu00533","mmu02010","mmu04929","mmu04540","mmu04260","mmu05323","mmu05416","mmu04659","mmu00601","mmu04390","mmu04927","mmu00230","mmu04961","mmu04810","mmu04918","mmu00260","mmu00380","mmu04915","mmu04923","mmu05144","mmu04928","mmu04934","mmu05145"],["Circadian entrainment - Mus musculus (house mouse)","Apelin signaling pathway - Mus musculus (house mouse)","Glutamatergic synapse - Mus musculus (house mouse)","Dilated cardiomyopathy - Mus musculus (house mouse)","Cell adhesion molecules - Mus musculus (house mouse)","Arrhythmogenic right ventricular cardiomyopathy - Mus musculus (house mouse)","cGMP-PKG signaling pathway - Mus musculus (house mouse)","Morphine addiction - Mus musculus (house mouse)","Hedgehog signaling pathway - Mus musculus (house mouse)","Cholinergic synapse - Mus musculus (house mouse)","Calcium signaling pathway - Mus musculus (house mouse)","ECM-receptor interaction - Mus musculus (house mouse)","Relaxin signaling pathway - Mus musculus (house mouse)","Rap1 signaling pathway - Mus musculus (house mouse)","Chemokine signaling pathway - Mus musculus (house mouse)","Hypertrophic cardiomyopathy - Mus musculus (house mouse)","Renin secretion - Mus musculus (house mouse)","Salivary secretion - Mus musculus (house mouse)","Dopaminergic synapse - Mus musculus (house mouse)","Focal adhesion - Mus musculus (house mouse)","Axon guidance - Mus musculus (house mouse)","Protein digestion and absorption - Mus musculus (house mouse)","GABAergic synapse - Mus musculus (house mouse)","Ras signaling pathway - Mus musculus (house mouse)","Oxytocin signaling pathway - Mus musculus (house mouse)","Adrenergic signaling in cardiomyocytes - Mus musculus (house mouse)","Hematopoietic cell lineage - Mus musculus (house mouse)","PI3K-Akt signaling pathway - Mus musculus (house mouse)","Basal cell carcinoma - Mus musculus (house mouse)","Leukocyte transendothelial migration - Mus musculus (house mouse)","Serotonergic synapse - Mus musculus (house mouse)","Melanogenesis - Mus musculus (house mouse)","Long-term depression - Mus musculus (house mouse)","Platelet activation - Mus musculus (house mouse)","Phospholipase D signaling pathway - Mus musculus (house mouse)","Taste transduction - Mus musculus (house mouse)","Other types of O-glycan biosynthesis - Mus musculus (house mouse)","Intestinal immune network for IgA production - Mus musculus (house mouse)","Proteoglycans in cancer - Mus musculus (house mouse)","Vascular smooth muscle contraction - Mus musculus (house mouse)","MAPK signaling pathway - Mus musculus (house mouse)","Th1 and Th2 cell differentiation - Mus musculus (house mouse)","Retrograde endocannabinoid signaling - Mus musculus (house mouse)","cAMP signaling pathway - Mus musculus (house mouse)","AGE-RAGE signaling pathway in diabetic complications - Mus musculus (house mouse)","Aldosterone synthesis and secretion - Mus musculus (house mouse)","Wnt signaling pathway - Mus musculus (house mouse)","Glycosaminoglycan biosynthesis - keratan sulfate - Mus musculus (house mouse)","ABC transporters - Mus musculus (house mouse)","GnRH secretion - Mus musculus (house mouse)","Gap junction - Mus musculus (house mouse)","Cardiac muscle contraction - Mus musculus (house mouse)","Rheumatoid arthritis - Mus musculus (house mouse)","Viral myocarditis - Mus musculus (house mouse)","Th17 cell differentiation - Mus musculus (house mouse)","Glycosphingolipid biosynthesis - lacto and neolacto series - Mus musculus (house mouse)","Hippo signaling pathway - Mus musculus (house mouse)","Cortisol synthesis and secretion - Mus musculus (house mouse)","Purine metabolism - Mus musculus (house mouse)","Endocrine and other factor-regulated calcium reabsorption - Mus musculus (house mouse)","Regulation of actin cytoskeleton - Mus musculus (house mouse)","Thyroid hormone synthesis - Mus musculus (house mouse)","Glycine, serine and threonine metabolism - Mus musculus (house mouse)","Tryptophan metabolism - Mus musculus (house mouse)","Estrogen signaling pathway - Mus musculus (house mouse)","Regulation of lipolysis in adipocytes - Mus musculus (house mouse)","Malaria - Mus musculus (house mouse)","Parathyroid hormone synthesis, secretion and action - Mus musculus (house mouse)","Cushing syndrome - Mus musculus (house mouse)","Toxoplasmosis - Mus musculus (house mouse)"],["0.245","0.19","0.202","0.213","0.158","0.221","0.156","0.198","0.241","0.179","0.13","0.191","0.163","0.136","0.141","0.187","0.197","0.186","0.156","0.133","0.138","0.167","0.18","0.123","0.137","0.135","0.16","0.102","0.175","0.134","0.13","0.14","0.167","0.128","0.121","0.141","0.186","0.186","0.107","0.118","0.0963","0.136","0.115","0.103","0.129","0.127","0.109","0.286","0.154","0.143","0.128","0.126","0.126","0.125","0.114","0.185","0.102","0.125","0.104","0.131","0.0909","0.122","0.15","0.135","0.0977","0.123","0.123","0.102","0.0926","0.101"],["8.37e-10","5.21e-08","9.6e-08","2.67e-07","5.22e-07","1.22e-06","1.75e-06","3.23e-06","3.48e-06","4.88e-06","7.57e-06","9.89e-06","1.28e-05","1.3e-05","1.3e-05","1.35e-05","2.18e-05","2.54e-05","2.6e-05","3.6e-05","3.7e-05","3.82e-05","3.95e-05","7.55e-05","0.000168","0.000221","0.000274","0.000479","0.000814","0.00122","0.0013","0.00164","0.00201","0.00206","0.00216","0.00218","0.00273","0.00273","0.00338","0.0036","0.0042","0.00429","0.00478","0.00479","0.00499","0.00542","0.00548","0.00678","0.00902","0.00937","0.00986","0.0107","0.0107","0.0116","0.0169","0.0173","0.0181","0.0214","0.0215","0.0226","0.0246","0.0252","0.0253","0.0281","0.0417","0.0435","0.0435","0.0453","0.0454","0.0479"],["9.08","7.28","7.02","6.57","6.28","5.91","5.76","5.49","5.46","5.31","5.12","5","4.89","4.89","4.88","4.87","4.66","4.59","4.58","4.44","4.43","4.42","4.4","4.12","3.78","3.66","3.56","3.32","3.09","2.91","2.89","2.79","2.7","2.69","2.67","2.66","2.56","2.56","2.47","2.44","2.38","2.37","2.32","2.32","2.3","2.27","2.26","2.17","2.04","2.03","2.01","1.97","1.97","1.93","1.77","1.76","1.74","1.67","1.67","1.65","1.61","1.6","1.6","1.55","1.38","1.36","1.36","1.34","1.34","1.32"],["8.37e-10","5.21e-08","9.6e-08","2.67e-07","5.22e-07","1.22e-06","1.75e-06","3.23e-06","3.48e-06","4.88e-06","7.57e-06","9.89e-06","1.28e-05","1.3e-05","1.3e-05","1.35e-05","2.18e-05","2.54e-05","2.6e-05","3.6e-05","3.7e-05","3.82e-05","3.95e-05","7.55e-05","0.000168","0.000221","0.000274","0.000479","0.000814","0.00122","0.0013","0.00164","0.00201","0.00206","0.00216","0.00218","0.00273","0.00273","0.00338","0.0036","0.0042","0.00429","0.00478","0.00479","0.00499","0.00542","0.00548","0.00678","0.00902","0.00937","0.00986","0.0107","0.0107","0.0116","0.0169","0.0173","0.0181","0.0214","0.0215","0.0226","0.0246","0.0252","0.0253","0.0281","0.0417","0.0435","0.0435","0.0453","0.0454","0.0479"],["2.07e-07","6.45e-06","7.91e-06","1.65e-05","2.58e-05","5.03e-05","6.19e-05","9.56e-05","9.56e-05","0.000121","0.00017","0.000204","0.000208","0.000208","0.000208","0.000208","0.000318","0.000339","0.000339","0.000425","0.000425","0.000425","0.000425","0.000778","0.00166","0.0021","0.00251","0.00423","0.00695","0.0101","0.0104","0.0127","0.015","0.015","0.015","0.015","0.0178","0.0178","0.0214","0.0223","0.0253","0.0253","0.0269","0.0269","0.0274","0.0289","0.0289","0.0349","0.0456","0.0464","0.0478","0.0501","0.0501","0.0533","0.0761","0.0764","0.0786","0.09","0.09","0.093","0.0995","0.0995","0.0995","0.109","0.159","0.161","0.161","0.163","0.163","0.169"],["Kcnj3/Gria4/Cacna1c/Ryr1/Gnai1/Gucy1a2/Grin2b/Adcy3/Gngt2/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Gng11/Adcy7/Adcyap1r1/Gng2/Rps6ka5/Plcb2/Ryr3/Cacna1h/Gng7/Gnb5/Gnb4","Ryr1/Nos3/Slc8a1/Gnai1/Mef2c/Adcy3/Gngt2/Pik3r6/Slc8a3/Plcb1/Adcy4/Gng11/Aplnr/Adcy7/Agtr1a/Myl3/Gng2/Plcb2/Ryr3/Ccn2/Apln/Gng7/Akt3/Gnb5/Gnb4/Lipe","Kcnj3/Slc1a3/Gria4/Grik1/Cacna1c/Gnai1/Grin2b/Adcy3/Shank2/Gngt2/Plcb1/Adcy4/Gng11/Adcy7/Grik2/Gng2/Plcb2/Slc38a1/Dlg4/Gng7/Gnb5/Gnb4/Cacna1a","Sgcd/Cacna1c/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Adcy3/Slc8a3/Itga1/Adcy4/Lama2/Adcy7/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4","Itga4/Lrrc4c/Ptprd/Nlgn1/Cdh2/Lrrc4b/Itgb7/Icam1/Jam3/Cldn8/Slitrk3/Cdh5/Ptprm/Cd226/Nlgn3/Esam/Itgal/H2-Ab1/Nfasc/Sele/H2-Ob/Pecam1/H2-Eb1/Cldn5/Neo1/H2-Aa/Ntng1/Itga8/Cldn1","Lef1/Sgcd/Cacna1c/Itga4/Slc8a1/Cdh2/Itgb7/Cacna2d1/Slc8a3/Itga1/Lama2/Itga7/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4","Cacna1c/Adra1a/Nos3/Slc8a1/Gnai1/Ednrb/Mef2c/Gucy1a2/Adcy3/Pik3r6/Slc8a3/Plcb1/Gucy1b1/Trpc6/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Bdkrb2/Plcb2/Pde5a/Nfatc4/Atp2b4/Akt3/Atp1a2","Kcnj3/Gnai1/Pde1b/Adcy3/Gngt2/Gabra4/Adcy4/Gng11/Adcy7/Pde1a/Gng2/Arrb1/Arrb2/Gng7/Gnb5/Gnb4/Gabbr1/Cacna1a","Gli1/Ihh/Hhip/Evc2/Ptch2/Evc/Kif7/Ccnd2/Gli3/Lrp2/Gpr161/Arrb1/Ptch1/Arrb2","Kcnj3/Cacna1c/Gnai1/Kcnq5/Adcy3/Gngt2/Pik3r6/Plcb1/Adcy4/Gng11/Adcy7/Gng2/Plcb2/Fyn/Gng7/Akt3/Gnb5/Gnb4/Kcnq4/Cacna1a","Hgf/Cacna1c/Ryr1/Kdr/Adra1a/Nos3/Slc8a1/P2rx2/Ednrb/Grin2b/Pde1b/Adcy3/Cacna1e/Ptgfr/Slc8a3/Plcb1/Adcy4/Adcy7/Vegfc/Pde1a/Cd38/Agtr1a/Ednra/Bdkrb2/Htr2a/Plcb2/Asph/Ryr3/Nfatc4/Cacna1h/Ptger3/Atp2b4/Cacna1a","Itga4/Col4a6/Npnt/Thbs3/Lamc3/Cd44/Lama4/Itgb7/Fn1/Itga1/Col9a2/Frem1/Lama2/Itga7/Sv2c/Itga8/Itgb4","Col4a6/Nos3/Gnai1/Ednrb/Mmp2/Shc3/Adcy3/Gngt2/Plcb1/Adcy4/Gng11/Adcy7/Vegfc/Gng2/Arrb1/Plcb2/Arrb2/Gng7/Akt3/Gnb5/Gnb4","Hgf/Angpt1/Kdr/Igf1/Evl/Rasgrp3/Gnai1/Pfn2/Grin2b/Lpar4/Prkd1/Tek/Adcy3/Plcb1/Rasgrp2/Adcy4/Kit/Adcy7/Itgal/Vegfc/Magi2/Arap3/Pard6g/Kitl/Plcb2/Tln2/Akt3/Lat/Apbb1ip","Cxcl15/Gnai1/Shc3/Itk/Cxcr3/Cxcl12/Adcy3/Gngt2/Pik3r6/Plcb1/Rasgrp2/Dock2/Adcy4/Gng11/Adcy7/Xcl1/Gng2/Arrb1/Plcb2/Xcr1/Arrb2/Gng7/Fgr/Stat5b/Akt3/Gnb5/Gnb4","Sgcd/Cacna1c/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Slc8a3/Itga1/Lama2/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4","Cacna1c/Gnai1/Gucy1a2/Pde1b/Edn3/Plcb1/Gucy1b1/Gucy1a1/Pde1a/Agtr1a/Kcnma1/Ednra/Adcyap1r1/Plcb2/Clca2","Adra1a/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Lyz2/Kcnma1/Plcb2/Ryr3/Lyz1/Atp2b4/Atp1a2","Kcnj3/Gria4/Maob/Cacna1c/Drd4/Gnai1/Scn1a/Grin2b/Maoa/Gngt2/Plcb1/Gng11/Gng2/Arrb1/Plcb2/Arrb2/Gng7/Akt3/Gnb5/Gnb4/Cacna1a","Hgf/Itga4/Kdr/Igf1/Col4a6/Thbs3/Lamc3/Lama4/Itgb7/Ccnd2/Fn1/Shc3/Pip5k1b/Itga1/Col9a2/Lama2/Vegfc/Parvg/Itga7/Parvb/Flnc/Tln2/Fyn/Akt3/Cav1/Itga8/Itgb4","Epha8/Efnb3/Sema5a/Lrrc4c/Dcc/Plxnb1/Sema5b/Sema7a/Nrp1/Gnai1/Cxcl12/Gdf7/Trpc6/Epha3/Sema3g/Pard6g/Epha7/Ptch1/Nfatc4/Neo1/Fyn/Unc5c/Sema6a/Ngef/Ntng1","Col11a1/Col26a1/Col14a1/Col4a6/Slc8a1/Pga5/Col16a1/Slc36a4/Slc8a3/Col9a2/Col13a1/Mme/Col5a2/Col15a1/Atp1a2/Col23a1/Slc7a7/Col18a1","Cacna1c/Abat/Gnai1/Adcy3/Gngt2/Gabra4/Adcy4/Gng11/Adcy7/Gng2/Slc38a1/Gng7/Gnb5/Gnb4/Gabbr1/Cacna1a","Hgf/Angpt1/Pla2g10/Kdr/Igf1/Rasgrp3/Ets1/Shc3/Rasa3/Grin2b/Tek/Gngt2/Rasgrp2/Flt3/Gng11/Kit/Vegfc/Igf2/Gng2/Pla1a/Kitl/Gng7/Ets2/Akt3/Zap70/Gnb5/Lat/Syngap1/Gnb4","Kcnj3/Cacna1c/Ryr1/Nos3/Gnai1/Cacna2d1/Mef2c/Gucy1a2/Adcy3/Pik3r6/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Cacng7/Plcb2/Ryr3/Nfatc4/Cacnb4","Cacna1c/Adra1a/Slc8a1/Scn4b/Gnai1/Cacna2d1/Adcy3/Pik3r6/Slc8a3/Plcb1/Adcy4/Adcy7/Agtr1a/Myl3/Rps6ka5/Cacng7/Plcb2/Atp2b4/Akt3/Atp1a2/Cacnb4","Itga4/Cd44/Il11ra1/Cd3g/Itga1/Flt3/Mme/Cd1d1/Kit/H2-Ab1/Cd38/H2-Ob/Kitl/H2-Eb1/H2-Aa","Hgf/Angpt1/Itga4/Kdr/Igf1/Col4a6/Thbs3/Nos3/Lamc3/Lama4/Areg/Itgb7/Ccnd2/Fn1/Lpar4/Tek/Gngt2/Pik3r6/Itga1/Col9a2/Flt3/Gng11/Kit/Lama2/Vegfc/Magi2/Igf2/Itga7/Gng2/Kitl/Gng7/Akt3/Gnb5/Gnb4/Il2rg/Itga8/Itgb4","Gli1/Wnt6/Hhip/Lef1/Wnt11/Ptch2/Kif7/Gli3/Wnt9a/Ptch1/Fzd2","Itga4/Gnai1/Mmp2/Icam1/Itk/Cxcl12/Jam3/Cldn8/Cdh5/Thy1/Msn/Esam/Itgal/Pecam1/Cldn5/Cldn1","Kcnj3/Maob/Cacna1c/Kcnd2/Gnai1/Maoa/Gngt2/Plcb1/Htr1b/Gng11/Htr2a/Gng2/Plcb2/Gng7/Gnb5/Gnb4/Cacna1a","Wnt6/Lef1/Wnt11/Gnai1/Ednrb/Adcy3/Plcb1/Wnt9a/Adcy4/Kit/Adcy7/Kitl/Plcb2/Fzd2","Ryr1/Igf1/Gnai1/Gucy1a2/Plcb1/Gucy1b1/Gucy1a1/Plcb2/Gnaz/Cacna1a","Nos3/Gnai1/Gucy1a2/Adcy3/Pik3r6/Plcb1/Rasgrp2/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Plcb2/Tln2/Fyn/Akt3/Apbb1ip","Shc3/Lpar4/Pip5k1b/Adcy3/Ptgfr/Pik3r6/Plcb1/Plpp3/Adcy4/Kit/Adcy7/Dgkb/Agtr1a/Kitl/Plcb2/Fyn/Akt3/Plpp1","Cacna1c/P2rx2/Pde1b/Plcb1/Htr1b/Gabra4/Scn2a/Adcy4/Pde1a/Plcb2/Scn3a/Gabbr1/Cacna1a","St6gal2/Gxylt2/B4galt2/Galnt16/Lfng/Eogt/Galntl6/Mfng","Itga4/Itgb7/Cxcl12/Tnfsf13b/H2-Ab1/H2-Ob/H2-Eb1/H2-Aa","Ihh/Hgf/Wnt6/Wnt11/Hoxd10/Kdr/Igf1/Cd44/Hcls1/Gpc3/Fn1/Mmp2/Ank2/Wnt9a/Msn/Igf2/Twist2/Ptch1/Flnc/Fzd2/Akt3/Cav1","Cacna1c/Pla2g10/Adra1a/Gucy1a2/Adcy3/Edn3/Calca/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Plcb2/Calcrl","Hgf/Angpt1/Cacna1c/Kdr/Igf1/Rasgrp3/Areg/Cacna2d1/Mef2c/Tek/Mapk8ip1/Cacna1e/Rasgrp2/Rps6ka2/Flt3/Kit/Vegfc/Igf2/Rps6ka5/Kitl/Arrb1/Cacng7/Flnc/Irak4/Cacna1h/Arrb2/Akt3/Cacna1a/Cacnb4","Cd3g/Jag2/H2-Ab1/H2-Ob/H2-Eb1/Nfkbie/H2-Aa/Stat5b/Zap70/Lat/Il12rb1/Il2rg","Kcnj3/Gria4/Cacna1c/Gnai1/Adcy3/Gngt2/Plcb1/Gabra4/Adcy4/Gng11/Adcy7/Gng2/Plcb2/Gng7/Gnb5/Gnb4/Cacna1a","Gli1/Gria4/Hhip/Cacna1c/Vipr2/Gnai1/Grin2b/Adcy3/Edn3/Gli3/Htr1b/Adcy4/Adcy7/Ednra/Adcyap1r1/Arap3/Ptch1/Ptger3/Atp2b4/Akt3/Atp1a2/Gabbr1/Lipe","Col4a6/Nos3/Fn1/Mmp2/Icam1/Plcb1/Vegfc/Nox4/Sele/Agtr1a/Plcb2/Stat5b/Akt3","Kcnk3/Cacna1c/Prkd1/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Atp2b4/Atp1a2/Lipe","Sfrp2/Wnt6/Lef1/Apcdd1/Wnt11/Nkd1/Ccnd2/Plcb1/Peg12/Wnt9a/Prickle2/Znrf3/Daam2/Prickle1/Tle1/Plcb2/Nfatc4/Fzd2/Nkd2","Chst2/B4galt2/St3gal2/Chst1","Abcg2/Abca5/Abcg1/Abca8b/Abcd2/Abcb1a/Abcc5/Abcg3","Kcnj3/Cacna1c/Plcb1/Arrb1/Plcb2/Cacna1h/Arrb2/Akt3/Gabbr1","Gnai1/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Tubb4a/Htr2a/Plcb2","Cacna1c/Slc8a1/Cacna2d1/Slc9a7/Slc8a3/Myl3/Cacng7/Asph/Cox4i2/Atp1a2/Cacnb4","Angpt1/Icam1/Tek/Cxcl12/Tnfsf13b/Itgal/H2-Ab1/H2-Ob/Atp6v0e2/H2-Eb1/H2-Aa","Sgcd/Icam1/Lama2/Itgal/H2-Ab1/H2-Ob/H2-Eb1/Fyn/H2-Aa/Sgcb/Cav1","Cd3g/Il27ra/H2-Ab1/H2-Ob/H2-Eb1/Nfkbie/H2-Aa/Stat5b/Zap70/Lat/Il12rb1/Il2rg","B3galnt1/B4galt2/Ggta1/B3gnt5/St3gal4","Wnt6/Lef1/Wnt11/Nkd1/Areg/Ccnd2/Gdf7/Wnt9a/Dlg2/Pard6g/Ccn2/Fzd2/Dlg4/Dlg5/Lats2/Nkd2","Kcnk3/Cacna1c/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h","Nme4/Gucy1a2/Pde1b/Adcy3/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Pde1a/Enpp1/Hddc3/Lacc1/Pde5a/Impdh1","Slc8a1/Slc8a3/Plcb1/Pth1r/Bdkrb2/Plcb2/Atp2b4/Atp1a2","Itga4/Itgb7/Fn1/Pfn2/Lpar4/Pip5k1b/Cxcl12/Itga1/Msn/Abi2/Itgal/Scin/Bdkrb2/Itga7/Nckap1l/Fgd1/Akt3/Actr3b/Cyfip2/Itga8/Itgb4","Ttr/Adcy3/Plcb1/Lrp2/Adcy4/Adcy7/Plcb2/Gpx7/Atp1a2","Maob/Cbs/Maoa/Gamt/Cth/Phgdh","Cyp1b1/Maob/Maoa/Kyat1/Dhtkd1/Echs1/Aox4","Krt23/Kcnj3/Nos3/Gnai1/Mmp2/Shc3/Adcy3/Plcb1/Adcy4/Adcy7/Plcb2/Akt3/Gabbr1","Gnai1/Adcy3/Adcy4/Adcy7/Ptger3/Akt3/Lipe","Hgf/Thbs3/Icam1/Gypc/Itgal/Sele/Pecam1","Mmp16/Gnai1/Mef2c/Adcy3/Plcb1/Pth1r/Adcy4/Adcy7/Arrb1/Plcb2/Arrb2","Kcnk3/Wnt6/Lef1/Wnt11/Cacna1c/Gnai1/Adcy3/Plcb1/Wnt9a/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Fzd2","Lamc3/Lama4/Gnai1/Pik3r6/Lama2/H2-Ab1/H2-Ob/H2-Eb1/Irak4/H2-Aa/Akt3"],["24","26","23","20","29","17","27","18","14","20","33","17","21","29","27","17","15","16","21","27","25","18","16","29","21","21","15","37","11","16","17","14","10","16","18","13","8","8","22","17","29","12","17","23","13","13","19","4","8","9","11","11","11","11","12","5","16","9","14","8","21","9","6","7","13","7","7","11","15","11"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>pvalue<\/th>\n      <th>logPval<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable(caption = &quot;Significantly enriched KEGG pathways&quot;) %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="dot-plot-1" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>kegg_dot[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-6-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-6-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-6-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/figure/kegg.Rmd/unnamed-chunk-6-1.png" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-6-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-6-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="upset-plot-1" class="section level3">
<h3>Upset Plot</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-7-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-7-1">
Past versions of unnamed-chunk-7-1.png
</button>
</p>
<div id="fig-unnamed-chunk-7-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-7-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-7-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-7-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-7-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="fc1.5" class="section level2 tabset">
<h2 class="tabset">FC=1.5</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="table-2" class="section level3">
<h3>Table</h3>
<pre class="r"><code>enrichKEGG_sig[[p]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;Significantly enriched KEGG pathways&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-27a6251d00cbd347f90b" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-27a6251d00cbd347f90b">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Significantly enriched KEGG pathways<\/caption>","data":[["mmu04340","mmu05217","mmu04512","mmu04360","mmu05205","mmu04974","mmu04020","mmu04728","mmu05412","mmu04080","mmu04724","mmu04510","mmu04015","mmu04713","mmu04916","mmu05033","mmu04151","mmu00910","mmu04390","mmu05030","mmu04979","mmu04926","mmu05410"],["Hedgehog signaling pathway - Mus musculus (house mouse)","Basal cell carcinoma - Mus musculus (house mouse)","ECM-receptor interaction - Mus musculus (house mouse)","Axon guidance - Mus musculus (house mouse)","Proteoglycans in cancer - Mus musculus (house mouse)","Protein digestion and absorption - Mus musculus (house mouse)","Calcium signaling pathway - Mus musculus (house mouse)","Dopaminergic synapse - Mus musculus (house mouse)","Arrhythmogenic right ventricular cardiomyopathy - Mus musculus (house mouse)","Neuroactive ligand-receptor interaction - Mus musculus (house mouse)","Glutamatergic synapse - Mus musculus (house mouse)","Focal adhesion - Mus musculus (house mouse)","Rap1 signaling pathway - Mus musculus (house mouse)","Circadian entrainment - Mus musculus (house mouse)","Melanogenesis - Mus musculus (house mouse)","Nicotine addiction - Mus musculus (house mouse)","PI3K-Akt signaling pathway - Mus musculus (house mouse)","Nitrogen metabolism - Mus musculus (house mouse)","Hippo signaling pathway - Mus musculus (house mouse)","Cocaine addiction - Mus musculus (house mouse)","Cholesterol metabolism - Mus musculus (house mouse)","Relaxin signaling pathway - Mus musculus (house mouse)","Hypertrophic cardiomyopathy - Mus musculus (house mouse)"],["0.138","0.111","0.0899","0.0608","0.0488","0.0648","0.0435","0.0519","0.0649","0.0332","0.0526","0.0394","0.0374","0.051","0.05","0.075","0.0302","0.118","0.0382","0.0625","0.06","0.0388","0.044"],["2.38e-06","4.42e-05","5.89e-05","9.47e-05","0.00112","0.00126","0.00163","0.00447","0.00624","0.00652","0.00776","0.0122","0.0164","0.0167","0.018","0.0226","0.0228","0.0269","0.0325","0.0363","0.0402","0.0468","0.0497"],["5.62","4.35","4.23","4.02","2.95","2.9","2.79","2.35","2.2","2.19","2.11","1.91","1.79","1.78","1.74","1.65","1.64","1.57","1.49","1.44","1.4","1.33","1.3"],["2.38e-06","4.42e-05","5.89e-05","9.47e-05","0.00112","0.00126","0.00163","0.00447","0.00624","0.00652","0.00776","0.0122","0.0164","0.0167","0.018","0.0226","0.0228","0.0269","0.0325","0.0363","0.0402","0.0468","0.0497"],["0.000525","0.00434","0.00434","0.00523","0.0465","0.0465","0.0513","0.123","0.144","0.144","0.156","0.225","0.263","0.263","0.266","0.297","0.297","0.331","0.378","0.401","0.423","0.433","0.433"],["Gli1/Ihh/Hhip/Ptch2/Evc2/Evc/Lrp2/Kif7","Gli1/Wnt6/Hhip/Wnt11/Lef1/Ptch2/Kif7","Col4a6/Lamc3/Itga4/Npnt/Cd44/Col9a2/Frem1/Lama4","Epha8/Efnb3/Sema5a/Dcc/Lrrc4c/Sema5b/Plxnb1/Gnai1/Sema7a/Cxcl12/Gdf7","Ihh/Hgf/Wnt6/Wnt11/Hoxd10/Igf1/Kdr/Gpc3/Cd44/Mmp2","Col11a1/Col26a1/Col14a1/Col4a6/Pga5/Col9a2/Slc8a3","Hgf/Ryr1/Adra1a/Kdr/Ednrb/Nos3/P2rx2/Ptgfr/Cacna1e/Grin2b/Slc8a3","Kcnj3/Maob/Gria4/Drd4/Scn1a/Gnai1/Grin2b","Lef1/Sgcd/Itga4/Cdh2/Slc8a3","Grid1/Grik1/Vipr2/Gria4/Adra1a/Drd4/Ednrb/Calca/Edn3/P2rx2/Ptgfr/Grin2b/Gabra4","Kcnj3/Slc1a3/Grik1/Gria4/Gnai1/Grin2b","Hgf/Col4a6/Igf1/Lamc3/Itga4/Kdr/Col9a2/Lama4","Hgf/Angpt1/Igf1/Kdr/Gnai1/Evl/Grin2b/Rasgrp3","Kcnj3/Gria4/Ryr1/Gnai1/Grin2b","Wnt6/Wnt11/Lef1/Ednrb/Gnai1","Gria4/Grin2b/Gabra4","Hgf/Angpt1/Col4a6/Igf1/Lamc3/Areg/Itga4/Kdr/Nos3/Col9a2/Lama4","Car4/Car8","Wnt6/Wnt11/Lef1/Areg/Nkd1/Gdf7","Maob/Gnai1/Grin2b","Apob/Lrp2/Soat1","Col4a6/Ednrb/Nos3/Gnai1/Mmp2","Sgcd/Igf1/Itga4/Slc8a3"],["8","7","8","11","10","7","11","7","5","13","6","8","8","5","5","3","11","2","6","3","3","5","4"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>pvalue<\/th>\n      <th>logPval<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable(caption = &quot;Significantly enriched KEGG pathways&quot;) %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="dot-plot-2" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>kegg_dot[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-10-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-10-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-10-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/figure/kegg.Rmd/unnamed-chunk-10-1.png" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-10-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-10-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="upset-plot-2" class="section level3">
<h3>Upset Plot</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-11-1.png" width="864" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-11-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-11-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-11-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-11-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="pathway-specific-heatmaps" class="section level1 tabset">
<h1 class="tabset">Pathway specific heatmaps</h1>
<div id="fc1.05-1" class="section level2 tabset">
<h2 class="tabset">FC=1.05</h2>
<pre class="r"><code>p=1</code></pre>
<div id="leukocyte-transendothelial-migration"
class="section level3 tabset">
<h3 class="tabset">Leukocyte transendothelial migration</h3>
<pre class="r"><code>q=1</code></pre>
<div id="heatmap" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code># create df with normalised read counts with an additional entrezid column for binding
logCPM &lt;- cpm(dge, prior.count = 3, log = TRUE)
logCPM &lt;- logCPM[,1:7]
logCPM &lt;- cbind(logCPM, dge$genes$entrezid)
rownames(logCPM) &lt;- dge$genes$gene_name
colnames(logCPM) &lt;- c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;, &quot;entrezid&quot;)

### full pathway method
# complete_pathway &lt;- kegg_pathway[[1]]$GENE %&gt;% as.data.frame()
# complete_pathway &lt;- focal_adhesion[seq(1, nrow(focal_adhesion), 2),]
# match_complete_pathway &lt;- logCPM[,&quot;entrezid&quot;] %in% complete_pathway


# df for heatmap annotation of sample group
anno &lt;- as.factor(dge$samples$group) %&gt;% as.data.frame() 
anno &lt;- anno[1:7,] %&gt;% as.data.frame()
colnames(anno) &lt;- &quot;Sample Groups&quot;
anno$`Sample Groups` &lt;- gsub(&quot;CONT&quot;, &quot;Control&quot;, anno$`Sample Groups`)
anno$`Sample Groups` &lt;- gsub(&quot;INT&quot;, &quot;Intact&quot;, anno$`Sample Groups`)
rownames(anno) &lt;- colnames(logCPM[, 1:7])

# setting colour of sample group annotation

# original sample colours
# anno_colours &lt;- c(&quot;#66C2A5&quot;, &quot;#FC8D62&quot;)

# new sample colours
anno_colours &lt;- c(&quot;#f8766d&quot;, &quot;#a3a500&quot;)

names(anno_colours) &lt;- c(&quot;Control&quot;, &quot;Intact&quot;)</code></pre>
<pre class="r"><code>matrix &lt;- list()
display_matrix &lt;- list()
kegg_heat=list()

my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,248,239, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)

for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  
  for (j in 1:length(kegg_id)) {
    y &lt;- kegg_pathway[[j]]$PATHWAY_MAP
    
    partial &lt;- enrichKEGG_all[[x]][, c(&quot;ID&quot;, &quot;geneID&quot;)]
    partial &lt;- partial[kegg_id[j], &quot;geneID&quot;] %&gt;% as.data.frame()
    partial &lt;- separate_rows(partial, ., sep = &quot;/&quot;)
    colnames(partial) &lt;- &quot;entrezid&quot;

    # heatmap matrix
    match &lt;- rownames(logCPM) %in% partial$entrezid
    matrix[[x]][[y]] &lt;- logCPM[match, c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;)] %&gt;% as.data.frame()
    
    # changing the colname to  numeric for some reason, cant remember
    
    matrix[[x]][[y]][, c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;)] &lt;- as.numeric(as.character(unlist(matrix[[x]][[y]][, c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;)])))

    # display matrix
    match2 &lt;- lmTreat_sig[[x]][, &quot;gene_name&quot;] %in% partial$entrezid
    display_matrix[[x]][[y]] &lt;- lmTreat_sig[[x]][match2, c(&quot;gene_name&quot;, &quot;logFC&quot;, &quot;P.Value&quot;, &quot;adj.P.Val&quot;, &quot;description&quot;)] %&gt;%
      as.data.frame()
    colnames(display_matrix[[x]][[y]]) &lt;- c(&quot;Gene Name&quot;, &quot;logFC&quot;, &quot;P Value&quot;, &quot;Adjusted P Value&quot;, &quot;Description&quot;)
    
    ## Heatmap
    kegg_heat[[x]][[y]] &lt;- pheatmap(
      mat = matrix[[x]][[y]],
      ### Publish
      show_colnames = T,
      main = paste0(y, &quot;\n&quot;),
      legend = F,
      annotation_legend = F,
      fontsize = 10,
      fontsize_col = 11,
      fontsize_number = 7,
      fontsize_row = 10,
      treeheight_row = 25,
      treeheight_col = 10,
      cluster_cols = T,
      clustering_distance_rows = &quot;euclidean&quot;,
      legend_breaks = c(seq(-3, 11, by = .5), 1.4),
      legend_labels = c(seq(-3, 11, by = .5), &quot;Z-Score&quot;),
      angle_col = 90,
      cutree_cols = 2,
      cutree_rows = 2,
      color = my_palette,
      scale = &quot;row&quot;,
      border_color = NA,
      annotation_col = anno,
      annotation_colors = list(&quot;Sample Groups&quot; = anno_colours),
      annotation_names_col = F,
      annotation = T,
      silent = T,
      
      labels_row = as.expression(lapply(rownames(matrix[[x]][[y]]), function(a) bquote(italic(.(a)))))
      
      ) %&gt;% as.ggplot()
    
    # save
    ggsave(filename = paste0(&quot;heat_&quot;, x, &quot;_&quot;, y, &quot;.svg&quot;), 
           plot = kegg_heat[[x]][[y]], 
           path = here::here(&quot;2_plots/kegg/&quot;),
           width = 166,
           height = 250,
           units = &quot;mm&quot;)}
}</code></pre>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-14-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-14-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-14-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-14-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="table-3" class="section level4">
<h4>Table</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-02880d8cbf9369760f4f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-02880d8cbf9369760f4f">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000027009","ENSMUSG00000057614","ENSMUSG00000031740","ENSMUSG00000037405","ENSMUSG00000020395","ENSMUSG00000031990","ENSMUSG00000061353","ENSMUSG00000050520","ENSMUSG00000031871","ENSMUSG00000032011","ENSMUSG00000031207","ENSMUSG00000030830","ENSMUSG00000001946","ENSMUSG00000020717","ENSMUSG00000041378","ENSMUSG00000022512","ENSMUSG00000056917","ENSMUSG00000053062"],["Itga4","Gnai1","Mmp2","Icam1","Itk","Jam3","Cxcl12","Cldn8","Cdh5","Thy1","Msn","Itgal","Esam","Pecam1","Cldn5","Cldn1","Sipa1","Jam2"],["2.07","2.41","2.27","1.99","2.08","1.93","2.57","1.95","2.5","1.78","1.38","1.82","1.94","1.73","1.92","1.84","1.25","3.16"],["1.55e-05","0.000101","0.000133","0.000159","0.00019","0.000193","0.000201","0.00027","4e-04","0.000413","0.000449","0.000648","0.000649","9e-04","0.00188","0.00378","0.00423","0.00463"],["0.00251","0.0055","0.00629","0.00687","0.00747","0.00752","0.00769","0.00934","0.0117","0.0118","0.0122","0.015","0.015","0.0179","0.0281","0.0436","0.0466","0.0493"],["integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","guanine nucleotide binding protein (G protein), alpha inhibiting 1 [Source:MGI Symbol;Acc:MGI:95771]","matrix metallopeptidase 2 [Source:MGI Symbol;Acc:MGI:97009]","intercellular adhesion molecule 1 [Source:MGI Symbol;Acc:MGI:96392]","IL2 inducible T cell kinase [Source:MGI Symbol;Acc:MGI:96621]","junction adhesion molecule 3 [Source:MGI Symbol;Acc:MGI:1933825]","chemokine (C-X-C motif) ligand 12 [Source:MGI Symbol;Acc:MGI:103556]","claudin 8 [Source:MGI Symbol;Acc:MGI:1859286]","cadherin 5 [Source:MGI Symbol;Acc:MGI:105057]","thymus cell antigen 1, theta [Source:MGI Symbol;Acc:MGI:98747]","moesin [Source:MGI Symbol;Acc:MGI:97167]","integrin alpha L [Source:MGI Symbol;Acc:MGI:96606]","endothelial cell-specific adhesion molecule [Source:MGI Symbol;Acc:MGI:1916774]","platelet/endothelial cell adhesion molecule 1 [Source:MGI Symbol;Acc:MGI:97537]","claudin 5 [Source:MGI Symbol;Acc:MGI:1276112]","claudin 1 [Source:MGI Symbol;Acc:MGI:1276109]","signal-induced proliferation associated gene 1 [Source:MGI Symbol;Acc:MGI:107576]","junction adhesion molecule 2 [Source:MGI Symbol;Acc:MGI:1933820]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview" class="section level4">
<h4>Pathview</h4>
<pre class="r"><code># adjusting the kegg id to suit the parameters of the pathview funtion
adj.keggID &lt;- gsub(&quot;mmu&quot;, &quot;&quot;, kegg_id)

for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  
  # extract the logFC from the DE gene list
  pathview_table &lt;- dplyr::select(.data = lmTreat_sig[[x]], c(&quot;logFC&quot;)) %&gt;% as.matrix()

  # run pathview with Ensembl ID instead of entrezID
  pathview &lt;- pathview(
    gene.data = pathview_table[, 1],
    gene.idtype = &quot;ENSEMBL&quot;,
    pathway.id = adj.keggID,
    species = &quot;mmu&quot;,
    out.suffix = &quot;pv&quot;,
    kegg.dir = here::here(&quot;2_plots/kegg/&quot;),
    kegg.native = T
  )

  # move the result file to the plot directory
  file.rename(
    from = paste0(&quot;mmu&quot;, adj.keggID, &quot;.pv.png&quot;),
    to = here::here(paste0(&quot;docs/figure/kegg.Rmd/pv_&quot;, x, &quot;_&quot;, kegg_id, &quot;.png&quot;))
  )
}</code></pre>
<pre><code>[1] &quot;Note: 163 of 1629 unique input IDs unmapped.&quot;</code></pre>
<pre><code>[1] &quot;Note: 148 of 1447 unique input IDs unmapped.&quot;</code></pre>
<pre><code>[1] &quot;Note: 60 of 388 unique input IDs unmapped.&quot;</code></pre>
<div class="float">
<img src="figure/kegg.Rmd/pv_1.5_mmu04670.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="hematopoietic-cell-lineage" class="section level3 tabset">
<h3 class="tabset">Hematopoietic cell lineage</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-1" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-1">
Past versions of unnamed-chunk-16-1.png
</button>
</p>
<div id="fig-unnamed-chunk-16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-16-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-16-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-16-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-16-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-16-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-3ddf61f37a3b681f1823" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3ddf61f37a3b681f1823">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000027009","ENSMUSG00000005087","ENSMUSG00000073889","ENSMUSG00000002033","ENSMUSG00000042284","ENSMUSG00000042817","ENSMUSG00000028076","ENSMUSG00000005672","ENSMUSG00000027820","ENSMUSG00000073421","ENSMUSG00000029084","ENSMUSG00000041538","ENSMUSG00000019966","ENSMUSG00000060586","ENSMUSG00000036594","ENSMUSG00000003882","ENSMUSG00000023274"],["Itga4","Cd44","Il11ra1","Cd3g","Itga1","Flt3","Cd1d1","Kit","Mme","H2-Ab1","Cd38","H2-Ob","Kitl","H2-Eb1","H2-Aa","Il7r","Cd4"],["2.07","2.27","1.54","3.65","1.94","2.08","2.34","1.52","3.74","1.74","1.71","1.72","1.31","1.69","1.67","2.26","2.41"],["1.55e-05","7.24e-05","0.000108","0.000212","0.000311","0.00041","0.000506","0.000531","0.000535","0.00065","0.000756","0.000831","0.00107","0.00138","0.00216","0.00405","0.00408"],["0.00251","0.00467","0.00566","0.00797","0.0102","0.0118","0.0131","0.0134","0.0135","0.015","0.0163","0.0171","0.0201","0.0232","0.0309","0.0455","0.0457"],["integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","CD44 antigen [Source:MGI Symbol;Acc:MGI:88338]","interleukin 11 receptor, alpha chain 1 [Source:MGI Symbol;Acc:MGI:107426]","CD3 antigen, gamma polypeptide [Source:MGI Symbol;Acc:MGI:88333]","integrin alpha 1 [Source:MGI Symbol;Acc:MGI:96599]","FMS-like tyrosine kinase 3 [Source:MGI Symbol;Acc:MGI:95559]","CD1d1 antigen [Source:MGI Symbol;Acc:MGI:107674]","KIT proto-oncogene receptor tyrosine kinase [Source:MGI Symbol;Acc:MGI:96677]","membrane metallo endopeptidase [Source:MGI Symbol;Acc:MGI:97004]","histocompatibility 2, class II antigen A, beta 1 [Source:MGI Symbol;Acc:MGI:103070]","CD38 antigen [Source:MGI Symbol;Acc:MGI:107474]","histocompatibility 2, O region beta locus [Source:MGI Symbol;Acc:MGI:95925]","kit ligand [Source:MGI Symbol;Acc:MGI:96974]","histocompatibility 2, class II antigen E beta [Source:MGI Symbol;Acc:MGI:95901]","histocompatibility 2, class II antigen A, alpha [Source:MGI Symbol;Acc:MGI:95895]","interleukin 7 receptor [Source:MGI Symbol;Acc:MGI:96562]","CD4 antigen [Source:MGI Symbol;Acc:MGI:88335]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-1" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_1.5_mmu04640.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="vascular-smooth-muscle-contraction"
class="section level3 tabset">
<h3 class="tabset">Vascular smooth muscle contraction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-2" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-19-1">
Past versions of unnamed-chunk-19-1.png
</button>
</p>
<div id="fig-unnamed-chunk-19-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-19-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-19-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-19-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-19-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-1" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-e7caa58884ced2907e90" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-e7caa58884ced2907e90">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000051331","ENSMUSG00000022683","ENSMUSG00000045875","ENSMUSG00000041624","ENSMUSG00000020654","ENSMUSG00000027524","ENSMUSG00000051177","ENSMUSG00000030669","ENSMUSG00000028005","ENSMUSG00000033910","ENSMUSG00000022220","ENSMUSG00000031659","ENSMUSG00000049115","ENSMUSG00000063142","ENSMUSG00000031616","ENSMUSG00000040061","ENSMUSG00000059588"],["Cacna1c","Pla2g10","Adra1a","Gucy1a2","Adcy3","Edn3","Plcb1","Calca","Gucy1b1","Gucy1a1","Adcy4","Adcy7","Agtr1a","Kcnma1","Ednra","Plcb2","Calcrl"],["1.12","5.77","4.13","2.2","1.65","3.85","1.96","5.76","1.86","1.99","1.87","1.31","1.76","2.43","1.78","1.5","1.49"],["9.35e-06","1.8e-05","5.6e-05","0.000165","0.00019","3e-04","0.00032","0.000366","0.000413","0.000517","0.000549","0.000557","0.000765","0.000843","0.00087","0.00154","0.00189"],["0.00199","0.00256","0.00427","0.00701","0.00747","0.01","0.0104","0.0111","0.0118","0.0132","0.0136","0.0137","0.0165","0.0172","0.0175","0.0249","0.0281"],["calcium channel, voltage-dependent, L type, alpha 1C subunit [Source:MGI Symbol;Acc:MGI:103013]","phospholipase A2, group X [Source:MGI Symbol;Acc:MGI:1347522]","adrenergic receptor, alpha 1a [Source:MGI Symbol;Acc:MGI:104773]","guanylate cyclase 1, soluble, alpha 2 [Source:MGI Symbol;Acc:MGI:2660877]","adenylate cyclase 3 [Source:MGI Symbol;Acc:MGI:99675]","endothelin 3 [Source:MGI Symbol;Acc:MGI:95285]","phospholipase C, beta 1 [Source:MGI Symbol;Acc:MGI:97613]","calcitonin/calcitonin-related polypeptide, alpha [Source:MGI Symbol;Acc:MGI:2151253]","guanylate cyclase 1, soluble, beta 1 [Source:MGI Symbol;Acc:MGI:1860604]","guanylate cyclase 1, soluble, alpha 1 [Source:MGI Symbol;Acc:MGI:1926562]","adenylate cyclase 4 [Source:MGI Symbol;Acc:MGI:99674]","adenylate cyclase 7 [Source:MGI Symbol;Acc:MGI:102891]","angiotensin II receptor, type 1a [Source:MGI Symbol;Acc:MGI:87964]","potassium large conductance calcium-activated channel, subfamily M, alpha member 1 [Source:MGI Symbol;Acc:MGI:99923]","endothelin receptor type A [Source:MGI Symbol;Acc:MGI:105923]","phospholipase C, beta 2 [Source:MGI Symbol;Acc:MGI:107465]","calcitonin receptor-like [Source:MGI Symbol;Acc:MGI:1926944]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-2" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_1.5_mmu04270.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="pi3k-akt-signaling-pathway" class="section level3 tabset">
<h3 class="tabset">PI3K-Akt signaling pathway</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-3" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-22-1">
Past versions of unnamed-chunk-22-1.png
</button>
</p>
<div id="fig-unnamed-chunk-22-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-22-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-22-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-22-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-22-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-2" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-a43969c2d20cde42b049" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-a43969c2d20cde42b049">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000028864","ENSMUSG00000022309","ENSMUSG00000027009","ENSMUSG00000062960","ENSMUSG00000020053","ENSMUSG00000031273","ENSMUSG00000028047","ENSMUSG00000028978","ENSMUSG00000026840","ENSMUSG00000019846","ENSMUSG00000000184","ENSMUSG00000001281","ENSMUSG00000026193","ENSMUSG00000029378","ENSMUSG00000049929","ENSMUSG00000006386","ENSMUSG00000038811","ENSMUSG00000046207","ENSMUSG00000042284","ENSMUSG00000028626","ENSMUSG00000042817","ENSMUSG00000005672","ENSMUSG00000032766","ENSMUSG00000019899","ENSMUSG00000031520","ENSMUSG00000040003","ENSMUSG00000048583","ENSMUSG00000043004","ENSMUSG00000025348","ENSMUSG00000019966","ENSMUSG00000019699","ENSMUSG00000032192","ENSMUSG00000048240","ENSMUSG00000027669","ENSMUSG00000024620","ENSMUSG00000031304","ENSMUSG00000026768","ENSMUSG00000020758","ENSMUSG00000048376","ENSMUSG00000003882","ENSMUSG00000022346","ENSMUSG00000031274","ENSMUSG00000055980"],["Hgf","Angpt1","Itga4","Kdr","Igf1","Col4a6","Thbs3","Nos3","Lamc3","Lama4","Ccnd2","Itgb7","Fn1","Areg","Lpar4","Tek","Gngt2","Pik3r6","Itga1","Col9a2","Flt3","Kit","Gng11","Lama2","Vegfc","Magi2","Igf2","Gng2","Itga7","Kitl","Akt3","Gnb5","Gng7","Gnb4","Pdgfrb","Il2rg","Itga8","Itgb4","F2r","Il7r","Myc","Col4a5","Irs1"],["4.13","3.39","2.07","2.05","2.78","3.47","1.67","2.15","3.8","1.94","1.71","1.81","1.97","5.25","2.15","2.34","1.82","1.69","1.94","4.13","2.08","1.52","2.02","1.83","2.12","1.52","2.43","1.24","1.58","1.31","1.2","1.03","2.31","1.16","0.957","1.88","1.59","-1.5","2.19","2.26","1.81","1.7","0.95"],["4.39e-06","1.02e-05","1.55e-05","2.26e-05","3.09e-05","4.09e-05","4.61e-05","5.81e-05","7.83e-05","8.23e-05","0.000108","0.000109","0.000114","0.00013","0.000166","0.000195","0.000221","0.00027","0.000311","0.00036","0.00041","0.000531","0.000574","0.000577","0.000751","0.000798","0.000933","0.00094","0.000987","0.00107","0.00209","0.00219","0.00227","0.00273","0.00298","0.00336","0.00337","0.00354","0.00405","0.00405","0.00431","0.00471","0.00472"],["0.0015","0.00211","0.00251","0.0028","0.00326","0.00373","0.00399","0.00435","0.00483","0.00498","0.00566","0.00566","0.00576","0.00622","0.00701","0.00753","0.00818","0.00934","0.0102","0.0111","0.0118","0.0134","0.0139","0.014","0.0163","0.0167","0.0183","0.0184","0.0189","0.0201","0.0302","0.0311","0.0316","0.0356","0.0376","0.0405","0.0407","0.042","0.0455","0.0455","0.0471","0.0498","0.0499"],["hepatocyte growth factor [Source:MGI Symbol;Acc:MGI:96079]","angiopoietin 1 [Source:MGI Symbol;Acc:MGI:108448]","integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","kinase insert domain protein receptor [Source:MGI Symbol;Acc:MGI:96683]","insulin-like growth factor 1 [Source:MGI Symbol;Acc:MGI:96432]","collagen, type IV, alpha 6 [Source:MGI Symbol;Acc:MGI:2152695]","thrombospondin 3 [Source:MGI Symbol;Acc:MGI:98739]","nitric oxide synthase 3, endothelial cell [Source:MGI Symbol;Acc:MGI:97362]","laminin gamma 3 [Source:MGI Symbol;Acc:MGI:1344394]","laminin, alpha 4 [Source:MGI Symbol;Acc:MGI:109321]","cyclin D2 [Source:MGI Symbol;Acc:MGI:88314]","integrin beta 7 [Source:MGI Symbol;Acc:MGI:96616]","fibronectin 1 [Source:MGI Symbol;Acc:MGI:95566]","amphiregulin [Source:MGI Symbol;Acc:MGI:88068]","lysophosphatidic acid receptor 4 [Source:MGI Symbol;Acc:MGI:1925384]","TEK receptor tyrosine kinase [Source:MGI Symbol;Acc:MGI:98664]","guanine nucleotide binding protein (G protein), gamma transducing activity polypeptide 2 [Source:MGI Symbol;Acc:MGI:893584]","phosphoinositide-3-kinase regulatory subunit 5 [Source:MGI Symbol;Acc:MGI:2144613]","integrin alpha 1 [Source:MGI Symbol;Acc:MGI:96599]","collagen, type IX, alpha 2 [Source:MGI Symbol;Acc:MGI:88466]","FMS-like tyrosine kinase 3 [Source:MGI Symbol;Acc:MGI:95559]","KIT proto-oncogene receptor tyrosine kinase [Source:MGI Symbol;Acc:MGI:96677]","guanine nucleotide binding protein (G protein), gamma 11 [Source:MGI Symbol;Acc:MGI:1913316]","laminin, alpha 2 [Source:MGI Symbol;Acc:MGI:99912]","vascular endothelial growth factor C [Source:MGI Symbol;Acc:MGI:109124]","membrane associated guanylate kinase, WW and PDZ domain containing 2 [Source:MGI Symbol;Acc:MGI:1354953]","insulin-like growth factor 2 [Source:MGI Symbol;Acc:MGI:96434]","guanine nucleotide binding protein (G protein), gamma 2 [Source:MGI Symbol;Acc:MGI:102705]","integrin alpha 7 [Source:MGI Symbol;Acc:MGI:102700]","kit ligand [Source:MGI Symbol;Acc:MGI:96974]","thymoma viral proto-oncogene 3 [Source:MGI Symbol;Acc:MGI:1345147]","guanine nucleotide binding protein (G protein), beta 5 [Source:MGI Symbol;Acc:MGI:101848]","guanine nucleotide binding protein (G protein), gamma 7 [Source:MGI Symbol;Acc:MGI:95787]","guanine nucleotide binding protein (G protein), beta 4 [Source:MGI Symbol;Acc:MGI:104581]","platelet derived growth factor receptor, beta polypeptide [Source:MGI Symbol;Acc:MGI:97531]","interleukin 2 receptor, gamma chain [Source:MGI Symbol;Acc:MGI:96551]","integrin alpha 8 [Source:MGI Symbol;Acc:MGI:109442]","integrin beta 4 [Source:MGI Symbol;Acc:MGI:96613]","coagulation factor II (thrombin) receptor [Source:MGI Symbol;Acc:MGI:101802]","interleukin 7 receptor [Source:MGI Symbol;Acc:MGI:96562]","myelocytomatosis oncogene [Source:MGI Symbol;Acc:MGI:97250]","collagen, type IV, alpha 5 [Source:MGI Symbol;Acc:MGI:88456]","insulin receptor substrate 1 [Source:MGI Symbol;Acc:MGI:99454]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-3" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_1.5_mmu04151.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="focal-adhesion" class="section level3 tabset">
<h3 class="tabset">Focal adhesion</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-4" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-25-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-25-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-25-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-25-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-3" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-376a78f75a7321aba26c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-376a78f75a7321aba26c">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000028864","ENSMUSG00000027009","ENSMUSG00000062960","ENSMUSG00000020053","ENSMUSG00000031273","ENSMUSG00000028047","ENSMUSG00000026840","ENSMUSG00000019846","ENSMUSG00000000184","ENSMUSG00000001281","ENSMUSG00000026193","ENSMUSG00000021448","ENSMUSG00000024867","ENSMUSG00000042284","ENSMUSG00000028626","ENSMUSG00000019899","ENSMUSG00000031520","ENSMUSG00000025348","ENSMUSG00000022439","ENSMUSG00000022438","ENSMUSG00000068699","ENSMUSG00000052698","ENSMUSG00000019843","ENSMUSG00000019699","ENSMUSG00000024620","ENSMUSG00000007655","ENSMUSG00000026768","ENSMUSG00000020758","ENSMUSG00000031274"],["Hgf","Itga4","Kdr","Igf1","Col4a6","Thbs3","Lamc3","Lama4","Ccnd2","Itgb7","Fn1","Shc3","Pip5k1b","Itga1","Col9a2","Lama2","Vegfc","Itga7","Parvg","Parvb","Flnc","Tln2","Fyn","Akt3","Pdgfrb","Cav1","Itga8","Itgb4","Col4a5"],["4.13","2.07","2.05","2.78","3.47","1.67","3.8","1.94","1.71","1.81","1.97","2.15","1.5","1.94","4.13","1.83","2.12","1.58","1.77","1.64","1.18","1.24","1.18","1.2","0.957","1.6","1.59","-1.5","1.7"],["4.39e-06","1.55e-05","2.26e-05","3.09e-05","4.09e-05","4.61e-05","7.83e-05","8.23e-05","0.000108","0.000109","0.000114","0.000137","0.000149","0.000311","0.00036","0.000577","0.000751","0.000987","0.00101","0.00124","0.0014","0.00148","0.00174","0.00209","0.00298","0.00318","0.00337","0.00354","0.00471"],["0.0015","0.00251","0.0028","0.00326","0.00373","0.00399","0.00483","0.00498","0.00566","0.00566","0.00576","0.00632","0.00668","0.0102","0.0111","0.014","0.0163","0.0189","0.0192","0.0215","0.0234","0.0244","0.0267","0.0302","0.0376","0.0393","0.0407","0.042","0.0498"],["hepatocyte growth factor [Source:MGI Symbol;Acc:MGI:96079]","integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","kinase insert domain protein receptor [Source:MGI Symbol;Acc:MGI:96683]","insulin-like growth factor 1 [Source:MGI Symbol;Acc:MGI:96432]","collagen, type IV, alpha 6 [Source:MGI Symbol;Acc:MGI:2152695]","thrombospondin 3 [Source:MGI Symbol;Acc:MGI:98739]","laminin gamma 3 [Source:MGI Symbol;Acc:MGI:1344394]","laminin, alpha 4 [Source:MGI Symbol;Acc:MGI:109321]","cyclin D2 [Source:MGI Symbol;Acc:MGI:88314]","integrin beta 7 [Source:MGI Symbol;Acc:MGI:96616]","fibronectin 1 [Source:MGI Symbol;Acc:MGI:95566]","src homology 2 domain-containing transforming protein C3 [Source:MGI Symbol;Acc:MGI:106179]","phosphatidylinositol-4-phosphate 5-kinase, type 1 beta [Source:MGI Symbol;Acc:MGI:107930]","integrin alpha 1 [Source:MGI Symbol;Acc:MGI:96599]","collagen, type IX, alpha 2 [Source:MGI Symbol;Acc:MGI:88466]","laminin, alpha 2 [Source:MGI Symbol;Acc:MGI:99912]","vascular endothelial growth factor C [Source:MGI Symbol;Acc:MGI:109124]","integrin alpha 7 [Source:MGI Symbol;Acc:MGI:102700]","parvin, gamma [Source:MGI Symbol;Acc:MGI:2158329]","parvin, beta [Source:MGI Symbol;Acc:MGI:2153063]","filamin C, gamma [Source:MGI Symbol;Acc:MGI:95557]","talin 2 [Source:MGI Symbol;Acc:MGI:1917799]","Fyn proto-oncogene [Source:MGI Symbol;Acc:MGI:95602]","thymoma viral proto-oncogene 3 [Source:MGI Symbol;Acc:MGI:1345147]","platelet derived growth factor receptor, beta polypeptide [Source:MGI Symbol;Acc:MGI:97531]","caveolin 1, caveolae protein [Source:MGI Symbol;Acc:MGI:102709]","integrin alpha 8 [Source:MGI Symbol;Acc:MGI:109442]","integrin beta 4 [Source:MGI Symbol;Acc:MGI:96613]","collagen, type IV, alpha 5 [Source:MGI Symbol;Acc:MGI:88456]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-4" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_1.5_mmu04510.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="cytokine-cytokine-receptor-interaction"
class="section level3 tabset">
<h3 class="tabset">Cytokine-cytokine receptor interaction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-5" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-28-1">
Past versions of unnamed-chunk-28-1.png
</button>
</p>
<div id="fig-unnamed-chunk-28-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-28-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-28-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-28-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-28-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-28-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-4" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-d5b236e9fe165e99b686" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d5b236e9fe165e99b686">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000029375","ENSMUSG00000034457","ENSMUSG00000073889","ENSMUSG00000050232","ENSMUSG00000061353","ENSMUSG00000025352","ENSMUSG00000037660","ENSMUSG00000031497","ENSMUSG00000021943","ENSMUSG00000005465","ENSMUSG00000063727","ENSMUSG00000026573","ENSMUSG00000060509","ENSMUSG00000071713","ENSMUSG00000000791","ENSMUSG00000031304","ENSMUSG00000003227","ENSMUSG00000008999","ENSMUSG00000003882","ENSMUSG00000023274"],["Cxcl15","Eda2r","Il11ra1","Cxcr3","Cxcl12","Gdf11","Gdf7","Tnfsf13b","Gdf10","Il27ra","Tnfrsf11b","Xcl1","Xcr1","Csf2rb","Il12rb1","Il2rg","Edar","Bmp7","Il7r","Cd4"],["6.92","3.52","1.54","2.28","2.57","2.25","3.41","2.22","2.74","1.94","2.13","2.67","2","2.03","2.13","1.88","2.57","1.59","2.26","2.41"],["1.72e-08","3.87e-05","0.000108","0.000195","0.000201","0.000243","4e-04","0.000434","0.000521","0.00067","0.00074","0.00111","0.00176","0.00179","0.00312","0.00336","0.00385","0.00405","0.00405","0.00408"],["0.000148","0.00366","0.00566","0.00753","0.00769","0.00876","0.0117","0.012","0.0133","0.0153","0.0162","0.0204","0.0269","0.0271","0.0387","0.0405","0.044","0.0455","0.0455","0.0457"],["chemokine (C-X-C motif) ligand 15 [Source:MGI Symbol;Acc:MGI:1339941]","ectodysplasin A2 receptor [Source:MGI Symbol;Acc:MGI:2442860]","interleukin 11 receptor, alpha chain 1 [Source:MGI Symbol;Acc:MGI:107426]","chemokine (C-X-C motif) receptor 3 [Source:MGI Symbol;Acc:MGI:1277207]","chemokine (C-X-C motif) ligand 12 [Source:MGI Symbol;Acc:MGI:103556]","growth differentiation factor 11 [Source:MGI Symbol;Acc:MGI:1338027]","growth differentiation factor 7 [Source:MGI Symbol;Acc:MGI:95690]","tumor necrosis factor (ligand) superfamily, member 13b [Source:MGI Symbol;Acc:MGI:1344376]","growth differentiation factor 10 [Source:MGI Symbol;Acc:MGI:95684]","interleukin 27 receptor, alpha [Source:MGI Symbol;Acc:MGI:1355318]","tumor necrosis factor receptor superfamily, member 11b (osteoprotegerin) [Source:MGI Symbol;Acc:MGI:109587]","chemokine (C motif) ligand 1 [Source:MGI Symbol;Acc:MGI:104593]","chemokine (C motif) receptor 1 [Source:MGI Symbol;Acc:MGI:1346338]","colony stimulating factor 2 receptor, beta, low-affinity (granulocyte-macrophage) [Source:MGI Symbol;Acc:MGI:1339759]","interleukin 12 receptor, beta 1 [Source:MGI Symbol;Acc:MGI:104579]","interleukin 2 receptor, gamma chain [Source:MGI Symbol;Acc:MGI:96551]","ectodysplasin-A receptor [Source:MGI Symbol;Acc:MGI:1343498]","bone morphogenetic protein 7 [Source:MGI Symbol;Acc:MGI:103302]","interleukin 7 receptor [Source:MGI Symbol;Acc:MGI:96562]","CD4 antigen [Source:MGI Symbol;Acc:MGI:88335]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-5" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_1.5_mmu04060.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
</div>
<div id="fc1.1-1" class="section level2 tabset">
<h2 class="tabset">FC=1.1</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="leukocyte-transendothelial-migration-1"
class="section level3 tabset">
<h3 class="tabset">Leukocyte transendothelial migration</h3>
<pre class="r"><code>q=1</code></pre>
<div id="heatmap-6" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-32-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-32-1">
Past versions of unnamed-chunk-32-1.png
</button>
</p>
<div id="fig-unnamed-chunk-32-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-32-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-32-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-32-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-32-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-5" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8ee42dd6fb8b59fc7bd7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8ee42dd6fb8b59fc7bd7">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000027009","ENSMUSG00000057614","ENSMUSG00000031740","ENSMUSG00000037405","ENSMUSG00000020395","ENSMUSG00000061353","ENSMUSG00000031990","ENSMUSG00000050520","ENSMUSG00000031871","ENSMUSG00000032011","ENSMUSG00000031207","ENSMUSG00000001946","ENSMUSG00000030830","ENSMUSG00000020717","ENSMUSG00000041378","ENSMUSG00000022512"],["Itga4","Gnai1","Mmp2","Icam1","Itk","Cxcl12","Jam3","Cldn8","Cdh5","Thy1","Msn","Esam","Itgal","Pecam1","Cldn5","Cldn1"],["2.07","2.41","2.27","1.99","2.08","2.57","1.93","1.95","2.5","1.78","1.38","1.94","1.82","1.73","1.92","1.84"],["1.91e-05","0.000115","0.000153","0.000189","0.000221","0.000222","0.00023","0.000318","0.00044","0.000493","0.000594","0.000747","0.000759","0.00106","0.0021","0.00418"],["0.00285","0.0064","0.00716","0.00826","0.0088","0.00881","0.00892","0.0111","0.0132","0.014","0.0154","0.0177","0.0179","0.0212","0.0325","0.0498"],["integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","guanine nucleotide binding protein (G protein), alpha inhibiting 1 [Source:MGI Symbol;Acc:MGI:95771]","matrix metallopeptidase 2 [Source:MGI Symbol;Acc:MGI:97009]","intercellular adhesion molecule 1 [Source:MGI Symbol;Acc:MGI:96392]","IL2 inducible T cell kinase [Source:MGI Symbol;Acc:MGI:96621]","chemokine (C-X-C motif) ligand 12 [Source:MGI Symbol;Acc:MGI:103556]","junction adhesion molecule 3 [Source:MGI Symbol;Acc:MGI:1933825]","claudin 8 [Source:MGI Symbol;Acc:MGI:1859286]","cadherin 5 [Source:MGI Symbol;Acc:MGI:105057]","thymus cell antigen 1, theta [Source:MGI Symbol;Acc:MGI:98747]","moesin [Source:MGI Symbol;Acc:MGI:97167]","endothelial cell-specific adhesion molecule [Source:MGI Symbol;Acc:MGI:1916774]","integrin alpha L [Source:MGI Symbol;Acc:MGI:96606]","platelet/endothelial cell adhesion molecule 1 [Source:MGI Symbol;Acc:MGI:97537]","claudin 5 [Source:MGI Symbol;Acc:MGI:1276112]","claudin 1 [Source:MGI Symbol;Acc:MGI:1276109]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-6" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04670.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="hematopoietic-cell-lineage-1" class="section level3 tabset">
<h3 class="tabset">Hematopoietic cell lineage</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-7" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-35-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-35-1">
Past versions of unnamed-chunk-35-1.png
</button>
</p>
<div id="fig-unnamed-chunk-35-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-35-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-35-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-35-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-35-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-6" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-62046a43780a3042490e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-62046a43780a3042490e">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000027009","ENSMUSG00000005087","ENSMUSG00000073889","ENSMUSG00000002033","ENSMUSG00000042284","ENSMUSG00000042817","ENSMUSG00000027820","ENSMUSG00000028076","ENSMUSG00000005672","ENSMUSG00000073421","ENSMUSG00000029084","ENSMUSG00000041538","ENSMUSG00000019966","ENSMUSG00000060586","ENSMUSG00000036594"],["Itga4","Cd44","Il11ra1","Cd3g","Itga1","Flt3","Mme","Cd1d1","Kit","H2-Ab1","Cd38","H2-Ob","Kitl","H2-Eb1","H2-Aa"],["2.07","2.27","1.54","3.65","1.94","2.08","3.74","2.34","1.52","1.74","1.71","1.72","1.31","1.69","1.67"],["1.91e-05","8.38e-05","0.000143","0.000223","0.000365","0.000469","0.000558","0.000561","0.000667","0.000771","0.000897","0.000982","0.00138","0.00161","0.00249"],["0.00285","0.00539","0.00689","0.00881","0.0118","0.0137","0.0149","0.0149","0.0164","0.0179","0.0194","0.0204","0.0249","0.0274","0.036"],["integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","CD44 antigen [Source:MGI Symbol;Acc:MGI:88338]","interleukin 11 receptor, alpha chain 1 [Source:MGI Symbol;Acc:MGI:107426]","CD3 antigen, gamma polypeptide [Source:MGI Symbol;Acc:MGI:88333]","integrin alpha 1 [Source:MGI Symbol;Acc:MGI:96599]","FMS-like tyrosine kinase 3 [Source:MGI Symbol;Acc:MGI:95559]","membrane metallo endopeptidase [Source:MGI Symbol;Acc:MGI:97004]","CD1d1 antigen [Source:MGI Symbol;Acc:MGI:107674]","KIT proto-oncogene receptor tyrosine kinase [Source:MGI Symbol;Acc:MGI:96677]","histocompatibility 2, class II antigen A, beta 1 [Source:MGI Symbol;Acc:MGI:103070]","CD38 antigen [Source:MGI Symbol;Acc:MGI:107474]","histocompatibility 2, O region beta locus [Source:MGI Symbol;Acc:MGI:95925]","kit ligand [Source:MGI Symbol;Acc:MGI:96974]","histocompatibility 2, class II antigen E beta [Source:MGI Symbol;Acc:MGI:95901]","histocompatibility 2, class II antigen A, alpha [Source:MGI Symbol;Acc:MGI:95895]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-7" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04640.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="vascular-smooth-muscle-contraction-1"
class="section level3 tabset">
<h3 class="tabset">Vascular smooth muscle contraction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-8" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-38-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-38-1">
Past versions of unnamed-chunk-38-1.png
</button>
</p>
<div id="fig-unnamed-chunk-38-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-38-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-38-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-7" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0629593683dbe9836c1f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0629593683dbe9836c1f">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000051331","ENSMUSG00000022683","ENSMUSG00000045875","ENSMUSG00000041624","ENSMUSG00000020654","ENSMUSG00000027524","ENSMUSG00000030669","ENSMUSG00000051177","ENSMUSG00000028005","ENSMUSG00000033910","ENSMUSG00000022220","ENSMUSG00000031659","ENSMUSG00000049115","ENSMUSG00000063142","ENSMUSG00000031616","ENSMUSG00000040061","ENSMUSG00000059588"],["Cacna1c","Pla2g10","Adra1a","Gucy1a2","Adcy3","Edn3","Calca","Plcb1","Gucy1b1","Gucy1a1","Adcy4","Adcy7","Agtr1a","Kcnma1","Ednra","Plcb2","Calcrl"],["1.12","5.77","4.13","2.2","1.65","3.85","5.76","1.96","1.86","1.99","1.87","1.31","1.76","2.43","1.78","1.5","1.49"],["1.72e-05","1.85e-05","5.88e-05","0.00019","0.00024","0.000313","0.000373","0.000374","0.000486","0.000594","0.000641","0.000747","0.000899","0.000919","0.00102","0.00185","0.00226"],["0.00285","0.00285","0.0048","0.00828","0.0092","0.011","0.012","0.012","0.0139","0.0154","0.0161","0.0177","0.0194","0.0196","0.0206","0.03","0.0341"],["calcium channel, voltage-dependent, L type, alpha 1C subunit [Source:MGI Symbol;Acc:MGI:103013]","phospholipase A2, group X [Source:MGI Symbol;Acc:MGI:1347522]","adrenergic receptor, alpha 1a [Source:MGI Symbol;Acc:MGI:104773]","guanylate cyclase 1, soluble, alpha 2 [Source:MGI Symbol;Acc:MGI:2660877]","adenylate cyclase 3 [Source:MGI Symbol;Acc:MGI:99675]","endothelin 3 [Source:MGI Symbol;Acc:MGI:95285]","calcitonin/calcitonin-related polypeptide, alpha [Source:MGI Symbol;Acc:MGI:2151253]","phospholipase C, beta 1 [Source:MGI Symbol;Acc:MGI:97613]","guanylate cyclase 1, soluble, beta 1 [Source:MGI Symbol;Acc:MGI:1860604]","guanylate cyclase 1, soluble, alpha 1 [Source:MGI Symbol;Acc:MGI:1926562]","adenylate cyclase 4 [Source:MGI Symbol;Acc:MGI:99674]","adenylate cyclase 7 [Source:MGI Symbol;Acc:MGI:102891]","angiotensin II receptor, type 1a [Source:MGI Symbol;Acc:MGI:87964]","potassium large conductance calcium-activated channel, subfamily M, alpha member 1 [Source:MGI Symbol;Acc:MGI:99923]","endothelin receptor type A [Source:MGI Symbol;Acc:MGI:105923]","phospholipase C, beta 2 [Source:MGI Symbol;Acc:MGI:107465]","calcitonin receptor-like [Source:MGI Symbol;Acc:MGI:1926944]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-8" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04270.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="pi3k-akt-signaling-pathway-1" class="section level3 tabset">
<h3 class="tabset">PI3K-Akt signaling pathway</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-9" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-41-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-41-1">
Past versions of unnamed-chunk-41-1.png
</button>
</p>
<div id="fig-unnamed-chunk-41-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-41-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-41-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-41-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-41-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-8" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-38d1f7e9c0dbafc85536" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-38d1f7e9c0dbafc85536">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000028864","ENSMUSG00000022309","ENSMUSG00000027009","ENSMUSG00000062960","ENSMUSG00000020053","ENSMUSG00000031273","ENSMUSG00000028047","ENSMUSG00000028978","ENSMUSG00000026840","ENSMUSG00000019846","ENSMUSG00000029378","ENSMUSG00000001281","ENSMUSG00000000184","ENSMUSG00000026193","ENSMUSG00000049929","ENSMUSG00000006386","ENSMUSG00000038811","ENSMUSG00000046207","ENSMUSG00000042284","ENSMUSG00000028626","ENSMUSG00000042817","ENSMUSG00000032766","ENSMUSG00000005672","ENSMUSG00000019899","ENSMUSG00000031520","ENSMUSG00000040003","ENSMUSG00000048583","ENSMUSG00000025348","ENSMUSG00000043004","ENSMUSG00000019966","ENSMUSG00000048240","ENSMUSG00000019699","ENSMUSG00000032192","ENSMUSG00000027669","ENSMUSG00000031304","ENSMUSG00000026768","ENSMUSG00000020758"],["Hgf","Angpt1","Itga4","Kdr","Igf1","Col4a6","Thbs3","Nos3","Lamc3","Lama4","Areg","Itgb7","Ccnd2","Fn1","Lpar4","Tek","Gngt2","Pik3r6","Itga1","Col9a2","Flt3","Gng11","Kit","Lama2","Vegfc","Magi2","Igf2","Itga7","Gng2","Kitl","Gng7","Akt3","Gnb5","Gnb4","Il2rg","Itga8","Itgb4"],["4.13","3.39","2.07","2.05","2.78","3.47","1.67","2.15","3.8","1.94","5.25","1.81","1.71","1.97","2.15","2.34","1.82","1.69","1.94","4.13","2.08","2.02","1.52","1.83","2.12","1.52","2.43","1.58","1.24","1.31","2.31","1.2","1.03","1.16","1.88","1.59","-1.5"],["4.67e-06","1.11e-05","1.91e-05","2.77e-05","3.45e-05","4.39e-05","6.04e-05","6.87e-05","8.26e-05","9.99e-05","0.000134","0.000135","0.000137","0.000137","0.000192","0.000221","0.000267","0.000332","0.000365","0.000373","0.000469","0.000655","0.000667","0.000677","0.000842","0.000983","0.00102","0.00119","0.00126","0.00138","0.00245","0.00271","0.00306","0.00354","0.00371","0.00387","0.00412"],["0.00158","0.00231","0.00285","0.00338","0.00364","0.00413","0.0048","0.00515","0.00535","0.00591","0.00671","0.00671","0.00673","0.00673","0.0083","0.0088","0.00986","0.0114","0.0118","0.012","0.0137","0.0163","0.0164","0.0166","0.0189","0.0204","0.0206","0.0227","0.0235","0.0249","0.0356","0.038","0.0413","0.0449","0.0462","0.0475","0.0495"],["hepatocyte growth factor [Source:MGI Symbol;Acc:MGI:96079]","angiopoietin 1 [Source:MGI Symbol;Acc:MGI:108448]","integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","kinase insert domain protein receptor [Source:MGI Symbol;Acc:MGI:96683]","insulin-like growth factor 1 [Source:MGI Symbol;Acc:MGI:96432]","collagen, type IV, alpha 6 [Source:MGI Symbol;Acc:MGI:2152695]","thrombospondin 3 [Source:MGI Symbol;Acc:MGI:98739]","nitric oxide synthase 3, endothelial cell [Source:MGI Symbol;Acc:MGI:97362]","laminin gamma 3 [Source:MGI Symbol;Acc:MGI:1344394]","laminin, alpha 4 [Source:MGI Symbol;Acc:MGI:109321]","amphiregulin [Source:MGI Symbol;Acc:MGI:88068]","integrin beta 7 [Source:MGI Symbol;Acc:MGI:96616]","cyclin D2 [Source:MGI Symbol;Acc:MGI:88314]","fibronectin 1 [Source:MGI Symbol;Acc:MGI:95566]","lysophosphatidic acid receptor 4 [Source:MGI Symbol;Acc:MGI:1925384]","TEK receptor tyrosine kinase [Source:MGI Symbol;Acc:MGI:98664]","guanine nucleotide binding protein (G protein), gamma transducing activity polypeptide 2 [Source:MGI Symbol;Acc:MGI:893584]","phosphoinositide-3-kinase regulatory subunit 5 [Source:MGI Symbol;Acc:MGI:2144613]","integrin alpha 1 [Source:MGI Symbol;Acc:MGI:96599]","collagen, type IX, alpha 2 [Source:MGI Symbol;Acc:MGI:88466]","FMS-like tyrosine kinase 3 [Source:MGI Symbol;Acc:MGI:95559]","guanine nucleotide binding protein (G protein), gamma 11 [Source:MGI Symbol;Acc:MGI:1913316]","KIT proto-oncogene receptor tyrosine kinase [Source:MGI Symbol;Acc:MGI:96677]","laminin, alpha 2 [Source:MGI Symbol;Acc:MGI:99912]","vascular endothelial growth factor C [Source:MGI Symbol;Acc:MGI:109124]","membrane associated guanylate kinase, WW and PDZ domain containing 2 [Source:MGI Symbol;Acc:MGI:1354953]","insulin-like growth factor 2 [Source:MGI Symbol;Acc:MGI:96434]","integrin alpha 7 [Source:MGI Symbol;Acc:MGI:102700]","guanine nucleotide binding protein (G protein), gamma 2 [Source:MGI Symbol;Acc:MGI:102705]","kit ligand [Source:MGI Symbol;Acc:MGI:96974]","guanine nucleotide binding protein (G protein), gamma 7 [Source:MGI Symbol;Acc:MGI:95787]","thymoma viral proto-oncogene 3 [Source:MGI Symbol;Acc:MGI:1345147]","guanine nucleotide binding protein (G protein), beta 5 [Source:MGI Symbol;Acc:MGI:101848]","guanine nucleotide binding protein (G protein), beta 4 [Source:MGI Symbol;Acc:MGI:104581]","interleukin 2 receptor, gamma chain [Source:MGI Symbol;Acc:MGI:96551]","integrin alpha 8 [Source:MGI Symbol;Acc:MGI:109442]","integrin beta 4 [Source:MGI Symbol;Acc:MGI:96613]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-9" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04151.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="focal-adhesion-1" class="section level3 tabset">
<h3 class="tabset">Focal adhesion</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-10" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-44-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-44-1">
Past versions of unnamed-chunk-44-1.png
</button>
</p>
<div id="fig-unnamed-chunk-44-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-44-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-44-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-9" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-900211c4c9f5865ef8c9" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-900211c4c9f5865ef8c9">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000028864","ENSMUSG00000027009","ENSMUSG00000062960","ENSMUSG00000020053","ENSMUSG00000031273","ENSMUSG00000028047","ENSMUSG00000026840","ENSMUSG00000019846","ENSMUSG00000001281","ENSMUSG00000000184","ENSMUSG00000026193","ENSMUSG00000021448","ENSMUSG00000024867","ENSMUSG00000042284","ENSMUSG00000028626","ENSMUSG00000019899","ENSMUSG00000031520","ENSMUSG00000022439","ENSMUSG00000025348","ENSMUSG00000022438","ENSMUSG00000068699","ENSMUSG00000052698","ENSMUSG00000019843","ENSMUSG00000019699","ENSMUSG00000007655","ENSMUSG00000026768","ENSMUSG00000020758"],["Hgf","Itga4","Kdr","Igf1","Col4a6","Thbs3","Lamc3","Lama4","Itgb7","Ccnd2","Fn1","Shc3","Pip5k1b","Itga1","Col9a2","Lama2","Vegfc","Parvg","Itga7","Parvb","Flnc","Tln2","Fyn","Akt3","Cav1","Itga8","Itgb4"],["4.13","2.07","2.05","2.78","3.47","1.67","3.8","1.94","1.81","1.71","1.97","2.15","1.5","1.94","4.13","1.83","2.12","1.77","1.58","1.64","1.18","1.24","1.18","1.2","1.6","1.59","-1.5"],["4.67e-06","1.91e-05","2.77e-05","3.45e-05","4.39e-05","6.04e-05","8.26e-05","9.99e-05","0.000135","0.000137","0.000137","0.000159","0.000198","0.000365","0.000373","0.000677","0.000842","0.00117","0.00119","0.00146","0.00187","0.00193","0.0023","0.00271","0.00365","0.00387","0.00412"],["0.00158","0.00285","0.00338","0.00364","0.00413","0.0048","0.00535","0.00591","0.00671","0.00673","0.00673","0.00732","0.00839","0.0118","0.012","0.0166","0.0189","0.0225","0.0227","0.0259","0.03","0.0305","0.0345","0.038","0.0457","0.0475","0.0495"],["hepatocyte growth factor [Source:MGI Symbol;Acc:MGI:96079]","integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","kinase insert domain protein receptor [Source:MGI Symbol;Acc:MGI:96683]","insulin-like growth factor 1 [Source:MGI Symbol;Acc:MGI:96432]","collagen, type IV, alpha 6 [Source:MGI Symbol;Acc:MGI:2152695]","thrombospondin 3 [Source:MGI Symbol;Acc:MGI:98739]","laminin gamma 3 [Source:MGI Symbol;Acc:MGI:1344394]","laminin, alpha 4 [Source:MGI Symbol;Acc:MGI:109321]","integrin beta 7 [Source:MGI Symbol;Acc:MGI:96616]","cyclin D2 [Source:MGI Symbol;Acc:MGI:88314]","fibronectin 1 [Source:MGI Symbol;Acc:MGI:95566]","src homology 2 domain-containing transforming protein C3 [Source:MGI Symbol;Acc:MGI:106179]","phosphatidylinositol-4-phosphate 5-kinase, type 1 beta [Source:MGI Symbol;Acc:MGI:107930]","integrin alpha 1 [Source:MGI Symbol;Acc:MGI:96599]","collagen, type IX, alpha 2 [Source:MGI Symbol;Acc:MGI:88466]","laminin, alpha 2 [Source:MGI Symbol;Acc:MGI:99912]","vascular endothelial growth factor C [Source:MGI Symbol;Acc:MGI:109124]","parvin, gamma [Source:MGI Symbol;Acc:MGI:2158329]","integrin alpha 7 [Source:MGI Symbol;Acc:MGI:102700]","parvin, beta [Source:MGI Symbol;Acc:MGI:2153063]","filamin C, gamma [Source:MGI Symbol;Acc:MGI:95557]","talin 2 [Source:MGI Symbol;Acc:MGI:1917799]","Fyn proto-oncogene [Source:MGI Symbol;Acc:MGI:95602]","thymoma viral proto-oncogene 3 [Source:MGI Symbol;Acc:MGI:1345147]","caveolin 1, caveolae protein [Source:MGI Symbol;Acc:MGI:102709]","integrin alpha 8 [Source:MGI Symbol;Acc:MGI:109442]","integrin beta 4 [Source:MGI Symbol;Acc:MGI:96613]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-10" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04510.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="cytokine-cytokine-receptor-interaction-1"
class="section level3 tabset">
<h3 class="tabset">Cytokine-cytokine receptor interaction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-11" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-47-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-47-1">
Past versions of unnamed-chunk-47-1.png
</button>
</p>
<div id="fig-unnamed-chunk-47-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-47-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-47-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-47-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-47-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-10" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-938b6143a28577166e43" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-938b6143a28577166e43">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000029375","ENSMUSG00000034457","ENSMUSG00000073889","ENSMUSG00000050232","ENSMUSG00000061353","ENSMUSG00000025352","ENSMUSG00000037660","ENSMUSG00000031497","ENSMUSG00000021943","ENSMUSG00000005465","ENSMUSG00000063727","ENSMUSG00000026573","ENSMUSG00000060509","ENSMUSG00000071713","ENSMUSG00000000791","ENSMUSG00000031304","ENSMUSG00000003227"],["Cxcl15","Eda2r","Il11ra1","Cxcr3","Cxcl12","Gdf11","Gdf7","Tnfsf13b","Gdf10","Il27ra","Tnfrsf11b","Xcl1","Xcr1","Csf2rb","Il12rb1","Il2rg","Edar"],["6.92","3.52","1.54","2.28","2.57","2.25","3.41","2.22","2.74","1.94","2.13","2.67","2","2.03","2.13","1.88","2.57"],["1.77e-08","4.14e-05","0.000143","0.000222","0.000222","0.000276","0.000422","0.000488","0.000561","0.00077","0.000829","0.00118","0.00195","0.00198","0.00338","0.00371","0.00406"],["0.000153","0.00405","0.00689","0.00881","0.00881","0.0101","0.0129","0.0139","0.0149","0.0179","0.0187","0.0226","0.0308","0.031","0.0437","0.0462","0.0491"],["chemokine (C-X-C motif) ligand 15 [Source:MGI Symbol;Acc:MGI:1339941]","ectodysplasin A2 receptor [Source:MGI Symbol;Acc:MGI:2442860]","interleukin 11 receptor, alpha chain 1 [Source:MGI Symbol;Acc:MGI:107426]","chemokine (C-X-C motif) receptor 3 [Source:MGI Symbol;Acc:MGI:1277207]","chemokine (C-X-C motif) ligand 12 [Source:MGI Symbol;Acc:MGI:103556]","growth differentiation factor 11 [Source:MGI Symbol;Acc:MGI:1338027]","growth differentiation factor 7 [Source:MGI Symbol;Acc:MGI:95690]","tumor necrosis factor (ligand) superfamily, member 13b [Source:MGI Symbol;Acc:MGI:1344376]","growth differentiation factor 10 [Source:MGI Symbol;Acc:MGI:95684]","interleukin 27 receptor, alpha [Source:MGI Symbol;Acc:MGI:1355318]","tumor necrosis factor receptor superfamily, member 11b (osteoprotegerin) [Source:MGI Symbol;Acc:MGI:109587]","chemokine (C motif) ligand 1 [Source:MGI Symbol;Acc:MGI:104593]","chemokine (C motif) receptor 1 [Source:MGI Symbol;Acc:MGI:1346338]","colony stimulating factor 2 receptor, beta, low-affinity (granulocyte-macrophage) [Source:MGI Symbol;Acc:MGI:1339759]","interleukin 12 receptor, beta 1 [Source:MGI Symbol;Acc:MGI:104579]","interleukin 2 receptor, gamma chain [Source:MGI Symbol;Acc:MGI:96551]","ectodysplasin-A receptor [Source:MGI Symbol;Acc:MGI:1343498]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-11" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04060.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
</div>
<div id="fc1.5-1" class="section level2 tabset">
<h2 class="tabset">FC=1.5</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="leukocyte-transendothelial-migration-2"
class="section level3 tabset">
<h3 class="tabset">Leukocyte transendothelial migration</h3>
<pre class="r"><code>q=1</code></pre>
<div id="heatmap-12" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-51-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-51-1">
Past versions of unnamed-chunk-51-1.png
</button>
</p>
<div id="fig-unnamed-chunk-51-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-51-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-51-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-51-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-51-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-11" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-ba1db11e62e7dc98af18" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-ba1db11e62e7dc98af18">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000027009","ENSMUSG00000057614","ENSMUSG00000031740","ENSMUSG00000061353"],["Itga4","Gnai1","Mmp2","Cxcl12"],["2.07","2.41","2.27","2.57"],["0.000232","0.000662","0.000956","0.000976"],["0.0204","0.0369","0.0456","0.0461"],["integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","guanine nucleotide binding protein (G protein), alpha inhibiting 1 [Source:MGI Symbol;Acc:MGI:95771]","matrix metallopeptidase 2 [Source:MGI Symbol;Acc:MGI:97009]","chemokine (C-X-C motif) ligand 12 [Source:MGI Symbol;Acc:MGI:103556]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-12" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04670.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="hematopoietic-cell-lineage-2" class="section level3 tabset">
<h3 class="tabset">Hematopoietic cell lineage</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-13" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-54-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-54-1">
Past versions of unnamed-chunk-54-1.png
</button>
</p>
<div id="fig-unnamed-chunk-54-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-54-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-54-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-12" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-9b4bff5b63074d919354" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-9b4bff5b63074d919354">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000027009","ENSMUSG00000002033","ENSMUSG00000005087"],["Itga4","Cd3g","Cd44"],["2.07","3.65","2.27"],["0.000232","0.000542","0.000591"],["0.0204","0.0333","0.0347"],["integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","CD3 antigen, gamma polypeptide [Source:MGI Symbol;Acc:MGI:88333]","CD44 antigen [Source:MGI Symbol;Acc:MGI:88338]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-13" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04640.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="vascular-smooth-muscle-contraction-2"
class="section level3 tabset">
<h3 class="tabset">Vascular smooth muscle contraction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-14" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-57-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-57-1">
Past versions of unnamed-chunk-57-1.png
</button>
</p>
<div id="fig-unnamed-chunk-57-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-57-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-57-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-57-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-57-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-57-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-13" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-142eccf70a3246ab973e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-142eccf70a3246ab973e">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000022683","ENSMUSG00000045875","ENSMUSG00000030669","ENSMUSG00000027524"],["Pla2g10","Adra1a","Calca","Edn3"],["5.77","4.13","5.76","3.85"],["3.22e-05","0.000138","0.000547","0.000681"],["0.0076","0.0159","0.0333","0.0375"],["phospholipase A2, group X [Source:MGI Symbol;Acc:MGI:1347522]","adrenergic receptor, alpha 1a [Source:MGI Symbol;Acc:MGI:104773]","calcitonin/calcitonin-related polypeptide, alpha [Source:MGI Symbol;Acc:MGI:2151253]","endothelin 3 [Source:MGI Symbol;Acc:MGI:95285]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-14" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04270.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="pi3k-akt-signaling-pathway-2" class="section level3 tabset">
<h3 class="tabset">PI3K-Akt signaling pathway</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-15" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-60-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-60-1">
Past versions of unnamed-chunk-60-1.png
</button>
</p>
<div id="fig-unnamed-chunk-60-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-60-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-60-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-14" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-e4f33d686dc7fc37d324" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-e4f33d686dc7fc37d324">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000028864","ENSMUSG00000022309","ENSMUSG00000031273","ENSMUSG00000020053","ENSMUSG00000026840","ENSMUSG00000029378","ENSMUSG00000027009","ENSMUSG00000062960","ENSMUSG00000028978","ENSMUSG00000028626","ENSMUSG00000019846"],["Hgf","Angpt1","Col4a6","Igf1","Lamc3","Areg","Itga4","Kdr","Nos3","Col9a2","Lama4"],["4.13","3.39","3.47","2.78","3.8","5.25","2.07","2.05","2.15","4.13","1.94"],["1.35e-05","4.23e-05","0.000139","0.000173","0.000211","0.000226","0.000232","0.000327","0.000585","0.000733","0.00105"],["0.00457","0.00854","0.0159","0.0173","0.019","0.0201","0.0204","0.0254","0.0347","0.0389","0.0475"],["hepatocyte growth factor [Source:MGI Symbol;Acc:MGI:96079]","angiopoietin 1 [Source:MGI Symbol;Acc:MGI:108448]","collagen, type IV, alpha 6 [Source:MGI Symbol;Acc:MGI:2152695]","insulin-like growth factor 1 [Source:MGI Symbol;Acc:MGI:96432]","laminin gamma 3 [Source:MGI Symbol;Acc:MGI:1344394]","amphiregulin [Source:MGI Symbol;Acc:MGI:88068]","integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","kinase insert domain protein receptor [Source:MGI Symbol;Acc:MGI:96683]","nitric oxide synthase 3, endothelial cell [Source:MGI Symbol;Acc:MGI:97362]","collagen, type IX, alpha 2 [Source:MGI Symbol;Acc:MGI:88466]","laminin, alpha 4 [Source:MGI Symbol;Acc:MGI:109321]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-15" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04151.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="focal-adhesion-2" class="section level3 tabset">
<h3 class="tabset">Focal adhesion</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-16" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-63-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-63-1">
Past versions of unnamed-chunk-63-1.png
</button>
</p>
<div id="fig-unnamed-chunk-63-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-63-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-63-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-63-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/kegg.Rmd/unnamed-chunk-63-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-15" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-0ad488a309c6394c6151" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0ad488a309c6394c6151">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000028864","ENSMUSG00000031273","ENSMUSG00000020053","ENSMUSG00000026840","ENSMUSG00000027009","ENSMUSG00000062960","ENSMUSG00000028626","ENSMUSG00000019846"],["Hgf","Col4a6","Igf1","Lamc3","Itga4","Kdr","Col9a2","Lama4"],["4.13","3.47","2.78","3.8","2.07","2.05","4.13","1.94"],["1.35e-05","0.000139","0.000173","0.000211","0.000232","0.000327","0.000733","0.00105"],["0.00457","0.0159","0.0173","0.019","0.0204","0.0254","0.0389","0.0475"],["hepatocyte growth factor [Source:MGI Symbol;Acc:MGI:96079]","collagen, type IV, alpha 6 [Source:MGI Symbol;Acc:MGI:2152695]","insulin-like growth factor 1 [Source:MGI Symbol;Acc:MGI:96432]","laminin gamma 3 [Source:MGI Symbol;Acc:MGI:1344394]","integrin alpha 4 [Source:MGI Symbol;Acc:MGI:96603]","kinase insert domain protein receptor [Source:MGI Symbol;Acc:MGI:96683]","collagen, type IX, alpha 2 [Source:MGI Symbol;Acc:MGI:88466]","laminin, alpha 4 [Source:MGI Symbol;Acc:MGI:109321]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-16" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04510.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
<div id="cytokine-cytokine-receptor-interaction-2"
class="section level3 tabset">
<h3 class="tabset">Cytokine-cytokine receptor interaction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-17" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-66-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-66-1">
Past versions of unnamed-chunk-66-1.png
</button>
</p>
<div id="fig-unnamed-chunk-66-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b44640abac9ec4e34694489147942349d93fa0c2/docs/figure/kegg.Rmd/unnamed-chunk-66-1.png" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/kegg.Rmd/unnamed-chunk-66-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/kegg.Rmd/unnamed-chunk-66-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/kegg.Rmd/unnamed-chunk-66-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tables-16" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(.,3)))) %&gt;%
  DT(.,caption = &quot;DE genes&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-ad80b168e4fab6515cbc" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-ad80b168e4fab6515cbc">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>DE genes<\/caption>","data":[["ENSMUSG00000029375","ENSMUSG00000034457","ENSMUSG00000061353","ENSMUSG00000037660"],["Cxcl15","Eda2r","Cxcl12","Gdf7"],["6.92","3.52","2.57","3.41"],["3.25e-08","0.000128","0.000976","0.00103"],["0.00028","0.0153","0.0461","0.0473"],["chemokine (C-X-C motif) ligand 15 [Source:MGI Symbol;Acc:MGI:1339941]","ectodysplasin A2 receptor [Source:MGI Symbol;Acc:MGI:2442860]","chemokine (C-X-C motif) ligand 12 [Source:MGI Symbol;Acc:MGI:103556]","growth differentiation factor 7 [Source:MGI Symbol;Acc:MGI:95690]"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene Name<\/th>\n      <th>logFC<\/th>\n      <th>P Value<\/th>\n      <th>Adjusted P Value<\/th>\n      <th>Description<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="pathview-17" class="section level4">
<h4>Pathview</h4>
<div class="float">
<img src="figure/kegg.Rmd/pv_NA_mmu04060.png" alt="Pathview" />
<div class="figcaption">Pathview</div>
</div>
</div>
</div>
</div>
</div>
<div id="export-data" class="section level1">
<h1>Export Data</h1>
<pre class="r"><code># save to csv
writexl::write_xlsx(x = enrichKEGG_all, here::here(&quot;3_output/enrichKEGG_all.xlsx&quot;))
writexl::write_xlsx(x = enrichKEGG_sig, here::here(&quot;3_output/enrichKEGG_sig.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_Australia.utf8  LC_CTYPE=English_Australia.utf8   
[3] LC_MONETARY=English_Australia.utf8 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.utf8    

time zone: Australia/Adelaide
tzcode source: internal

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] pathview_1.40.0       enrichplot_1.20.1     org.Mm.eg.db_3.17.0  
 [4] AnnotationDbi_1.62.2  IRanges_2.34.1        S4Vectors_0.38.1     
 [7] Biobase_2.60.0        BiocGenerics_0.46.0   clusterProfiler_4.8.3
[10] Glimma_2.10.0         edgeR_3.42.4          limma_3.56.2         
[13] ggrepel_0.9.3         ggbiplot_0.55         scales_1.2.1         
[16] plyr_1.8.8            ggpubr_0.6.0          ggplotify_0.1.2      
[19] DT_0.29               pheatmap_1.0.12       cowplot_1.1.1        
[22] viridis_0.6.4         viridisLite_0.4.2     pander_0.6.5         
[25] kableExtra_1.3.4      KEGGREST_1.40.0       lubridate_1.9.2      
[28] forcats_1.0.0         stringr_1.5.0         purrr_1.0.1          
[31] tidyr_1.3.0           ggplot2_3.4.3         tidyverse_2.0.0      
[34] reshape2_1.4.4        tibble_3.2.1          readr_2.1.4          
[37] magrittr_2.0.3        dplyr_1.1.2          

loaded via a namespace (and not attached):
  [1] splines_4.3.1               later_1.3.1                
  [3] bitops_1.0-7                polyclip_1.10-4            
  [5] graph_1.78.0                XML_3.99-0.14              
  [7] lifecycle_1.0.3             rstatix_0.7.2              
  [9] rprojroot_2.0.3             lattice_0.21-8             
 [11] MASS_7.3-60                 crosstalk_1.2.0            
 [13] backports_1.4.1             sass_0.4.7                 
 [15] rmarkdown_2.24              jquerylib_0.1.4            
 [17] yaml_2.3.7                  httpuv_1.6.11              
 [19] DBI_1.1.3                   RColorBrewer_1.1-3         
 [21] abind_1.4-5                 zlibbioc_1.46.0            
 [23] rvest_1.0.3                 GenomicRanges_1.52.0       
 [25] ggraph_2.1.0                RCurl_1.98-1.12            
 [27] yulab.utils_0.0.9           tweenr_2.0.2               
 [29] git2r_0.32.0                GenomeInfoDbData_1.2.10    
 [31] tidytree_0.4.5              svglite_2.1.1              
 [33] codetools_0.2-19            DelayedArray_0.26.7        
 [35] DOSE_3.26.1                 xml2_1.3.5                 
 [37] ggforce_0.4.1               tidyselect_1.2.0           
 [39] aplot_0.2.1                 farver_2.1.1               
 [41] matrixStats_1.0.0           webshot_0.5.5              
 [43] jsonlite_1.8.7              ellipsis_0.3.2             
 [45] tidygraph_1.2.3             systemfonts_1.0.4          
 [47] tools_4.3.1                 ragg_1.2.5                 
 [49] treeio_1.24.3               Rcpp_1.0.11                
 [51] glue_1.6.2                  gridExtra_2.3              
 [53] here_1.0.1                  xfun_0.39                  
 [55] DESeq2_1.40.2               qvalue_2.32.0              
 [57] MatrixGenerics_1.12.3       GenomeInfoDb_1.36.3        
 [59] withr_2.5.0                 fastmap_1.1.1              
 [61] fansi_1.0.4                 digest_0.6.33              
 [63] timechange_0.2.0            R6_2.5.1                   
 [65] gridGraphics_0.5-1          textshaping_0.3.6          
 [67] colorspace_2.1-0            GO.db_3.17.0               
 [69] RSQLite_2.3.1               utf8_1.2.3                 
 [71] generics_0.1.3              data.table_1.14.8          
 [73] graphlayouts_1.0.0          httr_1.4.7                 
 [75] htmlwidgets_1.6.2           S4Arrays_1.0.6             
 [77] scatterpie_0.2.1            whisker_0.4.1              
 [79] pkgconfig_2.0.3             gtable_0.3.4               
 [81] blob_1.2.4                  workflowr_1.7.1            
 [83] XVector_0.40.0              shadowtext_0.1.2           
 [85] htmltools_0.5.5             carData_3.0-5              
 [87] fgsea_1.26.0                ggupset_0.3.0              
 [89] png_0.1-8                   ggfun_0.1.3                
 [91] knitr_1.44                  rstudioapi_0.15.0          
 [93] tzdb_0.4.0                  curl_5.0.2                 
 [95] nlme_3.1-163                org.Hs.eg.db_3.17.0        
 [97] cachem_1.0.8                parallel_4.3.1             
 [99] HDO.db_0.99.1               pillar_1.9.0               
[101] vctrs_0.6.3                 promises_1.2.0.1           
[103] car_3.1-2                   Rgraphviz_2.44.0           
[105] KEGGgraph_1.60.0            evaluate_0.21              
[107] cli_3.6.1                   locfit_1.5-9.8             
[109] compiler_4.3.1              rlang_1.1.1                
[111] crayon_1.5.2                ggsignif_0.6.4             
[113] labeling_0.4.3              fs_1.6.3                   
[115] writexl_1.4.2               stringi_1.7.12             
[117] BiocParallel_1.34.2         munsell_0.5.0              
[119] Biostrings_2.68.1           lazyeval_0.2.2             
[121] GOSemSim_2.26.1             Matrix_1.6-1               
[123] hms_1.1.3                   patchwork_1.1.3            
[125] bit64_4.0.5                 SummarizedExperiment_1.30.2
[127] igraph_1.5.1                broom_1.0.5                
[129] memoise_2.0.1               bslib_0.5.1                
[131] ggtree_3.8.2                fastmatch_1.1-4            
[133] bit_4.0.5                   downloader_0.4             
[135] gson_0.1.0                  ape_5.7-1                  </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
