<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha Tran" />


<title>KEGG</title>

<script src="site_libs/header-attrs-2.17/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Mouse endometial transcriptome 2023</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="setUp.html">Pre-processing</a>
</li>
<li>
  <a href="deAnalysis.html">DGE Analysis</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Functional Enrichment Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="go.html">Gene Ontology</a>
    </li>
    <li>
      <a href="kegg.html">KEGG</a>
    </li>
    <li>
      <a href="reactome.html">Reactome</a>
    </li>
    <li>
      <a href="extraFigures.html">IPA</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">KEGG</h1>
<h4 class="author">Ha Tran</h4>
<h4 class="date">22/08/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-01-21
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SRB_2022/1_analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the
code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135SRB2022tree01a61cbf537cd052d7810928d8a67dba6481da8btargetblank01a61cba">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/tranmanhha135/SRB_2022/tree/01a61cbf537cd052d7810928d8a67dba6481da8b" target="_blank">01a61cb</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135SRB2022tree01a61cbf537cd052d7810928d8a67dba6481da8btargetblank01a61cba"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/tranmanhha135/SRB_2022/tree/01a61cbf537cd052d7810928d8a67dba6481da8b" target="_blank">01a61cb</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  .gitignore

Unstaged changes:
    Modified:   0_data/RDS_objects/dge.rds
    Modified:   0_data/RDS_objects/enrichGO.rds
    Modified:   0_data/RDS_objects/enrichGO_sig.rds
    Modified:   0_data/RDS_objects/fc.rds
    Modified:   0_data/RDS_objects/lfc.rds
    Modified:   0_data/RDS_objects/lmTreat.rds
    Modified:   0_data/RDS_objects/lmTreat_all.rds
    Modified:   0_data/RDS_objects/lmTreat_sig.rds
    Modified:   0_data/RDS_objects/pub.rds
    Deleted:    1_analysis/mmu04613.pv.multi.png
    Deleted:    1_analysis/mmu04658.pv.multi.png
    Deleted:    1_analysis/mmu04659.pv.multi.png
    Deleted:    1_analysis/mmu04660.pv.multi.png
    Deleted:    1_analysis/mmu04710.pv.multi.png
    Deleted:    1_analysis/mmu05235.pv.multi.png
    Deleted:    1_analysis/testGraph.gml
    Modified:   2_plots/de/heat_down_1.05.svg
    Modified:   2_plots/de/heat_down_1.1.svg
    Modified:   2_plots/de/heat_down_1.5.svg
    Modified:   2_plots/de/heat_up_1.05.svg
    Modified:   2_plots/de/heat_up_1.1.svg
    Modified:   2_plots/de/heat_up_1.5.svg
    Modified:   2_plots/de/ma_1.05.png
    Modified:   2_plots/de/ma_1.1.png
    Modified:   2_plots/de/ma_1.5.png
    Modified:   2_plots/de/pval_1.05.svg
    Modified:   2_plots/de/pval_1.1.svg
    Modified:   2_plots/de/pval_1.5.svg
    Modified:   2_plots/de/vol_1.05.png
    Modified:   2_plots/de/vol_1.1.png
    Modified:   2_plots/de/vol_1.5.png
    Modified:   2_plots/go/bp_dot_1.05.svg
    Modified:   2_plots/go/bp_dot_1.1.svg
    Modified:   2_plots/go/bp_dot_1.5.svg
    Modified:   2_plots/go/cc_dot_1.05.svg
    Modified:   2_plots/go/cc_dot_1.1.svg
    Modified:   2_plots/go/cc_dot_1.5.svg
    Modified:   2_plots/go/mf_dot_1.05.svg
    Modified:   2_plots/go/mf_dot_1.1.svg
    Modified:   2_plots/go/mf_dot_1.5.svg
    Modified:   2_plots/go/upset_1.05.svg
    Modified:   2_plots/go/upset_1.1.svg
    Modified:   2_plots/go/upset_1.5.svg
    Modified:   2_plots/ipa/Cardiovascular System.svg
    Modified:   2_plots/ipa/Cell-To-Cell Signaling.svg
    Modified:   2_plots/ipa/Cellular Movement.svg
    Modified:   2_plots/ipa/diseaseAndFunction.svg
    Modified:   2_plots/ipa/pathways.svg
    Modified:   2_plots/ipa/upstream_2.svg
    Modified:   2_plots/kegg/heat_1.05_Cytokine-cytokine receptor interaction.svg
    Modified:   2_plots/kegg/heat_1.05_Focal adhesion.svg
    Modified:   2_plots/kegg/heat_1.05_Hematopoietic cell lineage.svg
    Modified:   2_plots/kegg/heat_1.05_Leukocyte transendothelial migration.svg
    Modified:   2_plots/kegg/heat_1.05_PI3K-Akt signaling pathway.svg
    Modified:   2_plots/kegg/heat_1.05_Vascular smooth muscle contraction.svg
    Modified:   2_plots/kegg/heat_1.1_Cytokine-cytokine receptor interaction.svg
    Modified:   2_plots/kegg/heat_1.1_Focal adhesion.svg
    Modified:   2_plots/kegg/heat_1.1_Hematopoietic cell lineage.svg
    Modified:   2_plots/kegg/heat_1.1_Leukocyte transendothelial migration.svg
    Modified:   2_plots/kegg/heat_1.1_PI3K-Akt signaling pathway.svg
    Modified:   2_plots/kegg/heat_1.1_Vascular smooth muscle contraction.svg
    Modified:   2_plots/kegg/heat_1.5_Cytokine-cytokine receptor interaction.svg
    Modified:   2_plots/kegg/heat_1.5_Focal adhesion.svg
    Modified:   2_plots/kegg/heat_1.5_Hematopoietic cell lineage.svg
    Modified:   2_plots/kegg/heat_1.5_Leukocyte transendothelial migration.svg
    Modified:   2_plots/kegg/heat_1.5_PI3K-Akt signaling pathway.svg
    Modified:   2_plots/kegg/heat_1.5_Vascular smooth muscle contraction.svg
    Modified:   2_plots/kegg/kegg_dot_1.05.svg
    Modified:   2_plots/kegg/kegg_dot_1.1.svg
    Modified:   2_plots/kegg/kegg_dot_1.5.svg
    Modified:   2_plots/kegg/mmu04060.png
    Modified:   2_plots/kegg/mmu04060.xml
    Modified:   2_plots/kegg/mmu04151.xml
    Modified:   2_plots/kegg/mmu04270.xml
    Modified:   2_plots/kegg/mmu04510.xml
    Modified:   2_plots/kegg/mmu04640.png
    Modified:   2_plots/kegg/mmu04640.xml
    Modified:   2_plots/kegg/mmu04670.xml
    Deleted:    2_plots/kegg/pv_1.05_mmu04060.png
    Deleted:    2_plots/kegg/pv_1.05_mmu04151.png
    Deleted:    2_plots/kegg/pv_1.05_mmu04270.png
    Deleted:    2_plots/kegg/pv_1.05_mmu04510.png
    Deleted:    2_plots/kegg/pv_1.05_mmu04640.png
    Deleted:    2_plots/kegg/pv_1.05_mmu04670.png
    Deleted:    2_plots/kegg/pv_1.1_mmu04060.png
    Deleted:    2_plots/kegg/pv_1.1_mmu04151.png
    Deleted:    2_plots/kegg/pv_1.1_mmu04270.png
    Deleted:    2_plots/kegg/pv_1.1_mmu04510.png
    Deleted:    2_plots/kegg/pv_1.1_mmu04640.png
    Deleted:    2_plots/kegg/pv_1.1_mmu04670.png
    Deleted:    2_plots/kegg/pv_1.5_mmu04060.png
    Deleted:    2_plots/kegg/pv_1.5_mmu04151.png
    Deleted:    2_plots/kegg/pv_1.5_mmu04270.png
    Deleted:    2_plots/kegg/pv_1.5_mmu04510.png
    Deleted:    2_plots/kegg/pv_1.5_mmu04640.png
    Deleted:    2_plots/kegg/pv_1.5_mmu04670.png
    Modified:   2_plots/kegg/upset_kegg_1.05.svg
    Modified:   2_plots/kegg/upset_kegg_1.1.svg
    Modified:   2_plots/kegg/upset_kegg_1.5.svg
    Modified:   2_plots/qc/PCA_IntvsCont.svg
    Modified:   2_plots/qc/counts_after_filtering_3_3.svg
    Modified:   2_plots/qc/counts_before_after_filtering_3_3.svg
    Modified:   2_plots/qc/counts_before_filtering.svg
    Modified:   2_plots/qc/library_size.svg
    Modified:   2_plots/reactome/react_dot_1.05.svg
    Modified:   2_plots/reactome/react_dot_1.1.svg
    Modified:   2_plots/reactome/react_dot_1.5.svg
    Modified:   2_plots/reactome/upset_react_1.05.svg
    Modified:   2_plots/reactome/upset_react_1.1.svg
    Modified:   2_plots/reactome/upset_react_1.5.svg
    Modified:   3_output/enrichGO_sig.xlsx
    Modified:   3_output/enrichKEGG_all.xlsx
    Modified:   3_output/enrichKEGG_sig.xlsx
    Modified:   3_output/lmTreat_all.xlsx
    Modified:   3_output/lmTreat_fc1.5_voom2_all_fdr.xlsx
    Modified:   3_output/lmTreat_sig.xlsx
    Modified:   3_output/reactome_all.xlsx
    Modified:   3_output/reactome_sig.xlsx
    Modified:   README.md
    Modified:   SRB_2022.Rproj
    Deleted:    test plz delete me.Rmd
    Deleted:    test-plz-delete-me.html

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>1_analysis/kegg.Rmd</code>) and HTML
(<code>docs/kegg.html</code>) files. If you’ve configured a remote Git
repository (see <code>?wflow_git_remote</code>), click on the hyperlinks
in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/159f3529790db3ae75472a460696c82121d932a3/1_analysis/kegg.Rmd" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
adding for pathview
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/SRB_2022/159f3529790db3ae75472a460696c82121d932a3/docs/kegg.html" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
adding for pathview
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/1_analysis/kegg.Rmd" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
<td>
test png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/SRB_2022/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/kegg.html" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
<td>
test png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/SRB_2022/691cf3499cc98849bde703fde17b25047c072b71/docs/kegg.html" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/7f6bab239f8f9682dab208d1c314dd72dbbd21bc/1_analysis/kegg.Rmd" target="_blank">7f6bab2</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
workflowr::wflow_publish(here::here("1_analysis/*.Rmd"))
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/b6cf1906d2fd775f953bd1dd6b835234e260209c/1_analysis/kegg.Rmd" target="_blank">b6cf190</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-19
</td>
<td>
quick commit
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/1_analysis/kegg.Rmd" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/SRB_2022/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/kegg.html" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)
library(KEGGREST)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(viridis)
library(cowplot)
library(pheatmap)

# Custom ggplot
library(ggplotify)
library(ggpubr)
library(ggbiplot)
library(ggrepel)

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Mm.eg.db)
library(enrichplot)
library(pathview)

theme_set(theme_minimal())
pub &lt;- readRDS(here::here(&quot;0_data/RDS_objects/pub.rds&quot;))</code></pre>
<div id="import-dgelist-data" class="section level3">
<h3>Import DGElist Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and
gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/RDS_objects/dge.rds&quot;))
fc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/fc.rds&quot;))
lfc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lfc.rds&quot;))
lmTreat &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat.rds&quot;))
lmTreat_sig &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat_sig.rds&quot;))</code></pre>
</div>
</div>
<div id="kegg-analysis" class="section level1 tabset">
<h1 class="tabset">KEGG Analysis</h1>
<p>KEGG enrichment analysis is performed with the significant DE genes
that have absolute FC &gt; 1.5 ( genes from Limma). Top 30 most
significant KEGG are displayed. All enriched KEGG pathways are
exported</p>
<pre class="r"><code># chosing the pathways of interest
kegg_id &lt;- c(&quot;mmu04670&quot;, &quot;mmu04640&quot;, &quot;mmu04270&quot;, &quot;mmu04151&quot;, &quot;mmu04510&quot;, &quot;mmu04060&quot;)
kegg_pathway &lt;- KEGGREST::keggGet(kegg_id)</code></pre>
<pre class="r"><code>enrichKEGG &lt;- list()
enrichKEGG_all &lt;- list()
enrichKEGG_sig &lt;- list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  # find enriched KEGG pathways
  enrichKEGG[[x]] &lt;- clusterProfiler::enrichKEGG(
    gene = lmTreat_sig[[x]]$entrezid,
    keyType = &quot;kegg&quot;,
    organism = &quot;mmu&quot;,
    pvalueCutoff = 0.05,
    pAdjustMethod = &quot;none&quot;
  )

  enrichKEGG[[x]] &lt;- enrichKEGG[[x]] %&gt;% 
    clusterProfiler::setReadable(OrgDb = org.Mm.eg.db, keyType = &quot;ENTREZID&quot;)
   
  enrichKEGG_all[[x]] &lt;- enrichKEGG[[x]]@result

  # filter the significant and print top 30
  enrichKEGG_sig[[x]] &lt;- enrichKEGG_all[[x]] %&gt;%
    dplyr::filter(pvalue &lt;= 0.05) %&gt;%
    separate(col = BgRatio, sep = &quot;/&quot;, into = c(&quot;Total&quot;, &quot;Universe&quot;)) %&gt;%
    dplyr::mutate(
      logPval = -log(pvalue, 10),
      GeneRatio = Count / as.numeric(Total)
    ) %&gt;%
    dplyr::select(c(&quot;Description&quot;, &quot;GeneRatio&quot;, &quot;pvalue&quot;, &quot;logPval&quot;, &quot;p.adjust&quot;, &quot;qvalue&quot;, &quot;geneID&quot;, &quot;Count&quot;))
  
  # # at the beginnning of a word (after 35 characters), add a newline. shorten the y axis for dot plot 
  # enrichKEGG_sig[[x]]$Description &lt;- sub(pattern = &quot;(.{1,35})(?:$| )&quot;, 
  #                                      replacement = &quot;\\1\n&quot;, 
  #                                      x = enrichKEGG_sig[[x]]$Description)
  # 
  # # remove the additional newline at the end of the string
  # enrichKEGG_sig[[x]]$Description &lt;- sub(pattern = &quot;\n$&quot;, 
  #                                      replacement = &quot;&quot;, 
  #                                      x = enrichKEGG_sig[[x]]$Description)
}</code></pre>
<div id="fc1.05" class="section level2 tabset">
<h2 class="tabset">FC=1.05</h2>
<pre class="r"><code>p=1</code></pre>
<div id="table" class="section level3">
<h3>Table</h3>
<pre class="r"><code>enrichKEGG_sig[[p]] %&gt;%
  kable(caption = &quot;Significantly enriched KEGG pathways&quot;) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>
Significantly enriched KEGG pathways
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logPval
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mmu04713
</td>
<td style="text-align:left;">
Circadian entrainment
</td>
<td style="text-align:right;">
0.2448980
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
7.941626
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000025
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Cacna1c/Ryr1/Gnai1/Gucy1a2/Grin2b/Adcy3/Gngt2/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Gng11/Gng2/Adcyap1r1/Rps6ka5/Plcb2/Ryr3/Cacna1h/Gnb5/Gng7/Gnb4
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04514
</td>
<td style="text-align:left;">
Cell adhesion molecules
</td>
<td style="text-align:right;">
0.1868132
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
7.697813
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000025
</td>
<td style="text-align:left;">
Itga4/Lrrc4c/Ptprd/Nlgn1/Cdh2/Itgb7/Lrrc4b/Icam1/Jam3/Cldn8/Slitrk3/Ptprm/Cdh5/Cd226/Itgal/Esam/H2-Ab1/Nlgn3/Nfasc/H2-Ob/Sele/Pecam1/H2-Eb1/Neo1/Cldn5/H2-Aa/Ntng1/Itga8/Spn/Cldn1/Cd4/Vcan/Nlgn2/Jam2
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04724
</td>
<td style="text-align:left;">
Glutamatergic synapse
</td>
<td style="text-align:right;">
0.2123894
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
6.671493
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
0.0000174
</td>
<td style="text-align:left;">
Slc1a3/Kcnj3/Gria4/Cacna1c/Grik1/Gnai1/Grin2b/Adcy3/Shank2/Gngt2/Plcb1/Adcy4/Adcy7/Gng11/Grik2/Gng2/Plcb2/Slc38a1/Dlg4/Gnb5/Gng7/Gnb4/Cacna1a/Pld2
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04371
</td>
<td style="text-align:left;">
Apelin signaling pathway
</td>
<td style="text-align:right;">
0.1897810
</td>
<td style="text-align:right;">
0.0000007
</td>
<td style="text-align:right;">
6.160273
</td>
<td style="text-align:right;">
0.0000007
</td>
<td style="text-align:right;">
0.0000424
</td>
<td style="text-align:left;">
Ryr1/Slc8a1/Nos3/Gnai1/Mef2c/Adcy3/Gngt2/Pik3r6/Plcb1/Slc8a3/Adcy4/Adcy7/Gng11/Aplnr/Agtr1a/Gng2/Myl3/Plcb2/Ryr3/Akt3/Ccn2/Gnb5/Gng7/Apln/Gnb4/Lipe
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05414
</td>
<td style="text-align:left;">
Dilated cardiomyopathy
</td>
<td style="text-align:right;">
0.2127660
</td>
<td style="text-align:right;">
0.0000021
</td>
<td style="text-align:right;">
5.671588
</td>
<td style="text-align:right;">
0.0000021
</td>
<td style="text-align:right;">
0.0001045
</td>
<td style="text-align:left;">
Cacna1c/Sgcd/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Adcy3/Itga1/Slc8a3/Adcy4/Adcy7/Lama2/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04340
</td>
<td style="text-align:left;">
Hedgehog signaling pathway
</td>
<td style="text-align:right;">
0.2586207
</td>
<td style="text-align:right;">
0.0000031
</td>
<td style="text-align:right;">
5.503784
</td>
<td style="text-align:right;">
0.0000031
</td>
<td style="text-align:right;">
0.0001117
</td>
<td style="text-align:left;">
Gli1/Ihh/Hhip/Evc2/Ptch2/Evc/Kif7/Ccnd2/Gli3/Lrp2/Gpr161/Arrb1/Arrb2/Ptch1/Iqce
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04015
</td>
<td style="text-align:left;">
Rap1 signaling pathway
</td>
<td style="text-align:right;">
0.1542056
</td>
<td style="text-align:right;">
0.0000032
</td>
<td style="text-align:right;">
5.496372
</td>
<td style="text-align:right;">
0.0000032
</td>
<td style="text-align:right;">
0.0001117
</td>
<td style="text-align:left;">
Hgf/Angpt1/Kdr/Igf1/Evl/Rasgrp3/Gnai1/Pfn2/Lpar4/Grin2b/Prkd1/Adcy3/Tek/Plcb1/Rasgrp2/Kit/Adcy4/Adcy7/Itgal/Vegfc/Magi2/Pard6g/Kitl/Arap3/Tln2/Plcb2/Akt3/Apbb1ip/Lat/Pdgfrb/Lcp2/F2r/Sipa1
</td>
<td style="text-align:right;">
33
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05032
</td>
<td style="text-align:left;">
Morphine addiction
</td>
<td style="text-align:right;">
0.2087912
</td>
<td style="text-align:right;">
0.0000051
</td>
<td style="text-align:right;">
5.289652
</td>
<td style="text-align:right;">
0.0000051
</td>
<td style="text-align:right;">
0.0001574
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Pde1b/Adcy3/Gngt2/Adcy4/Adcy7/Gabra4/Gng11/Pde1a/Gng2/Arrb1/Arrb2/Gnb5/Gng7/Gnb4/Gabbr1/Pde8a/Cacna1a
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04022
</td>
<td style="text-align:left;">
cGMP-PKG signaling pathway
</td>
<td style="text-align:right;">
0.1618497
</td>
<td style="text-align:right;">
0.0000070
</td>
<td style="text-align:right;">
5.157122
</td>
<td style="text-align:right;">
0.0000070
</td>
<td style="text-align:right;">
0.0001788
</td>
<td style="text-align:left;">
Cacna1c/Adra1a/Slc8a1/Nos3/Gnai1/Ednrb/Mef2c/Gucy1a2/Adcy3/Pik3r6/Plcb1/Slc8a3/Trpc6/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Bdkrb2/Nfatc4/Plcb2/Pde5a/Akt3/Atp2b4/Atp1a2/Irs1
</td>
<td style="text-align:right;">
28
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05412
</td>
<td style="text-align:left;">
Arrhythmogenic right ventricular cardiomyopathy
</td>
<td style="text-align:right;">
0.2207792
</td>
<td style="text-align:right;">
0.0000073
</td>
<td style="text-align:right;">
5.137196
</td>
<td style="text-align:right;">
0.0000073
</td>
<td style="text-align:right;">
0.0001788
</td>
<td style="text-align:left;">
Cacna1c/Lef1/Sgcd/Itga4/Slc8a1/Cdh2/Itgb7/Cacna2d1/Itga1/Slc8a3/Lama2/Itga7/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:right;">
0.2045455
</td>
<td style="text-align:right;">
0.0000123
</td>
<td style="text-align:right;">
4.911452
</td>
<td style="text-align:right;">
0.0000123
</td>
<td style="text-align:right;">
0.0002734
</td>
<td style="text-align:left;">
Itga4/Col4a6/Thbs3/Npnt/Cd44/Lamc3/Lama4/Itgb7/Fn1/Itga1/Col9a2/Frem1/Lama2/Itga7/Sv2c/Itga8/Itgb4/Col4a5
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04020
</td>
<td style="text-align:left;">
Calcium signaling pathway
</td>
<td style="text-align:right;">
0.1416667
</td>
<td style="text-align:right;">
0.0000150
</td>
<td style="text-align:right;">
4.823136
</td>
<td style="text-align:right;">
0.0000150
</td>
<td style="text-align:right;">
0.0003071
</td>
<td style="text-align:left;">
Hgf/Cacna1c/Kdr/Ryr1/Adra1a/Slc8a1/Nos3/P2rx2/Ednrb/Pde1b/Adcy3/Cacna1e/Ptgfr/Plcb1/Slc8a3/Adcy4/Adcy7/Vegfc/Cd38/Agtr1a/Pde1a/Ednra/Bdkrb2/Htr2a/Plcb2/Asph/Ryr3/Cacna1h/Ptger3/Atp2b4/Pdgfrb/Cacna1a/F2r/Camk1
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04924
</td>
<td style="text-align:left;">
Renin secretion
</td>
<td style="text-align:right;">
0.2105263
</td>
<td style="text-align:right;">
0.0000255
</td>
<td style="text-align:right;">
4.594023
</td>
<td style="text-align:right;">
0.0000255
</td>
<td style="text-align:right;">
0.0004805
</td>
<td style="text-align:left;">
Cacna1c/Gnai1/Gucy1a2/Pde1b/Edn3/Plcb1/Gucy1b1/Gucy1a1/Agtr1a/Pde1a/Kcnma1/Ednra/Adcyap1r1/Plcb2/Clca2/Ptger2
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04926
</td>
<td style="text-align:left;">
Relaxin signaling pathway
</td>
<td style="text-align:right;">
0.1705426
</td>
<td style="text-align:right;">
0.0000292
</td>
<td style="text-align:right;">
4.535161
</td>
<td style="text-align:right;">
0.0000292
</td>
<td style="text-align:right;">
0.0005109
</td>
<td style="text-align:left;">
Col4a6/Nos3/Gnai1/Ednrb/Mmp2/Shc3/Adcy3/Gngt2/Plcb1/Adcy4/Adcy7/Gng11/Vegfc/Gng2/Arrb1/Plcb2/Arrb2/Akt3/Gnb5/Gng7/Gnb4/Col4a5
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04725
</td>
<td style="text-align:left;">
Cholinergic synapse
</td>
<td style="text-align:right;">
0.1785714
</td>
<td style="text-align:right;">
0.0000338
</td>
<td style="text-align:right;">
4.471504
</td>
<td style="text-align:right;">
0.0000338
</td>
<td style="text-align:right;">
0.0005521
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Gnai1/Kcnq5/Adcy3/Gngt2/Pik3r6/Plcb1/Adcy4/Adcy7/Gng11/Gng2/Plcb2/Fyn/Akt3/Gnb5/Gng7/Gnb4/Kcnq4/Cacna1a
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04510
</td>
<td style="text-align:left;">
Focal adhesion
</td>
<td style="text-align:right;">
0.1442786
</td>
<td style="text-align:right;">
0.0000452
</td>
<td style="text-align:right;">
4.345003
</td>
<td style="text-align:right;">
0.0000452
</td>
<td style="text-align:right;">
0.0006582
</td>
<td style="text-align:left;">
Hgf/Itga4/Kdr/Igf1/Col4a6/Thbs3/Lamc3/Lama4/Ccnd2/Itgb7/Fn1/Shc3/Pip5k1b/Itga1/Col9a2/Lama2/Vegfc/Itga7/Parvg/Parvb/Flnc/Tln2/Fyn/Akt3/Pdgfrb/Cav1/Itga8/Itgb4/Col4a5
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04360
</td>
<td style="text-align:left;">
Axon guidance
</td>
<td style="text-align:right;">
0.1491713
</td>
<td style="text-align:right;">
0.0000456
</td>
<td style="text-align:right;">
4.340829
</td>
<td style="text-align:right;">
0.0000456
</td>
<td style="text-align:right;">
0.0006582
</td>
<td style="text-align:left;">
Epha8/Efnb3/Sema5a/Lrrc4c/Dcc/Plxnb1/Sema5b/Nrp1/Sema7a/Gnai1/Cxcl12/Trpc6/Gdf7/Epha3/Sema3g/Pard6g/Nfatc4/Epha7/Neo1/Ptch1/Fyn/Unc5c/Ngef/Sema6a/Ntng1/Bmp7/Slit3
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04014
</td>
<td style="text-align:left;">
Ras signaling pathway
</td>
<td style="text-align:right;">
0.1361702
</td>
<td style="text-align:right;">
0.0000590
</td>
<td style="text-align:right;">
4.229098
</td>
<td style="text-align:right;">
0.0000590
</td>
<td style="text-align:right;">
0.0007718
</td>
<td style="text-align:left;">
Hgf/Angpt1/Pla2g10/Kdr/Igf1/Ets1/Rasgrp3/Shc3/Rasa3/Grin2b/Tek/Gngt2/Rasgrp2/Flt3/Kit/Gng11/Vegfc/Igf2/Gng2/Kitl/Pla1a/Akt3/Gnb5/Gng7/Ets2/Syngap1/Zap70/Gnb4/Lat/Pdgfrb/Pld2/Rasal3
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04728
</td>
<td style="text-align:left;">
Dopaminergic synapse
</td>
<td style="text-align:right;">
0.1629630
</td>
<td style="text-align:right;">
0.0000598
</td>
<td style="text-align:right;">
4.223382
</td>
<td style="text-align:right;">
0.0000598
</td>
<td style="text-align:right;">
0.0007718
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Cacna1c/Maob/Drd4/Gnai1/Scn1a/Grin2b/Maoa/Gngt2/Plcb1/Gng11/Gng2/Arrb1/Plcb2/Arrb2/Akt3/Gnb5/Gng7/Gnb4/Cacna1a/Kif5a
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04974
</td>
<td style="text-align:left;">
Protein digestion and absorption
</td>
<td style="text-align:right;">
0.1759259
</td>
<td style="text-align:right;">
0.0000653
</td>
<td style="text-align:right;">
4.185073
</td>
<td style="text-align:right;">
0.0000653
</td>
<td style="text-align:right;">
0.0008008
</td>
<td style="text-align:left;">
Col11a1/Col26a1/Col14a1/Col4a6/Slc8a1/Pga5/Col16a1/Slc36a4/Slc8a3/Col9a2/Col13a1/Mme/Col5a2/Col15a1/Atp1a2/Slc7a7/Col23a1/Col18a1/Col4a5
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05410
</td>
<td style="text-align:left;">
Hypertrophic cardiomyopathy
</td>
<td style="text-align:right;">
0.1868132
</td>
<td style="text-align:right;">
0.0000722
</td>
<td style="text-align:right;">
4.141373
</td>
<td style="text-align:right;">
0.0000722
</td>
<td style="text-align:right;">
0.0008434
</td>
<td style="text-align:left;">
Cacna1c/Sgcd/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Itga1/Slc8a3/Lama2/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:right;">
0.1197772
</td>
<td style="text-align:right;">
0.0000781
</td>
<td style="text-align:right;">
4.107568
</td>
<td style="text-align:right;">
0.0000781
</td>
<td style="text-align:right;">
0.0008702
</td>
<td style="text-align:left;">
Hgf/Angpt1/Itga4/Kdr/Igf1/Col4a6/Thbs3/Nos3/Lamc3/Lama4/Ccnd2/Itgb7/Fn1/Areg/Lpar4/Tek/Gngt2/Pik3r6/Itga1/Col9a2/Flt3/Kit/Gng11/Lama2/Vegfc/Magi2/Igf2/Gng2/Itga7/Kitl/Akt3/Gnb5/Gng7/Gnb4/Pdgfrb/Il2rg/Itga8/Itgb4/F2r/Il7r/Myc/Col4a5/Irs1
</td>
<td style="text-align:right;">
43
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04970
</td>
<td style="text-align:left;">
Salivary secretion
</td>
<td style="text-align:right;">
0.1882353
</td>
<td style="text-align:right;">
0.0001066
</td>
<td style="text-align:right;">
3.972271
</td>
<td style="text-align:right;">
0.0001066
</td>
<td style="text-align:right;">
0.0011249
</td>
<td style="text-align:left;">
Adra1a/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Lyz2/Kcnma1/Plcb2/Ryr3/Lyz1/Atp2b4/Atp1a2
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:right;">
0.1808511
</td>
<td style="text-align:right;">
0.0001101
</td>
<td style="text-align:right;">
3.958300
</td>
<td style="text-align:right;">
0.0001101
</td>
<td style="text-align:right;">
0.0011249
</td>
<td style="text-align:left;">
Itga4/Cd44/Il11ra1/Cd3g/Itga1/Flt3/Cd1d1/Kit/Mme/H2-Ab1/Cd38/H2-Ob/Kitl/H2-Eb1/H2-Aa/Il7r/Cd4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04062
</td>
<td style="text-align:left;">
Chemokine signaling pathway
</td>
<td style="text-align:right;">
0.1406250
</td>
<td style="text-align:right;">
0.0001280
</td>
<td style="text-align:right;">
3.892793
</td>
<td style="text-align:right;">
0.0001280
</td>
<td style="text-align:right;">
0.0012557
</td>
<td style="text-align:left;">
Cxcl15/Gnai1/Shc3/Itk/Adcy3/Cxcr3/Cxcl12/Gngt2/Pik3r6/Plcb1/Dock2/Rasgrp2/Adcy4/Adcy7/Gng11/Gng2/Xcl1/Arrb1/Plcb2/Arrb2/Stat5b/Xcr1/Fgr/Akt3/Gnb5/Gng7/Gnb4
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04727
</td>
<td style="text-align:left;">
GABAergic synapse
</td>
<td style="text-align:right;">
0.1797753
</td>
<td style="text-align:right;">
0.0001872
</td>
<td style="text-align:right;">
3.727651
</td>
<td style="text-align:right;">
0.0001872
</td>
<td style="text-align:right;">
0.0017661
</td>
<td style="text-align:left;">
Cacna1c/Abat/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gabra4/Gng11/Gng2/Slc38a1/Gnb5/Gng7/Gnb4/Gabbr1/Cacna1a
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00514
</td>
<td style="text-align:left;">
Other types of O-glycan biosynthesis
</td>
<td style="text-align:right;">
0.2325581
</td>
<td style="text-align:right;">
0.0003602
</td>
<td style="text-align:right;">
3.443503
</td>
<td style="text-align:right;">
0.0003602
</td>
<td style="text-align:right;">
0.0032716
</td>
<td style="text-align:left;">
St6gal2/Gxylt2/B4galt2/Galnt16/Lfng/Eogt/Galntl6/Pomt2/Mfng/Galnt18
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04921
</td>
<td style="text-align:left;">
Oxytocin signaling pathway
</td>
<td style="text-align:right;">
0.1437908
</td>
<td style="text-align:right;">
0.0003857
</td>
<td style="text-align:right;">
3.413717
</td>
<td style="text-align:right;">
0.0003857
</td>
<td style="text-align:right;">
0.0033788
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Ryr1/Nos3/Gnai1/Cacna2d1/Mef2c/Gucy1a2/Adcy3/Pik3r6/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Nfatc4/Cacng7/Plcb2/Ryr3/Cacnb4/Camk1
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04330
</td>
<td style="text-align:left;">
Notch signaling pathway
</td>
<td style="text-align:right;">
0.2000000
</td>
<td style="text-align:right;">
0.0004286
</td>
<td style="text-align:right;">
3.367967
</td>
<td style="text-align:right;">
0.0004286
</td>
<td style="text-align:right;">
0.0036247
</td>
<td style="text-align:left;">
Aph1b/Lfng/Jag2/Heyl/Dtx3/Tle1/Mfng/Psen1/Maml3/Tle2/Hey2/Numbl
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04670
</td>
<td style="text-align:left;">
Leukocyte transendothelial migration
</td>
<td style="text-align:right;">
0.1525424
</td>
<td style="text-align:right;">
0.0006285
</td>
<td style="text-align:right;">
3.201671
</td>
<td style="text-align:right;">
0.0006285
</td>
<td style="text-align:right;">
0.0051385
</td>
<td style="text-align:left;">
Itga4/Gnai1/Mmp2/Icam1/Itk/Jam3/Cxcl12/Cldn8/Cdh5/Thy1/Msn/Itgal/Esam/Pecam1/Cldn5/Cldn1/Sipa1/Jam2
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04072
</td>
<td style="text-align:left;">
Phospholipase D signaling pathway
</td>
<td style="text-align:right;">
0.1409396
</td>
<td style="text-align:right;">
0.0006807
</td>
<td style="text-align:right;">
3.167044
</td>
<td style="text-align:right;">
0.0006807
</td>
<td style="text-align:right;">
0.0053855
</td>
<td style="text-align:left;">
Shc3/Pip5k1b/Lpar4/Adcy3/Ptgfr/Pik3r6/Plcb1/Plpp3/Kit/Adcy4/Adcy7/Dgkb/Agtr1a/Kitl/Plcb2/Fyn/Akt3/Plpp1/Pdgfrb/Pld2/F2r
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04310
</td>
<td style="text-align:left;">
Wnt signaling pathway
</td>
<td style="text-align:right;">
0.1345029
</td>
<td style="text-align:right;">
0.0007521
</td>
<td style="text-align:right;">
3.123698
</td>
<td style="text-align:right;">
0.0007521
</td>
<td style="text-align:right;">
0.0057648
</td>
<td style="text-align:left;">
Sfrp2/Wnt6/Lef1/Apcdd1/Wnt11/Nkd1/Ccnd2/Plcb1/Peg12/Wnt9a/Znrf3/Prickle2/Daam2/Prickle1/Nfatc4/Plcb2/Tle1/Fzd2/Psen1/Nkd2/Tle2/Ror1/Myc
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04010
</td>
<td style="text-align:left;">
MAPK signaling pathway
</td>
<td style="text-align:right;">
0.1156463
</td>
<td style="text-align:right;">
0.0008208
</td>
<td style="text-align:right;">
3.085778
</td>
<td style="text-align:right;">
0.0008208
</td>
<td style="text-align:right;">
0.0061001
</td>
<td style="text-align:left;">
Hgf/Cacna1c/Angpt1/Kdr/Igf1/Rasgrp3/Cacna2d1/Areg/Mef2c/Tek/Mapk8ip1/Cacna1e/Rasgrp2/Rps6ka2/Flt3/Kit/Vegfc/Igf2/Kitl/Rps6ka5/Arrb1/Cacng7/Flnc/Arrb2/Irak4/Cacna1h/Akt3/Pdgfrb/Map4k2/Cacna1a/Mknk2/Cacnb4/Myc/Ptpn7
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04261
</td>
<td style="text-align:left;">
Adrenergic signaling in cardiomyocytes
</td>
<td style="text-align:right;">
0.1346154
</td>
<td style="text-align:right;">
0.0012437
</td>
<td style="text-align:right;">
2.905269
</td>
<td style="text-align:right;">
0.0012437
</td>
<td style="text-align:right;">
0.0087751
</td>
<td style="text-align:left;">
Cacna1c/Adra1a/Slc8a1/Scn4b/Gnai1/Cacna2d1/Adcy3/Pik3r6/Plcb1/Slc8a3/Adcy4/Adcy7/Agtr1a/Myl3/Rps6ka5/Cacng7/Plcb2/Akt3/Atp2b4/Atp1a2/Cacnb4
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04611
</td>
<td style="text-align:left;">
Platelet activation
</td>
<td style="text-align:right;">
0.1440000
</td>
<td style="text-align:right;">
0.0012522
</td>
<td style="text-align:right;">
2.902311
</td>
<td style="text-align:right;">
0.0012522
</td>
<td style="text-align:right;">
0.0087751
</td>
<td style="text-align:left;">
Nos3/Gnai1/Gucy1a2/Adcy3/Pik3r6/Plcb1/Rasgrp2/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Tln2/Plcb2/Fyn/Akt3/Apbb1ip/Lcp2/F2r
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04658
</td>
<td style="text-align:left;">
Th1 and Th2 cell differentiation
</td>
<td style="text-align:right;">
0.1590909
</td>
<td style="text-align:right;">
0.0016263
</td>
<td style="text-align:right;">
2.788812
</td>
<td style="text-align:right;">
0.0016263
</td>
<td style="text-align:right;">
0.0110794
</td>
<td style="text-align:left;">
Cd3g/Jag2/H2-Ab1/H2-Ob/H2-Eb1/Stat5b/Nfkbie/H2-Aa/Zap70/Lat/Il12rb1/Il2rg/Maml3/Cd4
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05205
</td>
<td style="text-align:left;">
Proteoglycans in cancer
</td>
<td style="text-align:right;">
0.1219512
</td>
<td style="text-align:right;">
0.0018821
</td>
<td style="text-align:right;">
2.725347
</td>
<td style="text-align:right;">
0.0018821
</td>
<td style="text-align:right;">
0.0124762
</td>
<td style="text-align:left;">
Ihh/Hgf/Wnt6/Wnt11/Hoxd10/Kdr/Igf1/Hcls1/Cd44/Gpc3/Fn1/Mmp2/Ank2/Wnt9a/Msn/Igf2/Twist2/Flnc/Ptch1/Fzd2/Akt3/Cav1/Lum/Myc/Twist1
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05217
</td>
<td style="text-align:left;">
Basal cell carcinoma
</td>
<td style="text-align:right;">
0.1746032
</td>
<td style="text-align:right;">
0.0023593
</td>
<td style="text-align:right;">
2.627208
</td>
<td style="text-align:right;">
0.0023593
</td>
<td style="text-align:right;">
0.0152279
</td>
<td style="text-align:left;">
Gli1/Wnt6/Hhip/Lef1/Wnt11/Ptch2/Kif7/Gli3/Wnt9a/Ptch1/Fzd2
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04024
</td>
<td style="text-align:left;">
cAMP signaling pathway
</td>
<td style="text-align:right;">
0.1160714
</td>
<td style="text-align:right;">
0.0030919
</td>
<td style="text-align:right;">
2.509779
</td>
<td style="text-align:right;">
0.0030919
</td>
<td style="text-align:right;">
0.0194441
</td>
<td style="text-align:left;">
Gli1/Gria4/Cacna1c/Hhip/Vipr2/Gnai1/Grin2b/Adcy3/Edn3/Gli3/Htr1b/Adcy4/Adcy7/Ednra/Adcyap1r1/Arap3/Ptch1/Akt3/Ptger3/Atp2b4/Atp1a2/Gabbr1/Lipe/Pld2/F2r/Ptger2
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04726
</td>
<td style="text-align:left;">
Serotonergic synapse
</td>
<td style="text-align:right;">
0.1297710
</td>
<td style="text-align:right;">
0.0051167
</td>
<td style="text-align:right;">
2.291010
</td>
<td style="text-align:right;">
0.0051167
</td>
<td style="text-align:right;">
0.0310394
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Maob/Kcnd2/Gnai1/Maoa/Gngt2/Plcb1/Htr1b/Gng11/Gng2/Htr2a/Plcb2/Gnb5/Gng7/Gnb4/Cacna1a
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04730
</td>
<td style="text-align:left;">
Long-term depression
</td>
<td style="text-align:right;">
0.1666667
</td>
<td style="text-align:right;">
0.0051888
</td>
<td style="text-align:right;">
2.284935
</td>
<td style="text-align:right;">
0.0051888
</td>
<td style="text-align:right;">
0.0310394
</td>
<td style="text-align:left;">
Ryr1/Igf1/Gnai1/Gucy1a2/Plcb1/Gucy1b1/Gucy1a1/Plcb2/Gnaz/Cacna1a
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04916
</td>
<td style="text-align:left;">
Melanogenesis
</td>
<td style="text-align:right;">
0.1400000
</td>
<td style="text-align:right;">
0.0054246
</td>
<td style="text-align:right;">
2.265636
</td>
<td style="text-align:right;">
0.0054246
</td>
<td style="text-align:right;">
0.0316772
</td>
<td style="text-align:left;">
Wnt6/Lef1/Wnt11/Gnai1/Ednrb/Adcy3/Plcb1/Wnt9a/Kit/Adcy4/Adcy7/Kitl/Plcb2/Fzd2
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:right;">
0.1386139
</td>
<td style="text-align:right;">
0.0059306
</td>
<td style="text-align:right;">
2.226903
</td>
<td style="text-align:right;">
0.0059306
</td>
<td style="text-align:right;">
0.0332161
</td>
<td style="text-align:left;">
Col4a6/Nos3/Fn1/Mmp2/Icam1/Plcb1/Vegfc/Agtr1a/Nox4/Sele/Plcb2/Stat5b/Akt3/Col4a5
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu02010
</td>
<td style="text-align:left;">
ABC transporters
</td>
<td style="text-align:right;">
0.1730769
</td>
<td style="text-align:right;">
0.0060942
</td>
<td style="text-align:right;">
2.215082
</td>
<td style="text-align:right;">
0.0060942
</td>
<td style="text-align:right;">
0.0332161
</td>
<td style="text-align:left;">
Abcg2/Abca5/Abcg1/Abca8b/Abcd2/Abcb1a/Abcc5/Abcg3/Abcg4
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04672
</td>
<td style="text-align:left;">
Intestinal immune network for IgA production
</td>
<td style="text-align:right;">
0.1860465
</td>
<td style="text-align:right;">
0.0060944
</td>
<td style="text-align:right;">
2.215071
</td>
<td style="text-align:right;">
0.0060944
</td>
<td style="text-align:right;">
0.0332161
</td>
<td style="text-align:left;">
Itga4/Itgb7/Cxcl12/Tnfsf13b/H2-Ab1/H2-Ob/H2-Eb1/H2-Aa
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04925
</td>
<td style="text-align:left;">
Aldosterone synthesis and secretion
</td>
<td style="text-align:right;">
0.1372549
</td>
<td style="text-align:right;">
0.0064738
</td>
<td style="text-align:right;">
2.188842
</td>
<td style="text-align:right;">
0.0064738
</td>
<td style="text-align:right;">
0.0345169
</td>
<td style="text-align:left;">
Kcnk3/Cacna1c/Prkd1/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Atp2b4/Atp1a2/Lipe/Camk1
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04742
</td>
<td style="text-align:left;">
Taste transduction
</td>
<td style="text-align:right;">
0.1413043
</td>
<td style="text-align:right;">
0.0066924
</td>
<td style="text-align:right;">
2.174421
</td>
<td style="text-align:right;">
0.0066924
</td>
<td style="text-align:right;">
0.0349232
</td>
<td style="text-align:left;">
Cacna1c/P2rx2/Pde1b/Plcb1/Htr1b/Scn2a/Adcy4/Gabra4/Pde1a/Plcb2/Scn3a/Gabbr1/Cacna1a
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04540
</td>
<td style="text-align:left;">
Gap junction
</td>
<td style="text-align:right;">
0.1395349
</td>
<td style="text-align:right;">
0.0099204
</td>
<td style="text-align:right;">
2.003472
</td>
<td style="text-align:right;">
0.0099204
</td>
<td style="text-align:right;">
0.0506896
</td>
<td style="text-align:left;">
Gnai1/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Tubb4a/Htr2a/Plcb2/Pdgfrb
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00533
</td>
<td style="text-align:left;">
Glycosaminoglycan biosynthesis - keratan sulfate
</td>
<td style="text-align:right;">
0.2857143
</td>
<td style="text-align:right;">
0.0107649
</td>
<td style="text-align:right;">
1.967990
</td>
<td style="text-align:right;">
0.0107649
</td>
<td style="text-align:right;">
0.0532021
</td>
<td style="text-align:left;">
Chst2/B4galt2/St3gal2/Chst1
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
0.1379310
</td>
<td style="text-align:right;">
0.0108459
</td>
<td style="text-align:right;">
1.964733
</td>
<td style="text-align:right;">
0.0108459
</td>
<td style="text-align:right;">
0.0532021
</td>
<td style="text-align:left;">
Angpt1/Icam1/Tek/Cxcl12/Tnfsf13b/Itgal/H2-Ab1/H2-Ob/Atp6v0e2/H2-Eb1/H2-Aa/Atp6v1b2
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04270
</td>
<td style="text-align:left;">
Vascular smooth muscle contraction
</td>
<td style="text-align:right;">
0.1180556
</td>
<td style="text-align:right;">
0.0128851
</td>
<td style="text-align:right;">
1.889912
</td>
<td style="text-align:right;">
0.0128851
</td>
<td style="text-align:right;">
0.0619656
</td>
<td style="text-align:left;">
Cacna1c/Pla2g10/Adra1a/Gucy1a2/Adcy3/Edn3/Plcb1/Calca/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Plcb2/Calcrl
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00260
</td>
<td style="text-align:left;">
Glycine, serine and threonine metabolism
</td>
<td style="text-align:right;">
0.1750000
</td>
<td style="text-align:right;">
0.0140499
</td>
<td style="text-align:right;">
1.852327
</td>
<td style="text-align:right;">
0.0140499
</td>
<td style="text-align:right;">
0.0644879
</td>
<td style="text-align:left;">
Maob/Cbs/Maoa/Gamt/Cth/Phgdh/Sardh
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00230
</td>
<td style="text-align:left;">
Purine metabolism
</td>
<td style="text-align:right;">
0.1194030
</td>
<td style="text-align:right;">
0.0140591
</td>
<td style="text-align:right;">
1.852041
</td>
<td style="text-align:right;">
0.0140591
</td>
<td style="text-align:right;">
0.0644879
</td>
<td style="text-align:left;">
Nme4/Gucy1a2/Pde1b/Adcy3/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Enpp1/Pde1a/Hddc3/Lacc1/Pde5a/Impdh1/Pde8a/Papss2
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04390
</td>
<td style="text-align:left;">
Hippo signaling pathway
</td>
<td style="text-align:right;">
0.1146497
</td>
<td style="text-align:right;">
0.0141984
</td>
<td style="text-align:right;">
1.847760
</td>
<td style="text-align:right;">
0.0141984
</td>
<td style="text-align:right;">
0.0644879
</td>
<td style="text-align:left;">
Wnt6/Lef1/Wnt11/Nkd1/Ccnd2/Areg/Gdf7/Wnt9a/Dlg2/Pard6g/Dlg4/Fzd2/Ccn2/Dlg5/Lats2/Nkd2/Bmp7/Myc
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04723
</td>
<td style="text-align:left;">
Retrograde endocannabinoid signaling
</td>
<td style="text-align:right;">
0.1148649
</td>
<td style="text-align:right;">
0.0165797
</td>
<td style="text-align:right;">
1.780422
</td>
<td style="text-align:right;">
0.0165797
</td>
<td style="text-align:right;">
0.0739345
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Cacna1c/Gnai1/Adcy3/Gngt2/Plcb1/Adcy4/Adcy7/Gabra4/Gng11/Gng2/Plcb2/Gnb5/Gng7/Gnb4/Cacna1a
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04927
</td>
<td style="text-align:left;">
Cortisol synthesis and secretion
</td>
<td style="text-align:right;">
0.1388889
</td>
<td style="text-align:right;">
0.0183049
</td>
<td style="text-align:right;">
1.737432
</td>
<td style="text-align:right;">
0.0183049
</td>
<td style="text-align:right;">
0.0801700
</td>
<td style="text-align:left;">
Kcnk3/Cacna1c/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Pde8a
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04659
</td>
<td style="text-align:left;">
Th17 cell differentiation
</td>
<td style="text-align:right;">
0.1238095
</td>
<td style="text-align:right;">
0.0193130
</td>
<td style="text-align:right;">
1.714151
</td>
<td style="text-align:right;">
0.0193130
</td>
<td style="text-align:right;">
0.0831011
</td>
<td style="text-align:left;">
Cd3g/H2-Ab1/Il27ra/H2-Ob/H2-Eb1/Stat5b/Nfkbie/H2-Aa/Zap70/Lat/Il12rb1/Il2rg/Cd4
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04929
</td>
<td style="text-align:left;">
GnRH secretion
</td>
<td style="text-align:right;">
0.1428571
</td>
<td style="text-align:right;">
0.0207490
</td>
<td style="text-align:right;">
1.683004
</td>
<td style="text-align:right;">
0.0207490
</td>
<td style="text-align:right;">
0.0877406
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Plcb1/Arrb1/Plcb2/Arrb2/Cacna1h/Akt3/Gabbr1
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00601
</td>
<td style="text-align:left;">
Glycosphingolipid biosynthesis - lacto and neolacto series
</td>
<td style="text-align:right;">
0.1923077
</td>
<td style="text-align:right;">
0.0247723
</td>
<td style="text-align:right;">
1.606034
</td>
<td style="text-align:right;">
0.0247723
</td>
<td style="text-align:right;">
0.1029783
</td>
<td style="text-align:left;">
B4galt2/B3galnt1/Ggta1/B3gnt5/St3gal4
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05100
</td>
<td style="text-align:left;">
Bacterial invasion of epithelial cells
</td>
<td style="text-align:right;">
0.1315789
</td>
<td style="text-align:right;">
0.0258348
</td>
<td style="text-align:right;">
1.587794
</td>
<td style="text-align:right;">
0.0258348
</td>
<td style="text-align:right;">
0.1049646
</td>
<td style="text-align:left;">
Hcls1/Fn1/Shc3/Septin3/Arhgap10/Arhgef26/Elmo2/Cav1/Actr3b/Septin6
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04260
</td>
<td style="text-align:left;">
Cardiac muscle contraction
</td>
<td style="text-align:right;">
0.1264368
</td>
<td style="text-align:right;">
0.0261060
</td>
<td style="text-align:right;">
1.583260
</td>
<td style="text-align:right;">
0.0261060
</td>
<td style="text-align:right;">
0.1049646
</td>
<td style="text-align:left;">
Cacna1c/Slc8a1/Cacna2d1/Slc9a7/Slc8a3/Myl3/Cacng7/Asph/Cox4i2/Atp1a2/Cacnb4
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05416
</td>
<td style="text-align:left;">
Viral myocarditis
</td>
<td style="text-align:right;">
0.1250000
</td>
<td style="text-align:right;">
0.0281445
</td>
<td style="text-align:right;">
1.550607
</td>
<td style="text-align:right;">
0.0281445
</td>
<td style="text-align:right;">
0.1113354
</td>
<td style="text-align:left;">
Sgcd/Icam1/Lama2/Itgal/H2-Ab1/H2-Ob/H2-Eb1/Fyn/H2-Aa/Sgcb/Cav1
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04810
</td>
<td style="text-align:left;">
Regulation of actin cytoskeleton
</td>
<td style="text-align:right;">
0.0995671
</td>
<td style="text-align:right;">
0.0289684
</td>
<td style="text-align:right;">
1.538075
</td>
<td style="text-align:right;">
0.0289684
</td>
<td style="text-align:right;">
0.1127759
</td>
<td style="text-align:left;">
Itga4/Itgb7/Fn1/Pip5k1b/Pfn2/Lpar4/Cxcl12/Itga1/Msn/Abi2/Itgal/Itga7/Scin/Nckap1l/Bdkrb2/Fgd1/Akt3/Pdgfrb/Cyfip2/Actr3b/Itga8/Itgb4/F2r
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04923
</td>
<td style="text-align:left;">
Regulation of lipolysis in adipocytes
</td>
<td style="text-align:right;">
0.1403509
</td>
<td style="text-align:right;">
0.0312306
</td>
<td style="text-align:right;">
1.505419
</td>
<td style="text-align:right;">
0.0312306
</td>
<td style="text-align:right;">
0.1196831
</td>
<td style="text-align:left;">
Gnai1/Adcy3/Adcy4/Adcy7/Akt3/Ptger3/Lipe/Irs1
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00061
</td>
<td style="text-align:left;">
Fatty acid biosynthesis
</td>
<td style="text-align:right;">
0.2105263
</td>
<td style="text-align:right;">
0.0321493
</td>
<td style="text-align:right;">
1.492829
</td>
<td style="text-align:right;">
0.0321493
</td>
<td style="text-align:right;">
0.1213082
</td>
<td style="text-align:left;">
Acsl6/Acacb/Acsl1/Olah
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00565
</td>
<td style="text-align:left;">
Ether lipid metabolism
</td>
<td style="text-align:right;">
0.1458333
</td>
<td style="text-align:right;">
0.0354397
</td>
<td style="text-align:right;">
1.450510
</td>
<td style="text-align:right;">
0.0354397
</td>
<td style="text-align:right;">
0.1316978
</td>
<td style="text-align:left;">
Pla2g10/Plpp3/Lpcat1/Plpp1/Pld2/Enpp2/Pafah2
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00062
</td>
<td style="text-align:left;">
Fatty acid elongation
</td>
<td style="text-align:right;">
0.1724138
</td>
<td style="text-align:right;">
0.0380928
</td>
<td style="text-align:right;">
1.419157
</td>
<td style="text-align:right;">
0.0380928
</td>
<td style="text-align:right;">
0.1394443
</td>
<td style="text-align:left;">
Elovl4/Them5/Echs1/Acot7/Hacd1
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04961
</td>
<td style="text-align:left;">
Endocrine and other factor-regulated calcium reabsorption
</td>
<td style="text-align:right;">
0.1311475
</td>
<td style="text-align:right;">
0.0444025
</td>
<td style="text-align:right;">
1.352593
</td>
<td style="text-align:right;">
0.0444025
</td>
<td style="text-align:right;">
0.1601513
</td>
<td style="text-align:left;">
Slc8a1/Plcb1/Slc8a3/Pth1r/Bdkrb2/Plcb2/Atp2b4/Atp1a2
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu01212
</td>
<td style="text-align:left;">
Fatty acid metabolism
</td>
<td style="text-align:right;">
0.1290323
</td>
<td style="text-align:right;">
0.0481897
</td>
<td style="text-align:right;">
1.317046
</td>
<td style="text-align:right;">
0.0481897
</td>
<td style="text-align:right;">
0.1706710
</td>
<td style="text-align:left;">
Acsl6/Fads2/Fads1/Elovl4/Echs1/Cpt1c/Acsl1/Hacd1
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04350
</td>
<td style="text-align:left;">
TGF-beta signaling pathway
</td>
<td style="text-align:right;">
0.1145833
</td>
<td style="text-align:right;">
0.0487108
</td>
<td style="text-align:right;">
1.312375
</td>
<td style="text-align:right;">
0.0487108
</td>
<td style="text-align:right;">
0.1706710
</td>
<td style="text-align:left;">
Nog/Fst/Id3/Id4/Gdf7/Tgif2/Smad9/Neo1/Zfyve9/Bmp7/Myc
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04928
</td>
<td style="text-align:left;">
Parathyroid hormone synthesis, secretion and action
</td>
<td style="text-align:right;">
0.1111111
</td>
<td style="text-align:right;">
0.0496358
</td>
<td style="text-align:right;">
1.304205
</td>
<td style="text-align:right;">
0.0496358
</td>
<td style="text-align:right;">
0.1711656
</td>
<td style="text-align:left;">
Mmp16/Gnai1/Mef2c/Adcy3/Plcb1/Pth1r/Adcy4/Adcy7/Arrb1/Plcb2/Arrb2/Pld2
</td>
<td style="text-align:right;">
12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dot-plot" class="section level3">
<h3>Dot Plot</h3>
<pre class="r"><code>kegg_dot &lt;- list()
upset=list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()

  # dot plot, save
  kegg_dot[[x]] &lt;- ggplot(enrichKEGG_sig[[x]][1:15, ]) +
    geom_point(aes(x = GeneRatio, y = reorder(Description, GeneRatio), colour = logPval, size = Count)) +
    scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits = c(0, NA)) +
    scale_size(range = c(1.5,5)) +
    ggtitle(&quot;KEGG Pathways&quot;) +
    ylab(label = &quot;&quot;) +
    xlab(label = &quot;Gene Ratio&quot;) +
    labs(color = expression(&quot;-log&quot;[10] * &quot;Pvalue&quot;), size = &quot;Gene Counts&quot;)
  ggsave(filename = paste0(&quot;kegg_dot_&quot;, fc[i], &quot;.svg&quot;), plot = kegg_dot[[x]] + pub, path = here::here(&quot;2_plots/kegg/&quot;), 
         width = 250, height = 130, units = &quot;mm&quot;)
  
  upset[[x]] &lt;- upsetplot(x = enrichKEGG[[x]], 10)
  ggsave(filename = paste0(&quot;upset_kegg_&quot;, fc[i], &quot;.svg&quot;), plot = upset[[x]], path = here::here(&quot;2_plots/kegg/&quot;), 
         width = 170, height = 130, units = &quot;mm&quot;)
}

kegg_dot[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/kegg_dot-1.png" width="864" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot" class="section level3">
<h3>Upset Plot</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/upset-1.png" width="864" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="fc1.1" class="section level2 tabset">
<h2 class="tabset">FC=1.1</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="table-1" class="section level3">
<h3>Table</h3>
<pre class="r"><code>enrichKEGG_sig[[p]] %&gt;%
  kable(caption = &quot;Significantly enriched KEGG pathways&quot;) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>
Significantly enriched KEGG pathways
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logPval
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mmu04713
</td>
<td style="text-align:left;">
Circadian entrainment
</td>
<td style="text-align:right;">
0.2448980
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
9.095929
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Cacna1c/Ryr1/Gnai1/Gucy1a2/Grin2b/Adcy3/Gngt2/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Gng11/Adcy7/Adcyap1r1/Gng2/Rps6ka5/Plcb2/Ryr3/Cacna1h/Gng7/Gnb5/Gnb4
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04371
</td>
<td style="text-align:left;">
Apelin signaling pathway
</td>
<td style="text-align:right;">
0.1897810
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
7.301574
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000061
</td>
<td style="text-align:left;">
Ryr1/Nos3/Slc8a1/Gnai1/Mef2c/Adcy3/Gngt2/Pik3r6/Slc8a3/Plcb1/Adcy4/Gng11/Aplnr/Adcy7/Agtr1a/Myl3/Gng2/Plcb2/Ryr3/Ccn2/Apln/Gng7/Akt3/Gnb5/Gnb4/Lipe
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04724
</td>
<td style="text-align:left;">
Glutamatergic synapse
</td>
<td style="text-align:right;">
0.2035398
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:right;">
7.108825
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:right;">
0.0000063
</td>
<td style="text-align:left;">
Kcnj3/Slc1a3/Gria4/Grik1/Cacna1c/Gnai1/Grin2b/Adcy3/Shank2/Gngt2/Plcb1/Adcy4/Gng11/Adcy7/Grik2/Gng2/Plcb2/Slc38a1/Dlg4/Gng7/Gnb5/Gnb4/Cacna1a
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05414
</td>
<td style="text-align:left;">
Dilated cardiomyopathy
</td>
<td style="text-align:right;">
0.2127660
</td>
<td style="text-align:right;">
0.0000003
</td>
<td style="text-align:right;">
6.587888
</td>
<td style="text-align:right;">
0.0000003
</td>
<td style="text-align:right;">
0.0000157
</td>
<td style="text-align:left;">
Sgcd/Cacna1c/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Adcy3/Slc8a3/Itga1/Adcy4/Lama2/Adcy7/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04514
</td>
<td style="text-align:left;">
Cell adhesion molecules
</td>
<td style="text-align:right;">
0.1593407
</td>
<td style="text-align:right;">
0.0000004
</td>
<td style="text-align:right;">
6.353176
</td>
<td style="text-align:right;">
0.0000004
</td>
<td style="text-align:right;">
0.0000216
</td>
<td style="text-align:left;">
Itga4/Lrrc4c/Ptprd/Nlgn1/Cdh2/Lrrc4b/Itgb7/Icam1/Jam3/Cldn8/Slitrk3/Cdh5/Ptprm/Cd226/Nlgn3/Esam/Itgal/H2-Ab1/Nfasc/Sele/H2-Ob/Pecam1/H2-Eb1/Cldn5/Neo1/H2-Aa/Ntng1/Itga8/Cldn1
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05412
</td>
<td style="text-align:left;">
Arrhythmogenic right ventricular cardiomyopathy
</td>
<td style="text-align:right;">
0.2207792
</td>
<td style="text-align:right;">
0.0000012
</td>
<td style="text-align:right;">
5.925876
</td>
<td style="text-align:right;">
0.0000012
</td>
<td style="text-align:right;">
0.0000481
</td>
<td style="text-align:left;">
Lef1/Sgcd/Cacna1c/Itga4/Slc8a1/Cdh2/Itgb7/Cacna2d1/Slc8a3/Itga1/Lama2/Itga7/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04022
</td>
<td style="text-align:left;">
cGMP-PKG signaling pathway
</td>
<td style="text-align:right;">
0.1560694
</td>
<td style="text-align:right;">
0.0000017
</td>
<td style="text-align:right;">
5.773851
</td>
<td style="text-align:right;">
0.0000017
</td>
<td style="text-align:right;">
0.0000585
</td>
<td style="text-align:left;">
Cacna1c/Adra1a/Nos3/Slc8a1/Gnai1/Ednrb/Mef2c/Gucy1a2/Adcy3/Pik3r6/Slc8a3/Plcb1/Gucy1b1/Trpc6/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Bdkrb2/Plcb2/Pde5a/Nfatc4/Atp2b4/Akt3/Atp1a2
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05032
</td>
<td style="text-align:left;">
Morphine addiction
</td>
<td style="text-align:right;">
0.1978022
</td>
<td style="text-align:right;">
0.0000031
</td>
<td style="text-align:right;">
5.503107
</td>
<td style="text-align:right;">
0.0000031
</td>
<td style="text-align:right;">
0.0000919
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Pde1b/Adcy3/Gngt2/Gabra4/Adcy4/Gng11/Adcy7/Pde1a/Gng2/Arrb1/Arrb2/Gng7/Gnb5/Gnb4/Gabbr1/Cacna1a
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04340
</td>
<td style="text-align:left;">
Hedgehog signaling pathway
</td>
<td style="text-align:right;">
0.2413793
</td>
<td style="text-align:right;">
0.0000034
</td>
<td style="text-align:right;">
5.468399
</td>
<td style="text-align:right;">
0.0000034
</td>
<td style="text-align:right;">
0.0000919
</td>
<td style="text-align:left;">
Gli1/Ihh/Hhip/Evc2/Ptch2/Evc/Kif7/Ccnd2/Gli3/Lrp2/Gpr161/Arrb1/Ptch1/Arrb2
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04725
</td>
<td style="text-align:left;">
Cholinergic synapse
</td>
<td style="text-align:right;">
0.1785714
</td>
<td style="text-align:right;">
0.0000047
</td>
<td style="text-align:right;">
5.324639
</td>
<td style="text-align:right;">
0.0000047
</td>
<td style="text-align:right;">
0.0001151
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Gnai1/Kcnq5/Adcy3/Gngt2/Pik3r6/Plcb1/Adcy4/Gng11/Adcy7/Gng2/Plcb2/Fyn/Gng7/Akt3/Gnb5/Gnb4/Kcnq4/Cacna1a
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:right;">
0.1931818
</td>
<td style="text-align:right;">
0.0000082
</td>
<td style="text-align:right;">
5.084669
</td>
<td style="text-align:right;">
0.0000082
</td>
<td style="text-align:right;">
0.0001819
</td>
<td style="text-align:left;">
Itga4/Col4a6/Npnt/Thbs3/Lamc3/Cd44/Lama4/Itgb7/Fn1/Itga1/Col9a2/Frem1/Lama2/Itga7/Sv2c/Itga8/Itgb4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04926
</td>
<td style="text-align:left;">
Relaxin signaling pathway
</td>
<td style="text-align:right;">
0.1627907
</td>
<td style="text-align:right;">
0.0000124
</td>
<td style="text-align:right;">
4.906907
</td>
<td style="text-align:right;">
0.0000124
</td>
<td style="text-align:right;">
0.0002129
</td>
<td style="text-align:left;">
Col4a6/Nos3/Gnai1/Ednrb/Mmp2/Shc3/Adcy3/Gngt2/Plcb1/Adcy4/Gng11/Adcy7/Vegfc/Gng2/Arrb1/Plcb2/Arrb2/Gng7/Akt3/Gnb5/Gnb4
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04015
</td>
<td style="text-align:left;">
Rap1 signaling pathway
</td>
<td style="text-align:right;">
0.1355140
</td>
<td style="text-align:right;">
0.0000125
</td>
<td style="text-align:right;">
4.904430
</td>
<td style="text-align:right;">
0.0000125
</td>
<td style="text-align:right;">
0.0002129
</td>
<td style="text-align:left;">
Hgf/Angpt1/Kdr/Igf1/Evl/Rasgrp3/Gnai1/Pfn2/Grin2b/Lpar4/Prkd1/Tek/Adcy3/Plcb1/Rasgrp2/Adcy4/Kit/Adcy7/Itgal/Vegfc/Magi2/Arap3/Pard6g/Kitl/Plcb2/Tln2/Akt3/Lat/Apbb1ip
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04062
</td>
<td style="text-align:left;">
Chemokine signaling pathway
</td>
<td style="text-align:right;">
0.1406250
</td>
<td style="text-align:right;">
0.0000126
</td>
<td style="text-align:right;">
4.901255
</td>
<td style="text-align:right;">
0.0000126
</td>
<td style="text-align:right;">
0.0002129
</td>
<td style="text-align:left;">
Cxcl15/Gnai1/Shc3/Itk/Cxcr3/Cxcl12/Adcy3/Gngt2/Pik3r6/Plcb1/Rasgrp2/Dock2/Adcy4/Gng11/Adcy7/Xcl1/Gng2/Arrb1/Plcb2/Xcr1/Arrb2/Gng7/Fgr/Stat5b/Akt3/Gnb5/Gnb4
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05410
</td>
<td style="text-align:left;">
Hypertrophic cardiomyopathy
</td>
<td style="text-align:right;">
0.1868132
</td>
<td style="text-align:right;">
0.0000131
</td>
<td style="text-align:right;">
4.881540
</td>
<td style="text-align:right;">
0.0000131
</td>
<td style="text-align:right;">
0.0002129
</td>
<td style="text-align:left;">
Sgcd/Cacna1c/Itga4/Igf1/Slc8a1/Itgb7/Cacna2d1/Slc8a3/Itga1/Lama2/Itga7/Myl3/Cacng7/Sgcb/Itga8/Itgb4/Cacnb4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04020
</td>
<td style="text-align:left;">
Calcium signaling pathway
</td>
<td style="text-align:right;">
0.1291667
</td>
<td style="text-align:right;">
0.0000167
</td>
<td style="text-align:right;">
4.776965
</td>
<td style="text-align:right;">
0.0000167
</td>
<td style="text-align:right;">
0.0002540
</td>
<td style="text-align:left;">
Hgf/Cacna1c/Ryr1/Kdr/Adra1a/Nos3/Slc8a1/P2rx2/Ednrb/Pde1b/Adcy3/Cacna1e/Ptgfr/Slc8a3/Plcb1/Adcy4/Adcy7/Vegfc/Pde1a/Cd38/Agtr1a/Ednra/Bdkrb2/Htr2a/Plcb2/Asph/Ryr3/Cacna1h/Ptger3/Atp2b4/Cacna1a
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04970
</td>
<td style="text-align:left;">
Salivary secretion
</td>
<td style="text-align:right;">
0.1882353
</td>
<td style="text-align:right;">
0.0000213
</td>
<td style="text-align:right;">
4.671419
</td>
<td style="text-align:right;">
0.0000213
</td>
<td style="text-align:right;">
0.0002884
</td>
<td style="text-align:left;">
Adra1a/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Lyz2/Kcnma1/Plcb2/Ryr3/Lyz1/Atp2b4/Atp1a2
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04924
</td>
<td style="text-align:left;">
Renin secretion
</td>
<td style="text-align:right;">
0.1973684
</td>
<td style="text-align:right;">
0.0000213
</td>
<td style="text-align:right;">
4.670649
</td>
<td style="text-align:right;">
0.0000213
</td>
<td style="text-align:right;">
0.0002884
</td>
<td style="text-align:left;">
Cacna1c/Gnai1/Gucy1a2/Pde1b/Edn3/Plcb1/Gucy1b1/Gucy1a1/Pde1a/Agtr1a/Kcnma1/Ednra/Adcyap1r1/Plcb2/Clca2
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04728
</td>
<td style="text-align:left;">
Dopaminergic synapse
</td>
<td style="text-align:right;">
0.1555556
</td>
<td style="text-align:right;">
0.0000253
</td>
<td style="text-align:right;">
4.597276
</td>
<td style="text-align:right;">
0.0000253
</td>
<td style="text-align:right;">
0.0003235
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Maob/Cacna1c/Drd4/Gnai1/Scn1a/Grin2b/Maoa/Gngt2/Plcb1/Gng11/Gng2/Arrb1/Plcb2/Arrb2/Gng7/Akt3/Gnb5/Gnb4/Cacna1a
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04510
</td>
<td style="text-align:left;">
Focal adhesion
</td>
<td style="text-align:right;">
0.1343284
</td>
<td style="text-align:right;">
0.0000291
</td>
<td style="text-align:right;">
4.536132
</td>
<td style="text-align:right;">
0.0000291
</td>
<td style="text-align:right;">
0.0003538
</td>
<td style="text-align:left;">
Hgf/Itga4/Kdr/Igf1/Col4a6/Thbs3/Lamc3/Lama4/Itgb7/Ccnd2/Fn1/Shc3/Pip5k1b/Itga1/Col9a2/Lama2/Vegfc/Parvg/Itga7/Parvb/Flnc/Tln2/Fyn/Akt3/Cav1/Itga8/Itgb4
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04360
</td>
<td style="text-align:left;">
Axon guidance
</td>
<td style="text-align:right;">
0.1381215
</td>
<td style="text-align:right;">
0.0000358
</td>
<td style="text-align:right;">
4.446679
</td>
<td style="text-align:right;">
0.0000358
</td>
<td style="text-align:right;">
0.0004084
</td>
<td style="text-align:left;">
Epha8/Efnb3/Sema5a/Lrrc4c/Dcc/Plxnb1/Sema5b/Sema7a/Nrp1/Gnai1/Cxcl12/Gdf7/Trpc6/Epha3/Sema3g/Pard6g/Epha7/Ptch1/Nfatc4/Neo1/Fyn/Unc5c/Sema6a/Ngef/Ntng1
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04974
</td>
<td style="text-align:left;">
Protein digestion and absorption
</td>
<td style="text-align:right;">
0.1666667
</td>
<td style="text-align:right;">
0.0000372
</td>
<td style="text-align:right;">
4.429015
</td>
<td style="text-align:right;">
0.0000372
</td>
<td style="text-align:right;">
0.0004084
</td>
<td style="text-align:left;">
Col11a1/Col26a1/Col14a1/Col4a6/Slc8a1/Pga5/Col16a1/Slc36a4/Slc8a3/Col9a2/Col13a1/Mme/Col5a2/Col15a1/Atp1a2/Col23a1/Slc7a7/Col18a1
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04727
</td>
<td style="text-align:left;">
GABAergic synapse
</td>
<td style="text-align:right;">
0.1797753
</td>
<td style="text-align:right;">
0.0000386
</td>
<td style="text-align:right;">
4.413099
</td>
<td style="text-align:right;">
0.0000386
</td>
<td style="text-align:right;">
0.0004084
</td>
<td style="text-align:left;">
Cacna1c/Abat/Gnai1/Adcy3/Gngt2/Gabra4/Adcy4/Gng11/Adcy7/Gng2/Slc38a1/Gng7/Gnb5/Gnb4/Gabbr1/Cacna1a
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04014
</td>
<td style="text-align:left;">
Ras signaling pathway
</td>
<td style="text-align:right;">
0.1234043
</td>
<td style="text-align:right;">
0.0000728
</td>
<td style="text-align:right;">
4.137934
</td>
<td style="text-align:right;">
0.0000728
</td>
<td style="text-align:right;">
0.0007375
</td>
<td style="text-align:left;">
Hgf/Angpt1/Pla2g10/Kdr/Igf1/Rasgrp3/Ets1/Shc3/Rasa3/Grin2b/Tek/Gngt2/Rasgrp2/Flt3/Gng11/Kit/Vegfc/Igf2/Gng2/Pla1a/Kitl/Gng7/Ets2/Akt3/Zap70/Gnb5/Lat/Syngap1/Gnb4
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04921
</td>
<td style="text-align:left;">
Oxytocin signaling pathway
</td>
<td style="text-align:right;">
0.1372549
</td>
<td style="text-align:right;">
0.0001631
</td>
<td style="text-align:right;">
3.787545
</td>
<td style="text-align:right;">
0.0001631
</td>
<td style="text-align:right;">
0.0015864
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Ryr1/Nos3/Gnai1/Cacna2d1/Mef2c/Gucy1a2/Adcy3/Pik3r6/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Cd38/Cacng7/Plcb2/Ryr3/Nfatc4/Cacnb4
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04261
</td>
<td style="text-align:left;">
Adrenergic signaling in cardiomyocytes
</td>
<td style="text-align:right;">
0.1346154
</td>
<td style="text-align:right;">
0.0002149
</td>
<td style="text-align:right;">
3.667725
</td>
<td style="text-align:right;">
0.0002149
</td>
<td style="text-align:right;">
0.0020100
</td>
<td style="text-align:left;">
Cacna1c/Adra1a/Slc8a1/Scn4b/Gnai1/Cacna2d1/Adcy3/Pik3r6/Slc8a3/Plcb1/Adcy4/Adcy7/Agtr1a/Myl3/Rps6ka5/Cacng7/Plcb2/Atp2b4/Akt3/Atp1a2/Cacnb4
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04640
</td>
<td style="text-align:left;">
Hematopoietic cell lineage
</td>
<td style="text-align:right;">
0.1595745
</td>
<td style="text-align:right;">
0.0002688
</td>
<td style="text-align:right;">
3.570545
</td>
<td style="text-align:right;">
0.0002688
</td>
<td style="text-align:right;">
0.0024209
</td>
<td style="text-align:left;">
Itga4/Cd44/Il11ra1/Cd3g/Itga1/Flt3/Mme/Cd1d1/Kit/H2-Ab1/Cd38/H2-Ob/Kitl/H2-Eb1/H2-Aa
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:right;">
0.1030641
</td>
<td style="text-align:right;">
0.0003520
</td>
<td style="text-align:right;">
3.453452
</td>
<td style="text-align:right;">
0.0003520
</td>
<td style="text-align:right;">
0.0030569
</td>
<td style="text-align:left;">
Hgf/Angpt1/Itga4/Kdr/Igf1/Col4a6/Thbs3/Nos3/Lamc3/Lama4/Areg/Itgb7/Ccnd2/Fn1/Lpar4/Tek/Gngt2/Pik3r6/Itga1/Col9a2/Flt3/Gng11/Kit/Lama2/Vegfc/Magi2/Igf2/Itga7/Gng2/Kitl/Gng7/Akt3/Gnb5/Gnb4/Il2rg/Itga8/Itgb4
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05217
</td>
<td style="text-align:left;">
Basal cell carcinoma
</td>
<td style="text-align:right;">
0.1746032
</td>
<td style="text-align:right;">
0.0008023
</td>
<td style="text-align:right;">
3.095668
</td>
<td style="text-align:right;">
0.0008023
</td>
<td style="text-align:right;">
0.0067270
</td>
<td style="text-align:left;">
Gli1/Wnt6/Hhip/Lef1/Wnt11/Ptch2/Kif7/Gli3/Wnt9a/Ptch1/Fzd2
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04670
</td>
<td style="text-align:left;">
Leukocyte transendothelial migration
</td>
<td style="text-align:right;">
0.1355932
</td>
<td style="text-align:right;">
0.0010931
</td>
<td style="text-align:right;">
2.961322
</td>
<td style="text-align:right;">
0.0010931
</td>
<td style="text-align:right;">
0.0088602
</td>
<td style="text-align:left;">
Itga4/Gnai1/Mmp2/Icam1/Itk/Cxcl12/Jam3/Cldn8/Cdh5/Thy1/Msn/Esam/Itgal/Pecam1/Cldn5/Cldn1
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04726
</td>
<td style="text-align:left;">
Serotonergic synapse
</td>
<td style="text-align:right;">
0.1297710
</td>
<td style="text-align:right;">
0.0012723
</td>
<td style="text-align:right;">
2.895414
</td>
<td style="text-align:right;">
0.0012723
</td>
<td style="text-align:right;">
0.0099796
</td>
<td style="text-align:left;">
Kcnj3/Maob/Cacna1c/Kcnd2/Gnai1/Maoa/Gngt2/Plcb1/Htr1b/Gng11/Htr2a/Gng2/Plcb2/Gng7/Gnb5/Gnb4/Cacna1a
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04916
</td>
<td style="text-align:left;">
Melanogenesis
</td>
<td style="text-align:right;">
0.1400000
</td>
<td style="text-align:right;">
0.0016114
</td>
<td style="text-align:right;">
2.792795
</td>
<td style="text-align:right;">
0.0016114
</td>
<td style="text-align:right;">
0.0122446
</td>
<td style="text-align:left;">
Wnt6/Lef1/Wnt11/Gnai1/Ednrb/Adcy3/Plcb1/Wnt9a/Adcy4/Kit/Adcy7/Kitl/Plcb2/Fzd2
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04730
</td>
<td style="text-align:left;">
Long-term depression
</td>
<td style="text-align:right;">
0.1666667
</td>
<td style="text-align:right;">
0.0019805
</td>
<td style="text-align:right;">
2.703220
</td>
<td style="text-align:right;">
0.0019805
</td>
<td style="text-align:right;">
0.0144529
</td>
<td style="text-align:left;">
Ryr1/Igf1/Gnai1/Gucy1a2/Plcb1/Gucy1b1/Gucy1a1/Plcb2/Gnaz/Cacna1a
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04611
</td>
<td style="text-align:left;">
Platelet activation
</td>
<td style="text-align:right;">
0.1280000
</td>
<td style="text-align:right;">
0.0020209
</td>
<td style="text-align:right;">
2.694455
</td>
<td style="text-align:right;">
0.0020209
</td>
<td style="text-align:right;">
0.0144529
</td>
<td style="text-align:left;">
Nos3/Gnai1/Gucy1a2/Adcy3/Pik3r6/Plcb1/Rasgrp2/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Plcb2/Tln2/Fyn/Akt3/Apbb1ip
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04072
</td>
<td style="text-align:left;">
Phospholipase D signaling pathway
</td>
<td style="text-align:right;">
0.1208054
</td>
<td style="text-align:right;">
0.0021127
</td>
<td style="text-align:right;">
2.675165
</td>
<td style="text-align:right;">
0.0021127
</td>
<td style="text-align:right;">
0.0145056
</td>
<td style="text-align:left;">
Shc3/Lpar4/Pip5k1b/Adcy3/Ptgfr/Pik3r6/Plcb1/Plpp3/Adcy4/Kit/Adcy7/Dgkb/Agtr1a/Kitl/Plcb2/Fyn/Akt3/Plpp1
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04742
</td>
<td style="text-align:left;">
Taste transduction
</td>
<td style="text-align:right;">
0.1413043
</td>
<td style="text-align:right;">
0.0021476
</td>
<td style="text-align:right;">
2.668049
</td>
<td style="text-align:right;">
0.0021476
</td>
<td style="text-align:right;">
0.0145056
</td>
<td style="text-align:left;">
Cacna1c/P2rx2/Pde1b/Plcb1/Htr1b/Gabra4/Scn2a/Adcy4/Pde1a/Plcb2/Scn3a/Gabbr1/Cacna1a
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00514
</td>
<td style="text-align:left;">
Other types of O-glycan biosynthesis
</td>
<td style="text-align:right;">
0.1860465
</td>
<td style="text-align:right;">
0.0027027
</td>
<td style="text-align:right;">
2.568200
</td>
<td style="text-align:right;">
0.0027027
</td>
<td style="text-align:right;">
0.0172944
</td>
<td style="text-align:left;">
St6gal2/Gxylt2/B4galt2/Galnt16/Lfng/Eogt/Galntl6/Mfng
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04672
</td>
<td style="text-align:left;">
Intestinal immune network for IgA production
</td>
<td style="text-align:right;">
0.1860465
</td>
<td style="text-align:right;">
0.0027027
</td>
<td style="text-align:right;">
2.568200
</td>
<td style="text-align:right;">
0.0027027
</td>
<td style="text-align:right;">
0.0172944
</td>
<td style="text-align:left;">
Itga4/Itgb7/Cxcl12/Tnfsf13b/H2-Ab1/H2-Ob/H2-Eb1/H2-Aa
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04010
</td>
<td style="text-align:left;">
MAPK signaling pathway
</td>
<td style="text-align:right;">
0.0986395
</td>
<td style="text-align:right;">
0.0029078
</td>
<td style="text-align:right;">
2.536442
</td>
<td style="text-align:right;">
0.0029078
</td>
<td style="text-align:right;">
0.0181293
</td>
<td style="text-align:left;">
Hgf/Angpt1/Cacna1c/Kdr/Igf1/Rasgrp3/Areg/Cacna2d1/Mef2c/Tek/Mapk8ip1/Cacna1e/Rasgrp2/Rps6ka2/Flt3/Kit/Vegfc/Igf2/Rps6ka5/Kitl/Arrb1/Cacng7/Flnc/Irak4/Cacna1h/Arrb2/Akt3/Cacna1a/Cacnb4
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05205
</td>
<td style="text-align:left;">
Proteoglycans in cancer
</td>
<td style="text-align:right;">
0.1073171
</td>
<td style="text-align:right;">
0.0032991
</td>
<td style="text-align:right;">
2.481605
</td>
<td style="text-align:right;">
0.0032991
</td>
<td style="text-align:right;">
0.0200551
</td>
<td style="text-align:left;">
Ihh/Hgf/Wnt6/Wnt11/Hoxd10/Kdr/Igf1/Cd44/Hcls1/Gpc3/Fn1/Mmp2/Ank2/Wnt9a/Msn/Igf2/Twist2/Ptch1/Flnc/Fzd2/Akt3/Cav1
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04270
</td>
<td style="text-align:left;">
Vascular smooth muscle contraction
</td>
<td style="text-align:right;">
0.1180556
</td>
<td style="text-align:right;">
0.0035371
</td>
<td style="text-align:right;">
2.451354
</td>
<td style="text-align:right;">
0.0035371
</td>
<td style="text-align:right;">
0.0209774
</td>
<td style="text-align:left;">
Cacna1c/Pla2g10/Adra1a/Gucy1a2/Adcy3/Edn3/Calca/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Agtr1a/Kcnma1/Ednra/Plcb2/Calcrl
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04310
</td>
<td style="text-align:left;">
Wnt signaling pathway
</td>
<td style="text-align:right;">
0.1111111
</td>
<td style="text-align:right;">
0.0041787
</td>
<td style="text-align:right;">
2.378956
</td>
<td style="text-align:right;">
0.0041787
</td>
<td style="text-align:right;">
0.0239016
</td>
<td style="text-align:left;">
Sfrp2/Wnt6/Lef1/Apcdd1/Wnt11/Nkd1/Ccnd2/Plcb1/Peg12/Wnt9a/Prickle2/Znrf3/Daam2/Prickle1/Tle1/Plcb2/Nfatc4/Fzd2/Nkd2
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04658
</td>
<td style="text-align:left;">
Th1 and Th2 cell differentiation
</td>
<td style="text-align:right;">
0.1363636
</td>
<td style="text-align:right;">
0.0042268
</td>
<td style="text-align:right;">
2.373993
</td>
<td style="text-align:right;">
0.0042268
</td>
<td style="text-align:right;">
0.0239016
</td>
<td style="text-align:left;">
Cd3g/Jag2/H2-Ab1/H2-Ob/H2-Eb1/Nfkbie/H2-Aa/Stat5b/Zap70/Lat/Il12rb1/Il2rg
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04024
</td>
<td style="text-align:left;">
cAMP signaling pathway
</td>
<td style="text-align:right;">
0.1026786
</td>
<td style="text-align:right;">
0.0046820
</td>
<td style="text-align:right;">
2.329568
</td>
<td style="text-align:right;">
0.0046820
</td>
<td style="text-align:right;">
0.0253449
</td>
<td style="text-align:left;">
Gli1/Gria4/Hhip/Cacna1c/Vipr2/Gnai1/Grin2b/Adcy3/Edn3/Gli3/Htr1b/Adcy4/Adcy7/Ednra/Adcyap1r1/Arap3/Ptch1/Ptger3/Atp2b4/Akt3/Atp1a2/Gabbr1/Lipe
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04723
</td>
<td style="text-align:left;">
Retrograde endocannabinoid signaling
</td>
<td style="text-align:right;">
0.1148649
</td>
<td style="text-align:right;">
0.0046905
</td>
<td style="text-align:right;">
2.328785
</td>
<td style="text-align:right;">
0.0046905
</td>
<td style="text-align:right;">
0.0253449
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Cacna1c/Gnai1/Adcy3/Gngt2/Plcb1/Gabra4/Adcy4/Gng11/Adcy7/Gng2/Plcb2/Gng7/Gnb5/Gnb4/Cacna1a
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04933
</td>
<td style="text-align:left;">
AGE-RAGE signaling pathway in diabetic complications
</td>
<td style="text-align:right;">
0.1287129
</td>
<td style="text-align:right;">
0.0049110
</td>
<td style="text-align:right;">
2.308831
</td>
<td style="text-align:right;">
0.0049110
</td>
<td style="text-align:right;">
0.0259597
</td>
<td style="text-align:left;">
Col4a6/Nos3/Fn1/Mmp2/Icam1/Plcb1/Vegfc/Nox4/Sele/Agtr1a/Plcb2/Stat5b/Akt3
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04925
</td>
<td style="text-align:left;">
Aldosterone synthesis and secretion
</td>
<td style="text-align:right;">
0.1274510
</td>
<td style="text-align:right;">
0.0053431
</td>
<td style="text-align:right;">
2.272203
</td>
<td style="text-align:right;">
0.0053431
</td>
<td style="text-align:right;">
0.0276432
</td>
<td style="text-align:left;">
Kcnk3/Cacna1c/Prkd1/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Atp2b4/Atp1a2/Lipe
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00533
</td>
<td style="text-align:left;">
Glycosaminoglycan biosynthesis - keratan sulfate
</td>
<td style="text-align:right;">
0.2857143
</td>
<td style="text-align:right;">
0.0067409
</td>
<td style="text-align:right;">
2.171279
</td>
<td style="text-align:right;">
0.0067409
</td>
<td style="text-align:right;">
0.0341482
</td>
<td style="text-align:left;">
Chst2/B4galt2/St3gal2/Chst1
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu02010
</td>
<td style="text-align:left;">
ABC transporters
</td>
<td style="text-align:right;">
0.1538462
</td>
<td style="text-align:right;">
0.0089348
</td>
<td style="text-align:right;">
2.048914
</td>
<td style="text-align:right;">
0.0089348
</td>
<td style="text-align:right;">
0.0443382
</td>
<td style="text-align:left;">
Abcg2/Abca5/Abcg1/Abca8b/Abcd2/Abcb1a/Abcc5/Abcg3
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04929
</td>
<td style="text-align:left;">
GnRH secretion
</td>
<td style="text-align:right;">
0.1428571
</td>
<td style="text-align:right;">
0.0092723
</td>
<td style="text-align:right;">
2.032814
</td>
<td style="text-align:right;">
0.0092723
</td>
<td style="text-align:right;">
0.0450925
</td>
<td style="text-align:left;">
Kcnj3/Cacna1c/Plcb1/Arrb1/Plcb2/Cacna1h/Arrb2/Akt3/Gabbr1
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04540
</td>
<td style="text-align:left;">
Gap junction
</td>
<td style="text-align:right;">
0.1279070
</td>
<td style="text-align:right;">
0.0097385
</td>
<td style="text-align:right;">
2.011509
</td>
<td style="text-align:right;">
0.0097385
</td>
<td style="text-align:right;">
0.0464312
</td>
<td style="text-align:left;">
Gnai1/Gucy1a2/Adcy3/Plcb1/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Tubb4a/Htr2a/Plcb2
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04260
</td>
<td style="text-align:left;">
Cardiac muscle contraction
</td>
<td style="text-align:right;">
0.1264368
</td>
<td style="text-align:right;">
0.0105916
</td>
<td style="text-align:right;">
1.975040
</td>
<td style="text-align:right;">
0.0105916
</td>
<td style="text-align:right;">
0.0485929
</td>
<td style="text-align:left;">
Cacna1c/Slc8a1/Cacna2d1/Slc9a7/Slc8a3/Myl3/Cacng7/Asph/Cox4i2/Atp1a2/Cacnb4
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05323
</td>
<td style="text-align:left;">
Rheumatoid arthritis
</td>
<td style="text-align:right;">
0.1264368
</td>
<td style="text-align:right;">
0.0105916
</td>
<td style="text-align:right;">
1.975040
</td>
<td style="text-align:right;">
0.0105916
</td>
<td style="text-align:right;">
0.0485929
</td>
<td style="text-align:left;">
Angpt1/Icam1/Tek/Cxcl12/Tnfsf13b/Itgal/H2-Ab1/H2-Ob/Atp6v0e2/H2-Eb1/H2-Aa
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05416
</td>
<td style="text-align:left;">
Viral myocarditis
</td>
<td style="text-align:right;">
0.1250000
</td>
<td style="text-align:right;">
0.0115008
</td>
<td style="text-align:right;">
1.939271
</td>
<td style="text-align:right;">
0.0115008
</td>
<td style="text-align:right;">
0.0517873
</td>
<td style="text-align:left;">
Sgcd/Icam1/Lama2/Itgal/H2-Ab1/H2-Ob/H2-Eb1/Fyn/H2-Aa/Sgcb/Cav1
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00601
</td>
<td style="text-align:left;">
Glycosphingolipid biosynthesis - lacto and neolacto series
</td>
<td style="text-align:right;">
0.1923077
</td>
<td style="text-align:right;">
0.0146653
</td>
<td style="text-align:right;">
1.833709
</td>
<td style="text-align:right;">
0.0146653
</td>
<td style="text-align:right;">
0.0648361
</td>
<td style="text-align:left;">
B3galnt1/B4galt2/Ggta1/B3gnt5/St3gal4
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04659
</td>
<td style="text-align:left;">
Th17 cell differentiation
</td>
<td style="text-align:right;">
0.1142857
</td>
<td style="text-align:right;">
0.0167183
</td>
<td style="text-align:right;">
1.776807
</td>
<td style="text-align:right;">
0.0167183
</td>
<td style="text-align:right;">
0.0725928
</td>
<td style="text-align:left;">
Cd3g/Il27ra/H2-Ab1/H2-Ob/H2-Eb1/Nfkbie/H2-Aa/Stat5b/Zap70/Lat/Il12rb1/Il2rg
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04390
</td>
<td style="text-align:left;">
Hippo signaling pathway
</td>
<td style="text-align:right;">
0.1019108
</td>
<td style="text-align:right;">
0.0178407
</td>
<td style="text-align:right;">
1.748587
</td>
<td style="text-align:right;">
0.0178407
</td>
<td style="text-align:right;">
0.0761073
</td>
<td style="text-align:left;">
Wnt6/Lef1/Wnt11/Nkd1/Areg/Ccnd2/Gdf7/Wnt9a/Dlg2/Pard6g/Ccn2/Fzd2/Dlg4/Dlg5/Lats2/Nkd2
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00230
</td>
<td style="text-align:left;">
Purine metabolism
</td>
<td style="text-align:right;">
0.1044776
</td>
<td style="text-align:right;">
0.0211764
</td>
<td style="text-align:right;">
1.674148
</td>
<td style="text-align:right;">
0.0211764
</td>
<td style="text-align:right;">
0.0874342
</td>
<td style="text-align:left;">
Nme4/Gucy1a2/Pde1b/Adcy3/Gucy1b1/Gucy1a1/Adcy4/Adcy7/Pde1a/Enpp1/Hddc3/Lacc1/Pde5a/Impdh1
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04927
</td>
<td style="text-align:left;">
Cortisol synthesis and secretion
</td>
<td style="text-align:right;">
0.1250000
</td>
<td style="text-align:right;">
0.0212151
</td>
<td style="text-align:right;">
1.673355
</td>
<td style="text-align:right;">
0.0212151
</td>
<td style="text-align:right;">
0.0874342
</td>
<td style="text-align:left;">
Kcnk3/Cacna1c/Adcy3/Plcb1/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04961
</td>
<td style="text-align:left;">
Endocrine and other factor-regulated calcium reabsorption
</td>
<td style="text-align:right;">
0.1311475
</td>
<td style="text-align:right;">
0.0223555
</td>
<td style="text-align:right;">
1.650616
</td>
<td style="text-align:right;">
0.0223555
</td>
<td style="text-align:right;">
0.0905986
</td>
<td style="text-align:left;">
Slc8a1/Slc8a3/Plcb1/Pth1r/Bdkrb2/Plcb2/Atp2b4/Atp1a2
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04810
</td>
<td style="text-align:left;">
Regulation of actin cytoskeleton
</td>
<td style="text-align:right;">
0.0909091
</td>
<td style="text-align:right;">
0.0241445
</td>
<td style="text-align:right;">
1.617182
</td>
<td style="text-align:right;">
0.0241445
</td>
<td style="text-align:right;">
0.0962447
</td>
<td style="text-align:left;">
Itga4/Itgb7/Fn1/Pfn2/Lpar4/Pip5k1b/Cxcl12/Itga1/Msn/Abi2/Itgal/Scin/Bdkrb2/Itga7/Nckap1l/Fgd1/Akt3/Actr3b/Cyfip2/Itga8/Itgb4
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04918
</td>
<td style="text-align:left;">
Thyroid hormone synthesis
</td>
<td style="text-align:right;">
0.1216216
</td>
<td style="text-align:right;">
0.0249269
</td>
<td style="text-align:right;">
1.603331
</td>
<td style="text-align:right;">
0.0249269
</td>
<td style="text-align:right;">
0.0970532
</td>
<td style="text-align:left;">
Ttr/Adcy3/Plcb1/Lrp2/Adcy4/Adcy7/Plcb2/Gpx7/Atp1a2
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00260
</td>
<td style="text-align:left;">
Glycine, serine and threonine metabolism
</td>
<td style="text-align:right;">
0.1500000
</td>
<td style="text-align:right;">
0.0251456
</td>
<td style="text-align:right;">
1.599538
</td>
<td style="text-align:right;">
0.0251456
</td>
<td style="text-align:right;">
0.0970532
</td>
<td style="text-align:left;">
Maob/Cbs/Maoa/Gamt/Cth/Phgdh
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00380
</td>
<td style="text-align:left;">
Tryptophan metabolism
</td>
<td style="text-align:right;">
0.1346154
</td>
<td style="text-align:right;">
0.0278565
</td>
<td style="text-align:right;">
1.555074
</td>
<td style="text-align:right;">
0.0278565
</td>
<td style="text-align:right;">
0.1058362
</td>
<td style="text-align:left;">
Cyp1b1/Maob/Maoa/Kyat1/Dhtkd1/Echs1/Aox4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04923
</td>
<td style="text-align:left;">
Regulation of lipolysis in adipocytes
</td>
<td style="text-align:right;">
0.1228070
</td>
<td style="text-align:right;">
0.0431941
</td>
<td style="text-align:right;">
1.364576
</td>
<td style="text-align:right;">
0.0431941
</td>
<td style="text-align:right;">
0.1573957
</td>
<td style="text-align:left;">
Gnai1/Adcy3/Adcy4/Adcy7/Ptger3/Akt3/Lipe
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05144
</td>
<td style="text-align:left;">
Malaria
</td>
<td style="text-align:right;">
0.1228070
</td>
<td style="text-align:right;">
0.0431941
</td>
<td style="text-align:right;">
1.364576
</td>
<td style="text-align:right;">
0.0431941
</td>
<td style="text-align:right;">
0.1573957
</td>
<td style="text-align:left;">
Hgf/Thbs3/Icam1/Gypc/Itgal/Sele/Pecam1
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04915
</td>
<td style="text-align:left;">
Estrogen signaling pathway
</td>
<td style="text-align:right;">
0.0970149
</td>
<td style="text-align:right;">
0.0433690
</td>
<td style="text-align:right;">
1.362821
</td>
<td style="text-align:right;">
0.0433690
</td>
<td style="text-align:right;">
0.1573957
</td>
<td style="text-align:left;">
Krt23/Kcnj3/Nos3/Gnai1/Mmp2/Shc3/Adcy3/Plcb1/Adcy4/Adcy7/Plcb2/Akt3/Gabbr1
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04934
</td>
<td style="text-align:left;">
Cushing syndrome
</td>
<td style="text-align:right;">
0.0925926
</td>
<td style="text-align:right;">
0.0448593
</td>
<td style="text-align:right;">
1.348148
</td>
<td style="text-align:right;">
0.0448593
</td>
<td style="text-align:right;">
0.1581074
</td>
<td style="text-align:left;">
Kcnk3/Wnt6/Lef1/Wnt11/Cacna1c/Gnai1/Adcy3/Plcb1/Wnt9a/Adcy4/Adcy7/Agtr1a/Plcb2/Cacna1h/Fzd2
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04928
</td>
<td style="text-align:left;">
Parathyroid hormone synthesis, secretion and action
</td>
<td style="text-align:right;">
0.1018519
</td>
<td style="text-align:right;">
0.0448655
</td>
<td style="text-align:right;">
1.348087
</td>
<td style="text-align:right;">
0.0448655
</td>
<td style="text-align:right;">
0.1581074
</td>
<td style="text-align:left;">
Mmp16/Gnai1/Mef2c/Adcy3/Plcb1/Pth1r/Adcy4/Adcy7/Arrb1/Plcb2/Arrb2
</td>
<td style="text-align:right;">
11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dot-plot-1" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>kegg_dot[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-6-1.png" width="864" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-1" class="section level3">
<h3>Upset Plot</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-7-1.png" width="864" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="fc1.5" class="section level2 tabset">
<h2 class="tabset">FC=1.5</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="table-2" class="section level3">
<h3>Table</h3>
<pre class="r"><code>enrichKEGG_sig[[p]] %&gt;%
  kable(caption = &quot;Significantly enriched KEGG pathways&quot;) %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<caption>
Significantly enriched KEGG pathways
</caption>
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logPval
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
mmu04340
</td>
<td style="text-align:left;">
Hedgehog signaling pathway
</td>
<td style="text-align:right;">
0.1379310
</td>
<td style="text-align:right;">
0.0000027
</td>
<td style="text-align:right;">
5.566527
</td>
<td style="text-align:right;">
0.0000027
</td>
<td style="text-align:right;">
0.0005883
</td>
<td style="text-align:left;">
Gli1/Ihh/Hhip/Ptch2/Evc2/Evc/Lrp2/Kif7
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05217
</td>
<td style="text-align:left;">
Basal cell carcinoma
</td>
<td style="text-align:right;">
0.1111111
</td>
<td style="text-align:right;">
0.0000495
</td>
<td style="text-align:right;">
4.305328
</td>
<td style="text-align:right;">
0.0000495
</td>
<td style="text-align:right;">
0.0044503
</td>
<td style="text-align:left;">
Gli1/Wnt6/Hhip/Wnt11/Lef1/Ptch2/Kif7
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04512
</td>
<td style="text-align:left;">
ECM-receptor interaction
</td>
<td style="text-align:right;">
0.0909091
</td>
<td style="text-align:right;">
0.0000616
</td>
<td style="text-align:right;">
4.210629
</td>
<td style="text-align:right;">
0.0000616
</td>
<td style="text-align:right;">
0.0044503
</td>
<td style="text-align:left;">
Col4a6/Lamc3/Itga4/Npnt/Cd44/Col9a2/Frem1/Lama4
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04360
</td>
<td style="text-align:left;">
Axon guidance
</td>
<td style="text-align:right;">
0.0607735
</td>
<td style="text-align:right;">
0.0001109
</td>
<td style="text-align:right;">
3.955120
</td>
<td style="text-align:right;">
0.0001109
</td>
<td style="text-align:right;">
0.0060112
</td>
<td style="text-align:left;">
Epha8/Efnb3/Sema5a/Dcc/Lrrc4c/Sema5b/Plxnb1/Gnai1/Sema7a/Cxcl12/Gdf7
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05205
</td>
<td style="text-align:left;">
Proteoglycans in cancer
</td>
<td style="text-align:right;">
0.0487805
</td>
<td style="text-align:right;">
0.0012850
</td>
<td style="text-align:right;">
2.891099
</td>
<td style="text-align:right;">
0.0012850
</td>
<td style="text-align:right;">
0.0506002
</td>
<td style="text-align:left;">
Ihh/Hgf/Wnt6/Wnt11/Hoxd10/Igf1/Kdr/Gpc3/Cd44/Mmp2
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04974
</td>
<td style="text-align:left;">
Protein digestion and absorption
</td>
<td style="text-align:right;">
0.0648148
</td>
<td style="text-align:right;">
0.0014001
</td>
<td style="text-align:right;">
2.853840
</td>
<td style="text-align:right;">
0.0014001
</td>
<td style="text-align:right;">
0.0506002
</td>
<td style="text-align:left;">
Col11a1/Col26a1/Col14a1/Col4a6/Pga5/Col9a2/Slc8a3
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04020
</td>
<td style="text-align:left;">
Calcium signaling pathway
</td>
<td style="text-align:right;">
0.0416667
</td>
<td style="text-align:right;">
0.0040661
</td>
<td style="text-align:right;">
2.390824
</td>
<td style="text-align:right;">
0.0040661
</td>
<td style="text-align:right;">
0.1259569
</td>
<td style="text-align:left;">
Hgf/Ryr1/Adra1a/Kdr/Ednrb/Nos3/P2rx2/Ptgfr/Cacna1e/Slc8a3
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04728
</td>
<td style="text-align:left;">
Dopaminergic synapse
</td>
<td style="text-align:right;">
0.0518519
</td>
<td style="text-align:right;">
0.0049253
</td>
<td style="text-align:right;">
2.307566
</td>
<td style="text-align:right;">
0.0049253
</td>
<td style="text-align:right;">
0.1335021
</td>
<td style="text-align:left;">
Kcnj3/Maob/Gria4/Drd4/Scn1a/Gnai1/Grin2b
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05412
</td>
<td style="text-align:left;">
Arrhythmogenic right ventricular cardiomyopathy
</td>
<td style="text-align:right;">
0.0649351
</td>
<td style="text-align:right;">
0.0067287
</td>
<td style="text-align:right;">
2.172067
</td>
<td style="text-align:right;">
0.0067287
</td>
<td style="text-align:right;">
0.1597633
</td>
<td style="text-align:left;">
Lef1/Sgcd/Itga4/Cdh2/Slc8a3
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04080
</td>
<td style="text-align:left;">
Neuroactive ligand-receptor interaction
</td>
<td style="text-align:right;">
0.0332481
</td>
<td style="text-align:right;">
0.0075343
</td>
<td style="text-align:right;">
2.122958
</td>
<td style="text-align:right;">
0.0075343
</td>
<td style="text-align:right;">
0.1597633
</td>
<td style="text-align:left;">
Grid1/Grik1/Vipr2/Gria4/Adra1a/Drd4/Ednrb/Calca/Edn3/P2rx2/Ptgfr/Grin2b/Gabra4
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04724
</td>
<td style="text-align:left;">
Glutamatergic synapse
</td>
<td style="text-align:right;">
0.0530973
</td>
<td style="text-align:right;">
0.0081045
</td>
<td style="text-align:right;">
2.091274
</td>
<td style="text-align:right;">
0.0081045
</td>
<td style="text-align:right;">
0.1597633
</td>
<td style="text-align:left;">
Kcnj3/Slc1a3/Grik1/Gria4/Gnai1/Grin2b
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04510
</td>
<td style="text-align:left;">
Focal adhesion
</td>
<td style="text-align:right;">
0.0398010
</td>
<td style="text-align:right;">
0.0127867
</td>
<td style="text-align:right;">
1.893241
</td>
<td style="text-align:right;">
0.0127867
</td>
<td style="text-align:right;">
0.2310583
</td>
<td style="text-align:left;">
Hgf/Col4a6/Igf1/Lamc3/Itga4/Kdr/Col9a2/Lama4
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04713
</td>
<td style="text-align:left;">
Circadian entrainment
</td>
<td style="text-align:right;">
0.0510204
</td>
<td style="text-align:right;">
0.0178954
</td>
<td style="text-align:right;">
1.747258
</td>
<td style="text-align:right;">
0.0178954
</td>
<td style="text-align:right;">
0.2793284
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Ryr1/Gnai1/Grin2b
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04015
</td>
<td style="text-align:left;">
Rap1 signaling pathway
</td>
<td style="text-align:right;">
0.0373832
</td>
<td style="text-align:right;">
0.0180343
</td>
<td style="text-align:right;">
1.743901
</td>
<td style="text-align:right;">
0.0180343
</td>
<td style="text-align:right;">
0.2793284
</td>
<td style="text-align:left;">
Hgf/Angpt1/Igf1/Kdr/Gnai1/Evl/Grin2b/Rasgrp3
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04916
</td>
<td style="text-align:left;">
Melanogenesis
</td>
<td style="text-align:right;">
0.0500000
</td>
<td style="text-align:right;">
0.0193626
</td>
<td style="text-align:right;">
1.713036
</td>
<td style="text-align:right;">
0.0193626
</td>
<td style="text-align:right;">
0.2799084
</td>
<td style="text-align:left;">
Wnt6/Wnt11/Lef1/Ednrb/Gnai1
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04151
</td>
<td style="text-align:left;">
PI3K-Akt signaling pathway
</td>
<td style="text-align:right;">
0.0306407
</td>
<td style="text-align:right;">
0.0234769
</td>
<td style="text-align:right;">
1.629359
</td>
<td style="text-align:right;">
0.0234769
</td>
<td style="text-align:right;">
0.3022251
</td>
<td style="text-align:left;">
Hgf/Angpt1/Col4a6/Igf1/Lamc3/Areg/Itga4/Kdr/Nos3/Col9a2/Lama4
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05033
</td>
<td style="text-align:left;">
Nicotine addiction
</td>
<td style="text-align:right;">
0.0750000
</td>
<td style="text-align:right;">
0.0236939
</td>
<td style="text-align:right;">
1.625364
</td>
<td style="text-align:right;">
0.0236939
</td>
<td style="text-align:right;">
0.3022251
</td>
<td style="text-align:left;">
Gria4/Grin2b/Gabra4
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu00910
</td>
<td style="text-align:left;">
Nitrogen metabolism
</td>
<td style="text-align:right;">
0.1176471
</td>
<td style="text-align:right;">
0.0278455
</td>
<td style="text-align:right;">
1.555244
</td>
<td style="text-align:right;">
0.0278455
</td>
<td style="text-align:right;">
0.3354493
</td>
<td style="text-align:left;">
Car4/Car8
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04390
</td>
<td style="text-align:left;">
Hippo signaling pathway
</td>
<td style="text-align:right;">
0.0382166
</td>
<td style="text-align:right;">
0.0350594
</td>
<td style="text-align:right;">
1.455195
</td>
<td style="text-align:right;">
0.0350594
</td>
<td style="text-align:right;">
0.4001241
</td>
<td style="text-align:left;">
Wnt6/Wnt11/Lef1/Areg/Nkd1/Gdf7
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu05030
</td>
<td style="text-align:left;">
Cocaine addiction
</td>
<td style="text-align:right;">
0.0625000
</td>
<td style="text-align:right;">
0.0379529
</td>
<td style="text-align:right;">
1.420754
</td>
<td style="text-align:right;">
0.0379529
</td>
<td style="text-align:right;">
0.4114899
</td>
<td style="text-align:left;">
Maob/Gnai1/Grin2b
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04979
</td>
<td style="text-align:left;">
Cholesterol metabolism
</td>
<td style="text-align:right;">
0.0600000
</td>
<td style="text-align:right;">
0.0420678
</td>
<td style="text-align:right;">
1.376050
</td>
<td style="text-align:right;">
0.0420678
</td>
<td style="text-align:right;">
0.4343844
</td>
<td style="text-align:left;">
Apob/Lrp2/Soat1
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04310
</td>
<td style="text-align:left;">
Wnt signaling pathway
</td>
<td style="text-align:right;">
0.0350877
</td>
<td style="text-align:right;">
0.0496570
</td>
<td style="text-align:right;">
1.304019
</td>
<td style="text-align:right;">
0.0496570
</td>
<td style="text-align:right;">
0.4584936
</td>
<td style="text-align:left;">
Wnt6/Sfrp2/Wnt11/Lef1/Apcdd1/Nkd1
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
mmu04926
</td>
<td style="text-align:left;">
Relaxin signaling pathway
</td>
<td style="text-align:right;">
0.0387597
</td>
<td style="text-align:right;">
0.0498596
</td>
<td style="text-align:right;">
1.302251
</td>
<td style="text-align:right;">
0.0498596
</td>
<td style="text-align:right;">
0.4584936
</td>
<td style="text-align:left;">
Col4a6/Ednrb/Nos3/Gnai1/Mmp2
</td>
<td style="text-align:right;">
5
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dot-plot-2" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>kegg_dot[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-10-1.png" width="864" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-plot-2" class="section level3">
<h3>Upset Plot</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-11-1.png" width="864" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="pathway-specific-heatmaps" class="section level1 tabset">
<h1 class="tabset">Pathway specific heatmaps</h1>
<div id="fc1.05-1" class="section level2 tabset">
<h2 class="tabset">FC=1.05</h2>
<pre class="r"><code>p=1</code></pre>
<div id="leukocyte-transendothelial-migration"
class="section level3 tabset">
<h3 class="tabset">Leukocyte transendothelial migration</h3>
<pre class="r"><code>q=1</code></pre>
<div id="heatmap" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code># create df with normalised read counts with an additional entrezid column for binding
logCPM &lt;- cpm(dge, prior.count = 3, log = TRUE)
logCPM &lt;- logCPM[,1:7]
logCPM &lt;- cbind(logCPM, dge$genes$entrezid)
rownames(logCPM) &lt;- dge$genes$gene_name
colnames(logCPM) &lt;- c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;, &quot;entrezid&quot;)

### full pathway method
# complete_pathway &lt;- kegg_pathway[[1]]$GENE %&gt;% as.data.frame()
# complete_pathway &lt;- focal_adhesion[seq(1, nrow(focal_adhesion), 2),]
# match_complete_pathway &lt;- logCPM[,&quot;entrezid&quot;] %in% complete_pathway


# df for heatmap annotation of sample group
anno &lt;- as.factor(dge$samples$group) %&gt;% as.data.frame() 
anno &lt;- anno[1:7,] %&gt;% as.data.frame()
colnames(anno) &lt;- &quot;Sample Groups&quot;
anno$`Sample Groups` &lt;- gsub(&quot;CONT&quot;, &quot;Control&quot;, anno$`Sample Groups`)
anno$`Sample Groups` &lt;- gsub(&quot;INT&quot;, &quot;Intact&quot;, anno$`Sample Groups`)
rownames(anno) &lt;- colnames(logCPM[, 1:7])

# setting colour of sample group annotation

# original sample colours
# anno_colours &lt;- c(&quot;#66C2A5&quot;, &quot;#FC8D62&quot;)

# new sample colours
anno_colours &lt;- c(&quot;#f8766d&quot;, &quot;#a3a500&quot;)

names(anno_colours) &lt;- c(&quot;Control&quot;, &quot;Intact&quot;)</code></pre>
<pre class="r"><code>matrix &lt;- list()
display_matrix &lt;- list()
kegg_heat=list()

my_palette &lt;- colorRampPalette(c(
  rgb(32,121,226, maxColorValue = 255),
  # rgb(144,203,180, maxColorValue = 255), 
  rgb(254,248,239, maxColorValue = 255), 
  # rgb(251,192,52, maxColorValue = 255), 
  rgb(226,46,45, maxColorValue = 255)))(n = 201)

for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  
  for (j in 1:length(kegg_id)) {
    y &lt;- kegg_pathway[[j]]$PATHWAY_MAP
    
    partial &lt;- enrichKEGG_all[[x]][, c(&quot;ID&quot;, &quot;geneID&quot;)]
    partial &lt;- partial[kegg_id[j], &quot;geneID&quot;] %&gt;% as.data.frame()
    partial &lt;- separate_rows(partial, ., sep = &quot;/&quot;)
    colnames(partial) &lt;- &quot;entrezid&quot;

    # heatmap matrix
    match &lt;- rownames(logCPM) %in% partial$entrezid
    matrix[[x]][[y]] &lt;- logCPM[match, c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;)] %&gt;% as.data.frame()
    
    # changing the colname to  numeric for some reason, cant remember
    
    matrix[[x]][[y]][, c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;)] &lt;- as.numeric(as.character(unlist(matrix[[x]][[y]][, c(&quot;Control 1&quot;, &quot;Control 2&quot;, &quot;Control 4&quot;, &quot;Intact 1&quot;, &quot;Intact 2&quot;, &quot;Intact 3&quot;, &quot;Intact 4&quot;)])))

    # display matrix
    match2 &lt;- lmTreat_sig[[x]][, &quot;entrezid&quot;] %in% partial$entrezid
    display_matrix[[x]][[y]] &lt;- lmTreat_sig[[x]][match2, c(&quot;gene_name&quot;, &quot;logFC&quot;, &quot;P.Value&quot;, &quot;adj.P.Val&quot;, &quot;description&quot;)] %&gt;%
      as.data.frame()
    colnames(display_matrix[[x]][[y]]) &lt;- c(&quot;Gene Name&quot;, &quot;logFC&quot;, &quot;P Value&quot;, &quot;Adjusted P Value&quot;, &quot;Description&quot;)
    
    ## Heatmap
    kegg_heat[[x]][[y]] &lt;- pheatmap(
      mat = matrix[[x]][[y]],
      ### Publish
      show_colnames = T,
      main = paste0(y, &quot;\n&quot;),
      legend = F,
      annotation_legend = F,
      fontsize = 8,
      fontsize_col = 9,
      fontsize_number = 7,
      fontsize_row = 8,
      treeheight_row = 25,
      treeheight_col = 10,
      cluster_cols = T,
      clustering_distance_rows = &quot;euclidean&quot;,
      legend_breaks = c(seq(-3, 11, by = .5), 1.4),
      legend_labels = c(seq(-3, 11, by = .5), &quot;Z-Score&quot;),
      angle_col = 90,
      cutree_cols = 2,
      cutree_rows = 2,
      color = my_palette,
      scale = &quot;row&quot;,
      border_color = NA,
      annotation_col = anno,
      annotation_colors = list(&quot;Sample Groups&quot; = anno_colours),
      annotation_names_col = F,
      annotation = T,
      silent = T,
      
      labels_row = as.expression(lapply(rownames(matrix[[x]][[y]]), function(a) bquote(italic(.(a)))))
      
      ) %&gt;% as.ggplot()
    
    # save
    ggsave(filename = paste0(&quot;heat_&quot;, x, &quot;_&quot;, y, &quot;.svg&quot;), 
           plot = kegg_heat[[x]][[y]], 
           path = here::here(&quot;2_plots/kegg/&quot;),
           width = 166,
           height = 250,
           units = &quot;mm&quot;)}
}</code></pre>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="table-3" class="section level4">
<h4>Table</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview" class="section level4">
<h4>Pathview</h4>
<pre class="r"><code># adjusting the kegg id to suit the parameters of the pathview funtion
adj.keggID &lt;- gsub(&quot;mmu&quot;, &quot;&quot;, kegg_id)

for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  
  # extract the logFC from the DE gene list
  pathview_table &lt;- dplyr::select(.data = lmTreat_sig[[x]], c(&quot;logFC&quot;)) %&gt;% as.matrix()

  # run pathview with Ensembl ID instead of entrezID
  pathview &lt;- pathview(
    gene.data = pathview_table[, 1],
    gene.idtype = &quot;ENSEMBL&quot;,
    pathway.id = adj.keggID,
    species = &quot;mmu&quot;,
    out.suffix = &quot;pv&quot;,
    kegg.dir = here::here(&quot;2_plots/kegg/&quot;),
    kegg.native = T
  )

  # move the result file to the plot directory
  file.rename(
    from = paste0(&quot;mmu&quot;, adj.keggID, &quot;.pv.png&quot;),
    to = here::here(paste0(&quot;docs/figure/kegg.Rmd/pv_&quot;, x, &quot;_&quot;, kegg_id, &quot;.png&quot;))
  )
}</code></pre>
<pre><code>[1] &quot;Note: 166 of 1629 unique input IDs unmapped.&quot;</code></pre>
<pre><code>[1] &quot;Note: 151 of 1447 unique input IDs unmapped.&quot;</code></pre>
<pre><code>[1] &quot;Note: 63 of 388 unique input IDs unmapped.&quot;</code></pre>
<div class="figure">
<img src="figure/kegg.Rmd/pv_1.5_mmu04670.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="hematopoietic-cell-lineage" class="section level3 tabset">
<h3 class="tabset">Hematopoietic cell lineage</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-1" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-1" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_1.5_mmu04640.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="vascular-smooth-muscle-contraction"
class="section level3 tabset">
<h3 class="tabset">Vascular smooth muscle contraction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-2" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-19-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-1" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-2" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_1.5_mmu04270.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="pi3k-akt-signaling-pathway" class="section level3 tabset">
<h3 class="tabset">PI3K-Akt signaling pathway</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-3" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-2" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-3" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_1.5_mmu04151.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="focal-adhesion" class="section level3 tabset">
<h3 class="tabset">Focal adhesion</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-4" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-3" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-4" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_1.5_mmu04510.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="cytokine-cytokine-receptor-interaction"
class="section level3 tabset">
<h3 class="tabset">Cytokine-cytokine receptor interaction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-5" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-4" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-5" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_1.5_mmu04060.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
</div>
<div id="fc1.1-1" class="section level2 tabset">
<h2 class="tabset">FC=1.1</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="leukocyte-transendothelial-migration-1"
class="section level3 tabset">
<h3 class="tabset">Leukocyte transendothelial migration</h3>
<pre class="r"><code>q=1</code></pre>
<div id="heatmap-6" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-32-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-5" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-6" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04670.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="hematopoietic-cell-lineage-1" class="section level3 tabset">
<h3 class="tabset">Hematopoietic cell lineage</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-7" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-35-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-6" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-7" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04640.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="vascular-smooth-muscle-contraction-1"
class="section level3 tabset">
<h3 class="tabset">Vascular smooth muscle contraction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-8" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-38-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-7" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-8" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04270.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="pi3k-akt-signaling-pathway-1" class="section level3 tabset">
<h3 class="tabset">PI3K-Akt signaling pathway</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-9" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-41-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-8" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-9" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04151.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="focal-adhesion-1" class="section level3 tabset">
<h3 class="tabset">Focal adhesion</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-10" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-44-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-9" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-10" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04510.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="cytokine-cytokine-receptor-interaction-1"
class="section level3 tabset">
<h3 class="tabset">Cytokine-cytokine receptor interaction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-11" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-47-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-10" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-11" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04060.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
</div>
<div id="fc1.5-1" class="section level2 tabset">
<h2 class="tabset">FC=1.5</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="leukocyte-transendothelial-migration-2"
class="section level3 tabset">
<h3 class="tabset">Leukocyte transendothelial migration</h3>
<pre class="r"><code>q=1</code></pre>
<div id="heatmap-12" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-51-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-11" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-12" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04670.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="hematopoietic-cell-lineage-2" class="section level3 tabset">
<h3 class="tabset">Hematopoietic cell lineage</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-13" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-54-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-12" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-13" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04640.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="vascular-smooth-muscle-contraction-2"
class="section level3 tabset">
<h3 class="tabset">Vascular smooth muscle contraction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-14" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-57-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-13" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-14" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04270.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="pi3k-akt-signaling-pathway-2" class="section level3 tabset">
<h3 class="tabset">PI3K-Akt signaling pathway</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-15" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-60-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-14" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-15" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04151.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="focal-adhesion-2" class="section level3 tabset">
<h3 class="tabset">Focal adhesion</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-16" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-63-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-15" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-16" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04510.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
<div id="cytokine-cytokine-receptor-interaction-2"
class="section level3 tabset">
<h3 class="tabset">Cytokine-cytokine receptor interaction</h3>
<pre class="r"><code>q=q+1</code></pre>
<div id="heatmap-17" class="section level4">
<h4>Heatmap</h4>
<pre class="r"><code>kegg_heat[[p]][[q]]</code></pre>
<p><img src="figure/kegg.Rmd/unnamed-chunk-66-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="tables-16" class="section level4">
<h4>Tables</h4>
<pre class="r"><code>display_matrix[[p]][[q]] %&gt;%
  kable() %&gt;%
  kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Gene Name
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFC
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
P Value
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Adjusted P Value
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
</tr>
</thead>
<tbody>
<tr>
</tr>
</tbody>
</table>
</div>
</div>
<div id="pathview-17" class="section level4">
<h4>Pathview</h4>
<div class="figure">
<img src="figure/kegg.Rmd/pv_NA_mmu04060.png" alt="" />
<p class="caption">Pathview</p>
</div>
</div>
</div>
</div>
</div>
<div id="export-data" class="section level1">
<h1>Export Data</h1>
<pre class="r"><code># save to csv
writexl::write_xlsx(x = enrichKEGG_all, here::here(&quot;3_output/enrichKEGG_all.xlsx&quot;))
writexl::write_xlsx(x = enrichKEGG_sig, here::here(&quot;3_output/enrichKEGG_sig.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.1 (2022-06-23 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.utf8  LC_CTYPE=English_Australia.utf8   
[3] LC_MONETARY=English_Australia.utf8 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.utf8    

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] pathview_1.36.1       enrichplot_1.16.2     org.Mm.eg.db_3.15.0  
 [4] AnnotationDbi_1.58.0  IRanges_2.30.1        S4Vectors_0.34.0     
 [7] Biobase_2.56.0        BiocGenerics_0.42.0   clusterProfiler_4.4.4
[10] Glimma_2.6.0          edgeR_3.38.4          limma_3.52.4         
[13] ggrepel_0.9.1         ggbiplot_0.55         scales_1.2.1         
[16] plyr_1.8.7            ggpubr_0.4.0          ggplotify_0.1.0      
[19] pheatmap_1.0.12       cowplot_1.1.1         viridis_0.6.2        
[22] viridisLite_0.4.1     pander_0.6.5          kableExtra_1.3.4     
[25] KEGGREST_1.36.3       forcats_0.5.2         stringr_1.4.1        
[28] purrr_0.3.5           tidyr_1.2.1           ggplot2_3.3.6        
[31] tidyverse_1.3.2       reshape2_1.4.4        tibble_3.1.8         
[34] readr_2.1.3           magrittr_2.0.3        dplyr_1.0.10         

loaded via a namespace (and not attached):
  [1] utf8_1.2.2                  tidyselect_1.2.0           
  [3] RSQLite_2.2.18              htmlwidgets_1.5.4          
  [5] BiocParallel_1.30.3         scatterpie_0.1.8           
  [7] munsell_0.5.0               ragg_1.2.3                 
  [9] codetools_0.2-18            withr_2.5.0                
 [11] colorspace_2.0-3            GOSemSim_2.22.0            
 [13] highr_0.9                   knitr_1.40                 
 [15] rstudioapi_0.14             ggsignif_0.6.3             
 [17] DOSE_3.22.1                 labeling_0.4.2             
 [19] MatrixGenerics_1.8.1        KEGGgraph_1.56.0           
 [21] git2r_0.30.1                GenomeInfoDbData_1.2.8     
 [23] polyclip_1.10-0             bit64_4.0.5                
 [25] farver_2.1.1                rprojroot_2.0.3            
 [27] downloader_0.4              treeio_1.20.2              
 [29] vctrs_0.4.2                 generics_0.1.3             
 [31] xfun_0.33                   R6_2.5.1                   
 [33] GenomeInfoDb_1.32.4         graphlayouts_0.8.2         
 [35] locfit_1.5-9.6              bitops_1.0-7               
 [37] cachem_1.0.6                fgsea_1.22.0               
 [39] gridGraphics_0.5-1          DelayedArray_0.22.0        
 [41] assertthat_0.2.1            promises_1.2.0.1           
 [43] ggraph_2.1.0                googlesheets4_1.0.1        
 [45] gtable_0.3.1                tidygraph_1.2.2            
 [47] workflowr_1.7.0             rlang_1.0.6                
 [49] genefilter_1.78.0           systemfonts_1.0.4          
 [51] splines_4.2.1               lazyeval_0.2.2             
 [53] rstatix_0.7.0               gargle_1.2.1               
 [55] broom_1.0.1                 yaml_2.3.5                 
 [57] abind_1.4-5                 modelr_0.1.9               
 [59] backports_1.4.1             httpuv_1.6.6               
 [61] qvalue_2.28.0               tools_4.2.1                
 [63] ellipsis_0.3.2              jquerylib_0.1.4            
 [65] RColorBrewer_1.1-3          Rcpp_1.0.9                 
 [67] zlibbioc_1.42.0             RCurl_1.98-1.9             
 [69] SummarizedExperiment_1.26.1 haven_2.5.1                
 [71] here_1.0.1                  fs_1.5.2                   
 [73] data.table_1.14.2           DO.db_2.9                  
 [75] reprex_2.0.2                googledrive_2.0.0          
 [77] whisker_0.4                 matrixStats_0.62.0         
 [79] patchwork_1.1.2             hms_1.1.2                  
 [81] evaluate_0.17               xtable_1.8-4               
 [83] XML_3.99-0.11               readxl_1.4.1               
 [85] gridExtra_2.3               ggupset_0.3.0              
 [87] compiler_4.2.1              writexl_1.4.0              
 [89] shadowtext_0.1.2            crayon_1.5.2               
 [91] htmltools_0.5.3             ggfun_0.0.7                
 [93] later_1.3.0                 tzdb_0.3.0                 
 [95] geneplotter_1.74.0          aplot_0.1.8                
 [97] lubridate_1.8.0             DBI_1.1.3                  
 [99] tweenr_2.0.2                dbplyr_2.2.1               
[101] MASS_7.3-57                 Matrix_1.5-1               
[103] car_3.1-0                   cli_3.4.1                  
[105] parallel_4.2.1              igraph_1.3.5               
[107] GenomicRanges_1.48.0        pkgconfig_2.0.3            
[109] xml2_1.3.3                  ggtree_3.4.4               
[111] svglite_2.1.0               annotate_1.74.0            
[113] bslib_0.4.0                 webshot_0.5.4              
[115] XVector_0.36.0              rvest_1.0.3                
[117] yulab.utils_0.0.5           digest_0.6.29              
[119] graph_1.74.0                Biostrings_2.64.1          
[121] rmarkdown_2.17              cellranger_1.1.0           
[123] fastmatch_1.1-3             tidytree_0.4.1             
[125] curl_4.3.3                  nlme_3.1-157               
[127] lifecycle_1.0.3             jsonlite_1.8.2             
[129] carData_3.0-5               fansi_1.0.3                
[131] pillar_1.8.1                lattice_0.20-45            
[133] fastmap_1.1.0               httr_1.4.4                 
[135] survival_3.3-1              GO.db_3.15.0               
[137] glue_1.6.2                  png_0.1-7                  
[139] Rgraphviz_2.40.0            bit_4.0.4                  
[141] ggforce_0.4.1               stringi_1.7.8              
[143] sass_0.4.2                  blob_1.2.3                 
[145] textshaping_0.3.6           org.Hs.eg.db_3.15.0        
[147] DESeq2_1.36.0               memoise_2.0.1              
[149] ape_5.6-2                  </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
