<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha Tran" />


<title>DE Analysis (Robertson Lab)</title>

<script src="site_libs/header-attrs-2.17/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">SRB_2022</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="setUp.html">Pre-processing</a>
</li>
<li>
  <a href="deAnalysis.html">DGE Analysis</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Functional Enrichment Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="go.html">Gene Ontology</a>
    </li>
    <li>
      <a href="kegg.html">KEGG</a>
    </li>
    <li>
      <a href="reactome.html">Reactome</a>
    </li>
    <li>
      <a href="extraFigures.html">IPA</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">DE Analysis (Robertson Lab)</h1>
<h4 class="author">Ha Tran</h4>
<h4 class="date">22/08/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-01-20
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>SRB_2022/1_analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the
code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135SRB2022tree7f6bab239f8f9682dab208d1c314dd72dbbd21bctargetblank7f6bab2a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/tranmanhha135/SRB_2022/tree/7f6bab239f8f9682dab208d1c314dd72dbbd21bc" target="_blank">7f6bab2</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135SRB2022tree7f6bab239f8f9682dab208d1c314dd72dbbd21bctargetblank7f6bab2a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/tranmanhha135/SRB_2022/tree/7f6bab239f8f9682dab208d1c314dd72dbbd21bc" target="_blank">7f6bab2</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  .gitignore

Unstaged changes:
    Modified:   0_data/RDS_objects/enrichGO_sig.rds
    Modified:   0_data/RDS_objects/pub.rds
    Deleted:    1_analysis/figure/kegg.Rmd/kegg_dot-1.png
    Deleted:    1_analysis/figure/kegg.Rmd/unnamed-chunk-10-1.png
    Deleted:    1_analysis/figure/kegg.Rmd/unnamed-chunk-11-1.png
    Deleted:    1_analysis/figure/kegg.Rmd/unnamed-chunk-6-1.png
    Deleted:    1_analysis/figure/kegg.Rmd/unnamed-chunk-7-1.png
    Deleted:    1_analysis/figure/kegg.Rmd/upset-1.png
    Modified:   2_plots/de/ma_1.05.png
    Modified:   2_plots/de/ma_1.1.png
    Modified:   2_plots/de/ma_1.5.png
    Modified:   2_plots/de/pval_1.05.svg
    Modified:   2_plots/de/vol_1.05.png
    Modified:   2_plots/de/vol_1.1.png
    Modified:   2_plots/de/vol_1.5.png
    Modified:   2_plots/go/bp_dot_1.05.svg
    Modified:   2_plots/go/bp_dot_1.1.svg
    Modified:   2_plots/go/bp_dot_1.5.svg
    Modified:   2_plots/go/cc_dot_1.05.svg
    Modified:   2_plots/go/cc_dot_1.1.svg
    Modified:   2_plots/go/cc_dot_1.5.svg
    Modified:   2_plots/go/mf_dot_1.05.svg
    Modified:   2_plots/go/mf_dot_1.1.svg
    Modified:   2_plots/go/mf_dot_1.5.svg
    Modified:   2_plots/ipa/Cardiovascular System.svg
    Modified:   2_plots/ipa/Cell-To-Cell Signaling.svg
    Modified:   2_plots/ipa/Cellular Movement.svg
    Modified:   2_plots/ipa/pathways.svg
    Modified:   2_plots/kegg/heat_1.05_Cytokine-cytokine receptor interaction.svg
    Modified:   2_plots/kegg/heat_1.05_Focal adhesion.svg
    Modified:   2_plots/kegg/heat_1.05_Hematopoietic cell lineage.svg
    Modified:   2_plots/kegg/heat_1.05_Leukocyte transendothelial migration.svg
    Modified:   2_plots/kegg/heat_1.05_PI3K-Akt signaling pathway.svg
    Modified:   2_plots/kegg/heat_1.05_Vascular smooth muscle contraction.svg
    Modified:   2_plots/kegg/heat_1.1_Cytokine-cytokine receptor interaction.svg
    Modified:   2_plots/kegg/heat_1.1_Focal adhesion.svg
    Modified:   2_plots/kegg/heat_1.1_Hematopoietic cell lineage.svg
    Modified:   2_plots/kegg/heat_1.1_Leukocyte transendothelial migration.svg
    Modified:   2_plots/kegg/heat_1.1_PI3K-Akt signaling pathway.svg
    Modified:   2_plots/kegg/heat_1.1_Vascular smooth muscle contraction.svg
    Modified:   2_plots/kegg/heat_1.5_Cytokine-cytokine receptor interaction.svg
    Modified:   2_plots/kegg/heat_1.5_Focal adhesion.svg
    Modified:   2_plots/kegg/heat_1.5_Hematopoietic cell lineage.svg
    Modified:   2_plots/kegg/heat_1.5_Leukocyte transendothelial migration.svg
    Modified:   2_plots/kegg/heat_1.5_PI3K-Akt signaling pathway.svg
    Modified:   2_plots/kegg/heat_1.5_Vascular smooth muscle contraction.svg
    Modified:   2_plots/kegg/kegg_dot_1.05.svg
    Modified:   2_plots/kegg/kegg_dot_1.1.svg
    Modified:   2_plots/kegg/kegg_dot_1.5.svg
    Modified:   2_plots/kegg/upset_kegg_1.05.svg
    Modified:   2_plots/kegg/upset_kegg_1.1.svg
    Modified:   2_plots/kegg/upset_kegg_1.5.svg
    Modified:   2_plots/qc/PCA_IntvsCont.svg
    Modified:   2_plots/reactome/react_dot_1.05.svg
    Modified:   2_plots/reactome/react_dot_1.1.svg
    Modified:   2_plots/reactome/react_dot_1.5.svg
    Modified:   2_plots/reactome/upset_react_1.05.svg
    Modified:   2_plots/reactome/upset_react_1.1.svg
    Modified:   2_plots/reactome/upset_react_1.5.svg
    Modified:   3_output/enrichGO_sig.xlsx
    Modified:   3_output/enrichKEGG_all.xlsx
    Modified:   3_output/enrichKEGG_sig.xlsx
    Modified:   3_output/lmTreat_all.xlsx
    Modified:   3_output/lmTreat_fc1.5_voom2_all_fdr.xlsx
    Modified:   3_output/lmTreat_sig.xlsx
    Modified:   3_output/reactome_all.xlsx
    Modified:   3_output/reactome_sig.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>1_analysis/reactome.Rmd</code>) and HTML
(<code>docs/reactome.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/7f6bab239f8f9682dab208d1c314dd72dbbd21bc/1_analysis/reactome.Rmd" target="_blank">7f6bab2</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
workflowr::wflow_publish(here::here("1_analysis/*.Rmd"))
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/b6cf1906d2fd775f953bd1dd6b835234e260209c/1_analysis/reactome.Rmd" target="_blank">b6cf190</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-19
</td>
<td>
quick commit
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/SRB_2022/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/1_analysis/reactome.Rmd" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/SRB_2022/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/reactome.html" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)
library(pheatmap)

# Custom ggplot
library(ggbiplot)
library(ggrepel)
theme_set(theme_light())

pub &lt;- readRDS(here::here(&quot;0_data/RDS_objects/pub.rds&quot;))

# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Mm.eg.db)
library(enrichplot)
library(ReactomePA)</code></pre>
<div id="import-dgelist-data" class="section level3">
<h3>Import DGElist Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and
gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/RDS_objects/dge.rds&quot;))
fc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/fc.rds&quot;))
lfc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lfc.rds&quot;))
lmTreat &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat.rds&quot;))
lmTreat_sig &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat_sig.rds&quot;))</code></pre>
</div>
</div>
<div id="reactome" class="section level1 tabset">
<h1 class="tabset">Reactome</h1>
<div id="fc1.05" class="section level2 tabset">
<h2 class="tabset">FC=1.05</h2>
<pre class="r"><code>p=1</code></pre>
<div id="enriched-pathways" class="section level3">
<h3>Enriched pathways</h3>
<pre class="r"><code>reactome=list()
reactome_all=list()
reactome_sig=list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  
  reactome[[x]] &lt;- enrichPathway(gene = lmTreat_sig[[x]]$entrezid, organism = &quot;mouse&quot;, pvalueCutoff = 0.05, pAdjustMethod = &quot;fdr&quot;, readable = T)

reactome_all[[x]] &lt;- reactome[[x]]@result
reactome_sig[[x]] &lt;- reactome_all[[x]] %&gt;% dplyr::filter(p.adjust &lt;= 0.05) %&gt;% 
  separate(col = BgRatio, sep = &quot;/&quot;, into = c(&quot;Total&quot;, &quot;Universe&quot;)) %&gt;%
  dplyr::mutate(
    logFDR = -log(p.adjust, 10),
    GeneRatio = Count / as.numeric(Total))%&gt;%
    dplyr::select(c(&quot;Description&quot;, &quot;GeneRatio&quot;, &quot;pvalue&quot;, &quot;p.adjust&quot;, &quot;logFDR&quot;, &quot;qvalue&quot;, &quot;geneID&quot;, &quot;Count&quot;))

 # at the beginnning of a word (after 35 characters), add a newline. shorten the y axis for dot plot 
  reactome_sig[[x]]$Description &lt;- sub(pattern = &quot;(.{1,35})(?:$| )&quot;, 
                                       replacement = &quot;\\1\n&quot;, 
                                       x = reactome_sig[[x]]$Description)
  
  # remove the additional newline at the end of the string
  reactome_sig[[x]]$Description &lt;- sub(pattern = &quot;\n$&quot;, 
                                       replacement = &quot;&quot;, 
                                       x = reactome_sig[[x]]$Description)
}
 
kable(x = reactome_sig[[p]]) %&gt;%
kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% 
scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFDR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
R-MMU-112316
</td>
<td style="text-align:left;">
Neuronal System
</td>
<td style="text-align:right;">
0.1744548
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000023
</td>
<td style="text-align:right;">
5.646701
</td>
<td style="text-align:right;">
0.0000019
</td>
<td style="text-align:left;">
Kcnk3/Kcnb2/Syn2/Slc1a3/Kcnj3/Gria4/Kcna6/Kcnd2/Ptprd/Kcnd3/Nlgn1/Epb41l3/Abat/Lrrc7/Gnai1/Lrrc4b/Kcnq5/Grin2b/Adcy3/Shank2/Maoa/Gngt2/Plcb1/Ppfia2/Slitrk3/Rps6ka2/Glrb/Ppfia4/Adcy4/Adcy7/Gabra4/Dlg2/Gng11/Lin7a/Nlgn3/Tspan7/Kcnma1/Gjc1/Gng2/Kcnc4/Plcb2/Slc38a1/Lrfn2/Dlg4/Kcng4/Gnb5/Arhgef9/Gnb4/Gabbr1/Kcnq4/Cacna1a/Cacnb4/Prkar1b/Kcnh1/Camk1/Nlgn2
</td>
<td style="text-align:right;">
56
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1474244
</td>
<td style="text-align:left;">
Extracellular matrix organization
</td>
<td style="text-align:right;">
0.1904762
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000040
</td>
<td style="text-align:right;">
5.400249
</td>
<td style="text-align:right;">
0.0000034
</td>
<td style="text-align:left;">
Col11a1/Mmp16/Col26a1/Itga4/P3h3/Col14a1/Capn6/Ddr2/Col4a6/Cd44/Lama4/Col16a1/Itgb7/Fn1/Mmp2/Icam1/Nid1/Jam3/Itga1/Col13a1/Loxl3/Itgal/Emilin1/Pecam1/Col5a2/Ltbp3/Mmp19/Bgn/Col15a1/P3h1/Adam19/Efemp2/Crtap/Col23a1/Col18a1/Itga8/Itgb4/Tmprss6/Lum/Nid2/Bmp7/Jam2/Fbln5/Col4a5
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-109582
</td>
<td style="text-align:left;">
Hemostasis
</td>
<td style="text-align:right;">
0.1451943
</td>
<td style="text-align:right;">
0.0000001
</td>
<td style="text-align:right;">
0.0000176
</td>
<td style="text-align:right;">
4.754163
</td>
<td style="text-align:right;">
0.0000150
</td>
<td style="text-align:left;">
Apob/Hgf/Angpt1/Itga4/Cd109/Igf1/Tfpi/Slc8a1/Nos3/Cd44/Pcdh7/P2rx2/Gnai1/Fn1/Pde1b/Serpine2/Jam3/Tek/Gngt2/Pik3r6/Dock2/Rasgrp2/Slc8a3/Trpc6/Sh2b2/Kif26a/Dock10/Maged2/Islr/Procr/Gng11/Itgal/Esam/Dgkb/Tubb4a/Vegfc/Pde1a/Sele/Pecam1/Kif6/Igf2/Gng2/Pros1/Arrb1/Sccpdh/Arrb2/Fyn/Pde5a/Cd84/Tor4a/Fgr/Habp4/Atp2b4/Gnb5/Fam3c/Selenop/Gnb4/Apbb1ip/Sri/Lat/Vps45/Kif16b/Spn/Lcp2/Pafah2/Lhfpl2/F2r/Prkar1b/Kif5a/Plek/Jam2
</td>
<td style="text-align:right;">
71
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-418346
</td>
<td style="text-align:left;">
Platelet homeostasis
</td>
<td style="text-align:right;">
0.2676056
</td>
<td style="text-align:right;">
0.0000010
</td>
<td style="text-align:right;">
0.0001715
</td>
<td style="text-align:right;">
3.765660
</td>
<td style="text-align:right;">
0.0001464
</td>
<td style="text-align:left;">
Apob/Slc8a1/Nos3/P2rx2/Pde1b/Gngt2/Slc8a3/Trpc6/Gng11/Pde1a/Pecam1/Gng2/Pde5a/Fgr/Atp2b4/Gnb5/Gnb4/Sri/Pafah2
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-216083
</td>
<td style="text-align:left;">
Integrin cell surface interactions
</td>
<td style="text-align:right;">
0.2727273
</td>
<td style="text-align:right;">
0.0000014
</td>
<td style="text-align:right;">
0.0001956
</td>
<td style="text-align:right;">
3.708584
</td>
<td style="text-align:right;">
0.0001670
</td>
<td style="text-align:left;">
Itga4/Col4a6/Cd44/Col16a1/Itgb7/Fn1/Icam1/Jam3/Itga1/Col13a1/Itgal/Pecam1/Col5a2/Col18a1/Itga8/Lum/Jam2/Col4a5
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-112315
</td>
<td style="text-align:left;">
Transmission across Chemical Synapses
</td>
<td style="text-align:right;">
0.1758794
</td>
<td style="text-align:right;">
0.0000026
</td>
<td style="text-align:right;">
0.0002986
</td>
<td style="text-align:right;">
3.524891
</td>
<td style="text-align:right;">
0.0002549
</td>
<td style="text-align:left;">
Syn2/Slc1a3/Kcnj3/Gria4/Abat/Lrrc7/Gnai1/Grin2b/Adcy3/Maoa/Gngt2/Plcb1/Ppfia2/Rps6ka2/Glrb/Ppfia4/Adcy4/Adcy7/Gabra4/Dlg2/Gng11/Lin7a/Tspan7/Gng2/Plcb2/Slc38a1/Dlg4/Gnb5/Arhgef9/Gnb4/Gabbr1/Cacna1a/Cacnb4/Prkar1b/Camk1
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-5173214
</td>
<td style="text-align:left;">
O-glycosylation of TSR domain-containing proteins
</td>
<td style="text-align:right;">
0.3636364
</td>
<td style="text-align:right;">
0.0000030
</td>
<td style="text-align:right;">
0.0002986
</td>
<td style="text-align:right;">
3.524891
</td>
<td style="text-align:right;">
0.0002549
</td>
<td style="text-align:left;">
Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Adamts10/Adamts7/Adamtsl1/Sbspon/Adamts17
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-5173105
</td>
<td style="text-align:left;">
O-linked glycosylation
</td>
<td style="text-align:right;">
0.2187500
</td>
<td style="text-align:right;">
0.0000090
</td>
<td style="text-align:right;">
0.0007894
</td>
<td style="text-align:right;">
3.102709
</td>
<td style="text-align:right;">
0.0006737
</td>
<td style="text-align:left;">
Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Galnt16/St3gal2/St6galnac4/Adamts10/Adamts7/Galntl6/Adamtsl1/Pomt2/Large1/B3gnt5/St3gal4/Sbspon/Adamts17/Galnt18
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-112314
</td>
<td style="text-align:left;">
Neurotransmitter receptors and postsynaptic signal transmission
</td>
<td style="text-align:right;">
0.1838235
</td>
<td style="text-align:right;">
0.0000325
</td>
<td style="text-align:right;">
0.0025359
</td>
<td style="text-align:right;">
2.595874
</td>
<td style="text-align:right;">
0.0021643
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Lrrc7/Gnai1/Grin2b/Adcy3/Gngt2/Plcb1/Rps6ka2/Glrb/Adcy4/Adcy7/Gabra4/Dlg2/Gng11/Tspan7/Gng2/Plcb2/Dlg4/Gnb5/Arhgef9/Gnb4/Gabbr1/Prkar1b/Camk1
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-416482
</td>
<td style="text-align:left;">
G alpha (12/13) signalling events
</td>
<td style="text-align:right;">
0.2285714
</td>
<td style="text-align:right;">
0.0000592
</td>
<td style="text-align:right;">
0.0041624
</td>
<td style="text-align:right;">
2.380653
</td>
<td style="text-align:right;">
0.0035526
</td>
<td style="text-align:left;">
Plxnb1/Adra1a/Arhgef25/Gngt2/Gng11/Arhgef15/Gng2/Arhgef26/Net1/Fgd1/Gnb5/Ngef/Arhgef9/Gnb4/Fgd2/Plekhg5
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-418597
</td>
<td style="text-align:left;">
G alpha (z) signalling events
</td>
<td style="text-align:right;">
0.3225806
</td>
<td style="text-align:right;">
0.0000665
</td>
<td style="text-align:right;">
0.0042502
</td>
<td style="text-align:right;">
2.371590
</td>
<td style="text-align:right;">
0.0036275
</td>
<td style="text-align:left;">
Rgs16/Adcy3/Gngt2/Adcy4/Adcy7/Gng11/Gng2/Gnaz/Gnb5/Gnb4
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-6794362
</td>
<td style="text-align:left;">
Protein-protein interactions at synapses
</td>
<td style="text-align:right;">
0.2272727
</td>
<td style="text-align:right;">
0.0001074
</td>
<td style="text-align:right;">
0.0062930
</td>
<td style="text-align:right;">
2.201141
</td>
<td style="text-align:right;">
0.0053710
</td>
<td style="text-align:left;">
Gria4/Ptprd/Nlgn1/Epb41l3/Lrrc4b/Grin2b/Shank2/Ppfia2/Slitrk3/Ppfia4/Dlg2/Nlgn3/Lrfn2/Dlg4/Nlgn2
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1650814
</td>
<td style="text-align:left;">
Collagen biosynthesis and modifying enzymes
</td>
<td style="text-align:right;">
0.2333333
</td>
<td style="text-align:right;">
0.0001343
</td>
<td style="text-align:right;">
0.0068857
</td>
<td style="text-align:right;">
2.162050
</td>
<td style="text-align:right;">
0.0058769
</td>
<td style="text-align:left;">
Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1/Col4a5
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-977444
</td>
<td style="text-align:left;">
GABA B receptor activation
</td>
<td style="text-align:right;">
0.2750000
</td>
<td style="text-align:right;">
0.0001469
</td>
<td style="text-align:right;">
0.0068857
</td>
<td style="text-align:right;">
2.162050
</td>
<td style="text-align:right;">
0.0058769
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gng11/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-991365
</td>
<td style="text-align:left;">
Activation of GABAB receptors
</td>
<td style="text-align:right;">
0.2750000
</td>
<td style="text-align:right;">
0.0001469
</td>
<td style="text-align:right;">
0.0068857
</td>
<td style="text-align:right;">
2.162050
</td>
<td style="text-align:right;">
0.0058769
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gng11/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-76002
</td>
<td style="text-align:left;">
Platelet activation, signaling and aggregation
</td>
<td style="text-align:right;">
0.1440329
</td>
<td style="text-align:right;">
0.0002003
</td>
<td style="text-align:right;">
0.0087992
</td>
<td style="text-align:right;">
2.055556
</td>
<td style="text-align:right;">
0.0075100
</td>
<td style="text-align:left;">
Hgf/Cd109/Igf1/Pcdh7/Gnai1/Fn1/Gngt2/Pik3r6/Rasgrp2/Trpc6/Maged2/Islr/Gng11/Dgkb/Vegfc/Pecam1/Igf2/Gng2/Pros1/Arrb1/Sccpdh/Arrb2/Fyn/Tor4a/Habp4/Gnb5/Fam3c/Selenop/Gnb4/Apbb1ip/Lat/Lcp2/Lhfpl2/F2r/Plek
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-977443
</td>
<td style="text-align:left;">
GABA receptor activation
</td>
<td style="text-align:right;">
0.2321429
</td>
<td style="text-align:right;">
0.0002447
</td>
<td style="text-align:right;">
0.0101197
</td>
<td style="text-align:right;">
1.994832
</td>
<td style="text-align:right;">
0.0086370
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gabra4/Gng11/Gng2/Gnb5/Arhgef9/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-397795
</td>
<td style="text-align:left;">
G-protein beta:gamma signalling
</td>
<td style="text-align:right;">
0.3000000
</td>
<td style="text-align:right;">
0.0002876
</td>
<td style="text-align:right;">
0.0111282
</td>
<td style="text-align:right;">
1.953575
</td>
<td style="text-align:right;">
0.0094978
</td>
<td style="text-align:left;">
Gngt2/Pik3r6/Plcb1/Gng11/Gng2/Plcb2/Akt3/Gnb5/Gnb4
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1296071
</td>
<td style="text-align:left;">
Potassium Channels
</td>
<td style="text-align:right;">
0.1875000
</td>
<td style="text-align:right;">
0.0003137
</td>
<td style="text-align:right;">
0.0111282
</td>
<td style="text-align:right;">
1.953575
</td>
<td style="text-align:right;">
0.0094978
</td>
<td style="text-align:left;">
Kcnk3/Kcnb2/Kcnj3/Kcna6/Kcnd2/Kcnd3/Kcnq5/Gngt2/Gng11/Kcnma1/Gng2/Kcnc4/Kcng4/Gnb5/Gnb4/Gabbr1/Kcnq4/Kcnh1
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-202433
</td>
<td style="text-align:left;">
Generation of second messenger molecules
</td>
<td style="text-align:right;">
0.3684211
</td>
<td style="text-align:right;">
0.0003372
</td>
<td style="text-align:right;">
0.0111282
</td>
<td style="text-align:right;">
1.953575
</td>
<td style="text-align:right;">
0.0094978
</td>
<td style="text-align:left;">
Itk/Cd3g/Zap70/Lat/Lcp2/Grap2/Cd4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-418217
</td>
<td style="text-align:left;">
G beta:gamma signalling through PLC beta
</td>
<td style="text-align:right;">
0.3684211
</td>
<td style="text-align:right;">
0.0003372
</td>
<td style="text-align:right;">
0.0111282
</td>
<td style="text-align:right;">
1.953575
</td>
<td style="text-align:right;">
0.0094978
</td>
<td style="text-align:left;">
Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1474290
</td>
<td style="text-align:left;">
Collagen formation
</td>
<td style="text-align:right;">
0.2054795
</td>
<td style="text-align:right;">
0.0003519
</td>
<td style="text-align:right;">
0.0111282
</td>
<td style="text-align:right;">
1.953575
</td>
<td style="text-align:right;">
0.0094978
</td>
<td style="text-align:left;">
Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Loxl3/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1/Col4a5
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-202733
</td>
<td style="text-align:left;">
Cell surface interactions at the vascular wall
</td>
<td style="text-align:right;">
0.1910112
</td>
<td style="text-align:right;">
0.0003641
</td>
<td style="text-align:right;">
0.0111282
</td>
<td style="text-align:right;">
1.953575
</td>
<td style="text-align:right;">
0.0094978
</td>
<td style="text-align:left;">
Apob/Angpt1/Itga4/Cd44/Fn1/Jam3/Tek/Procr/Itgal/Esam/Sele/Pecam1/Pros1/Fyn/Cd84/Spn/Jam2
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-3000157
</td>
<td style="text-align:left;">
Laminin interactions
</td>
<td style="text-align:right;">
0.5000000
</td>
<td style="text-align:right;">
0.0004767
</td>
<td style="text-align:right;">
0.0136328
</td>
<td style="text-align:right;">
1.865417
</td>
<td style="text-align:right;">
0.0116353
</td>
<td style="text-align:left;">
Col4a6/Lama4/Nid1/Nid2/Col4a5
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-500657
</td>
<td style="text-align:left;">
Presynaptic function of Kainate receptors
</td>
<td style="text-align:right;">
0.3500000
</td>
<td style="text-align:right;">
0.0004848
</td>
<td style="text-align:right;">
0.0136328
</td>
<td style="text-align:right;">
1.865417
</td>
<td style="text-align:right;">
0.0116353
</td>
<td style="text-align:left;">
Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-111885
</td>
<td style="text-align:left;">
Opioid Signalling
</td>
<td style="text-align:right;">
0.2028986
</td>
<td style="text-align:right;">
0.0006283
</td>
<td style="text-align:right;">
0.0163874
</td>
<td style="text-align:right;">
1.785490
</td>
<td style="text-align:right;">
0.0139864
</td>
<td style="text-align:left;">
Gnai1/Pde1b/Adcy3/Gngt2/Plcb1/Adcy4/Adcy7/Gng11/Pde1a/Gng2/Plcb2/Gnb5/Gnb4/Prkar1b
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-4085001
</td>
<td style="text-align:left;">
Sialic acid metabolism
</td>
<td style="text-align:right;">
0.2727273
</td>
<td style="text-align:right;">
0.0006294
</td>
<td style="text-align:right;">
0.0163874
</td>
<td style="text-align:right;">
1.785490
</td>
<td style="text-align:right;">
0.0139864
</td>
<td style="text-align:left;">
St6gal2/St8sia2/St8sia4/Npl/St3gal2/St6galnac4/St8sia6/St6galnac6/St3gal4
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-8948216
</td>
<td style="text-align:left;">
Collagen chain trimerization
</td>
<td style="text-align:right;">
0.2571429
</td>
<td style="text-align:right;">
0.0010032
</td>
<td style="text-align:right;">
0.0251865
</td>
<td style="text-align:right;">
1.598832
</td>
<td style="text-align:right;">
0.0214963
</td>
<td style="text-align:left;">
Col11a1/Col26a1/Col14a1/Col16a1/Col13a1/Col5a2/Col15a1/Col23a1/Col18a1
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-451326
</td>
<td style="text-align:left;">
Activation of kainate receptors upon glutamate binding
</td>
<td style="text-align:right;">
0.2758621
</td>
<td style="text-align:right;">
0.0011595
</td>
<td style="text-align:right;">
0.0281070
</td>
<td style="text-align:right;">
1.551185
</td>
<td style="text-align:right;">
0.0239889
</td>
<td style="text-align:left;">
Gngt2/Plcb1/Gng11/Gng2/Plcb2/Dlg4/Gnb5/Gnb4
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-4086398
</td>
<td style="text-align:left;">
Ca2+ pathway
</td>
<td style="text-align:right;">
0.2325581
</td>
<td style="text-align:right;">
0.0012401
</td>
<td style="text-align:right;">
0.0290608
</td>
<td style="text-align:right;">
1.536692
</td>
<td style="text-align:right;">
0.0248030
</td>
<td style="text-align:left;">
Lef1/Wnt11/Gngt2/Plcb1/Gng11/Gng2/Plcb2/Fzd2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9013149
</td>
<td style="text-align:left;">
RAC1 GTPase cycle
</td>
<td style="text-align:right;">
0.1481481
</td>
<td style="text-align:right;">
0.0013126
</td>
<td style="text-align:right;">
0.0290804
</td>
<td style="text-align:right;">
1.536399
</td>
<td style="text-align:right;">
0.0248197
</td>
<td style="text-align:left;">
Nhs/Mcam/Fermt2/Wasf3/Arhgap45/Arhgef25/Arhgap25/Dock2/Dock10/Abi2/Sh3bp1/Arhgef15/Nckap1l/Arap3/Prex2/Arhgap10/Dlc1/Arhgap31/Ngef/Fgd5/Arhgap15/Cav1/Cyfip2/Pld2
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9012999
</td>
<td style="text-align:left;">
RHO GTPase cycle
</td>
<td style="text-align:right;">
0.1198980
</td>
<td style="text-align:right;">
0.0013237
</td>
<td style="text-align:right;">
0.0290804
</td>
<td style="text-align:right;">
1.536399
</td>
<td style="text-align:right;">
0.0248197
</td>
<td style="text-align:left;">
Nhs/Mcam/Plxnb1/Ccdc88a/Pcdh7/Fermt2/Wasf3/Arhgap45/Arhgef25/Akap12/Arhgap25/Dock2/Slitrk3/Myo9a/Plxnd1/Dock10/Abi2/Sh3bp1/Arhgap6/Arhgef15/Nckap1l/Arap3/Prex2/Arhgap10/Dlc1/Arhgef26/Arhgap28/Net1/Pde5a/Fgd1/Dlg5/Arhgap31/Arfgap3/Zap70/Cep97/Ngef/Arhgef9/Fgd5/Arhgap15/Elmo2/Cav1/Cyfip2/Pld2/Wdr6/Fmnl3/Fgd2/Plekhg5
</td>
<td style="text-align:right;">
47
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1630316
</td>
<td style="text-align:left;">
Glycosaminoglycan metabolism
</td>
<td style="text-align:right;">
0.1623932
</td>
<td style="text-align:right;">
0.0013730
</td>
<td style="text-align:right;">
0.0292500
</td>
<td style="text-align:right;">
1.533874
</td>
<td style="text-align:right;">
0.0249644
</td>
<td style="text-align:left;">
Gpc6/Chst2/Cd44/Cspg4/Gpc3/B4galt2/Has2/St3gal2/Chst1/Glb1l/Bgn/Dse/Xylt1/St3gal4/Arsb/Abcc5/Gpc2/Lum/Papss2
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9013148
</td>
<td style="text-align:left;">
CDC42 GTPase cycle
</td>
<td style="text-align:right;">
0.1807229
</td>
<td style="text-align:right;">
0.0014375
</td>
<td style="text-align:right;">
0.0297233
</td>
<td style="text-align:right;">
1.526903
</td>
<td style="text-align:right;">
0.0253684
</td>
<td style="text-align:left;">
Arhgap45/Arhgef25/Dock10/Arhgef15/Arap3/Prex2/Arhgap10/Dlc1/Arhgef26/Fgd1/Arhgap31/Ngef/Arhgef9/Cav1/Fgd2
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-392451
</td>
<td style="text-align:left;">
G beta:gamma signalling through PI3Kgamma
</td>
<td style="text-align:right;">
0.2916667
</td>
<td style="text-align:right;">
0.0016515
</td>
<td style="text-align:right;">
0.0331723
</td>
<td style="text-align:right;">
1.479225
</td>
<td style="text-align:right;">
0.0283120
</td>
<td style="text-align:left;">
Gngt2/Pik3r6/Gng11/Gng2/Akt3/Gnb5/Gnb4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-456926
</td>
<td style="text-align:left;">
Thrombin signalling through proteinase activated receptors (PARs)
</td>
<td style="text-align:right;">
0.2580645
</td>
<td style="text-align:right;">
0.0018586
</td>
<td style="text-align:right;">
0.0362946
</td>
<td style="text-align:right;">
1.440157
</td>
<td style="text-align:right;">
0.0309769
</td>
<td style="text-align:left;">
Gngt2/Gng11/Gng2/Arrb1/Arrb2/Gnb5/Gnb4/F2r
</td>
<td style="text-align:right;">
8
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dot-plot" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>react_dot=list()
upset=list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  react_dot[[x]] &lt;- ggplot(reactome_sig[[x]][1:12, ]) +
  geom_point(aes(x = GeneRatio, y = reorder(Description, GeneRatio), colour = logFDR, size = Count)) +
  scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits = c(0, NA)) +
  scale_size(range = c(1.5,5)) +
  # ggtitle(&quot;Reactome Pathways&quot;) +
  ylab(label = &quot;&quot;) +
  xlab(label = &quot;Gene Ratio&quot;) +
  labs(color = expression(&quot;-log&quot;[10] * &quot;FDR&quot;), size = &quot;Gene Counts&quot;)
  ggsave(filename = paste0(&quot;react_dot_&quot;, x, &quot;.svg&quot;), plot = react_dot[[x]] + pub, path = here::here(&quot;2_plots/reactome/&quot;), 
       width = 170, height = 130, units = &quot;mm&quot;)
  
  upset[[x]] &lt;- upsetplot(x = reactome[[x]], 9)
  ggsave(filename = paste0(&quot;upset_react_&quot;, fc[i], &quot;.svg&quot;), plot = upset[[x]], path = here::here(&quot;2_plots/reactome/&quot;))
}

react_dot[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/dotplot-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="upset" class="section level3">
<h3>Upset</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="fc1.1" class="section level2 tabset">
<h2 class="tabset">FC=1.1</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="enriched-pathways-1" class="section level3">
<h3>Enriched pathways</h3>
<pre class="r"><code>kable(x = reactome_sig[[p]]) %&gt;%
kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% 
scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFDR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
R-MMU-112316
</td>
<td style="text-align:left;">
Neuronal System
</td>
<td style="text-align:right;">
0.1619938
</td>
<td style="text-align:right;">
0.0000000
</td>
<td style="text-align:right;">
0.0000009
</td>
<td style="text-align:right;">
6.027692
</td>
<td style="text-align:right;">
0.0000008
</td>
<td style="text-align:left;">
Kcnk3/Kcnb2/Syn2/Kcnj3/Slc1a3/Gria4/Kcna6/Kcnd2/Ptprd/Kcnd3/Nlgn1/Epb41l3/Lrrc7/Abat/Gnai1/Lrrc4b/Kcnq5/Grin2b/Adcy3/Maoa/Shank2/Gngt2/Ppfia2/Plcb1/Slitrk3/Rps6ka2/Glrb/Gabra4/Dlg2/Adcy4/Ppfia4/Gng11/Nlgn3/Lin7a/Adcy7/Tspan7/Kcnma1/Kcnc4/Gjc1/Gng2/Plcb2/Lrfn2/Kcng4/Slc38a1/Dlg4/Gnb5/Arhgef9/Gnb4/Kcnq4/Gabbr1/Cacna1a/Cacnb4
</td>
<td style="text-align:right;">
52
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1474244
</td>
<td style="text-align:left;">
Extracellular matrix organization
</td>
<td style="text-align:right;">
0.1645022
</td>
<td style="text-align:right;">
0.0000002
</td>
<td style="text-align:right;">
0.0000561
</td>
<td style="text-align:right;">
4.251364
</td>
<td style="text-align:right;">
0.0000474
</td>
<td style="text-align:left;">
Col11a1/Mmp16/Col26a1/Itga4/P3h3/Col14a1/Capn6/Col4a6/Ddr2/Cd44/Lama4/Itgb7/Fn1/Mmp2/Col16a1/Icam1/Nid1/Jam3/Itga1/Col13a1/Loxl3/Itgal/Emilin1/Pecam1/Col5a2/Col15a1/Ltbp3/Mmp19/Bgn/P3h1/Adam19/Crtap/Efemp2/Col23a1/Col18a1/Itga8/Tmprss6/Itgb4
</td>
<td style="text-align:right;">
38
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-5173214
</td>
<td style="text-align:left;">
O-glycosylation of TSR domain-containing proteins
</td>
<td style="text-align:right;">
0.3636364
</td>
<td style="text-align:right;">
0.0000007
</td>
<td style="text-align:right;">
0.0001520
</td>
<td style="text-align:right;">
3.818039
</td>
<td style="text-align:right;">
0.0001285
</td>
<td style="text-align:left;">
Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Adamts10/Adamts7/Adamtsl1/Sbspon/Adamts17
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-112315
</td>
<td style="text-align:left;">
Transmission across Chemical Synapses
</td>
<td style="text-align:right;">
0.1658291
</td>
<td style="text-align:right;">
0.0000009
</td>
<td style="text-align:right;">
0.0001533
</td>
<td style="text-align:right;">
3.814372
</td>
<td style="text-align:right;">
0.0001296
</td>
<td style="text-align:left;">
Syn2/Kcnj3/Slc1a3/Gria4/Lrrc7/Abat/Gnai1/Grin2b/Adcy3/Maoa/Gngt2/Ppfia2/Plcb1/Rps6ka2/Glrb/Gabra4/Dlg2/Adcy4/Ppfia4/Gng11/Lin7a/Adcy7/Tspan7/Gng2/Plcb2/Slc38a1/Dlg4/Gnb5/Arhgef9/Gnb4/Gabbr1/Cacna1a/Cacnb4
</td>
<td style="text-align:right;">
33
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-109582
</td>
<td style="text-align:left;">
Hemostasis
</td>
<td style="text-align:right;">
0.1226994
</td>
<td style="text-align:right;">
0.0000024
</td>
<td style="text-align:right;">
0.0003200
</td>
<td style="text-align:right;">
3.494859
</td>
<td style="text-align:right;">
0.0002705
</td>
<td style="text-align:left;">
Apob/Hgf/Angpt1/Itga4/Cd109/Igf1/Tfpi/Nos3/Slc8a1/Cd44/Pcdh7/P2rx2/Gnai1/Fn1/Serpine2/Tek/Pde1b/Jam3/Gngt2/Pik3r6/Slc8a3/Rasgrp2/Dock2/Sh2b2/Kif26a/Dock10/Trpc6/Procr/Gng11/Islr/Esam/Itgal/Dgkb/Vegfc/Tubb4a/Pde1a/Maged2/Sele/Kif6/Igf2/Pecam1/Pros1/Gng2/Arrb1/Sccpdh/Pde5a/Cd84/Arrb2/Fyn/Fgr/Atp2b4/Tor4a/Selenop/Gnb5/Lat/Fam3c/Habp4/Apbb1ip/Gnb4/Kif16b
</td>
<td style="text-align:right;">
60
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-418346
</td>
<td style="text-align:left;">
Platelet homeostasis
</td>
<td style="text-align:right;">
0.2394366
</td>
<td style="text-align:right;">
0.0000029
</td>
<td style="text-align:right;">
0.0003220
</td>
<td style="text-align:right;">
3.492147
</td>
<td style="text-align:right;">
0.0002722
</td>
<td style="text-align:left;">
Apob/Nos3/Slc8a1/P2rx2/Pde1b/Gngt2/Slc8a3/Trpc6/Gng11/Pde1a/Pecam1/Gng2/Pde5a/Fgr/Atp2b4/Gnb5/Gnb4
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-5173105
</td>
<td style="text-align:left;">
O-linked glycosylation
</td>
<td style="text-align:right;">
0.1979167
</td>
<td style="text-align:right;">
0.0000153
</td>
<td style="text-align:right;">
0.0014582
</td>
<td style="text-align:right;">
2.836168
</td>
<td style="text-align:right;">
0.0012326
</td>
<td style="text-align:left;">
Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Galnt16/St6galnac4/St3gal2/Adamts10/Adamts7/Galntl6/Adamtsl1/B3gnt5/Sbspon/St3gal4/Large1/Adamts17
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-418597
</td>
<td style="text-align:left;">
G alpha (z) signalling events
</td>
<td style="text-align:right;">
0.3225806
</td>
<td style="text-align:right;">
0.0000202
</td>
<td style="text-align:right;">
0.0016123
</td>
<td style="text-align:right;">
2.792544
</td>
<td style="text-align:right;">
0.0013629
</td>
<td style="text-align:left;">
Rgs16/Adcy3/Gngt2/Adcy4/Gng11/Adcy7/Gng2/Gnaz/Gnb5/Gnb4
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-216083
</td>
<td style="text-align:left;">
Integrin cell surface interactions
</td>
<td style="text-align:right;">
0.2272727
</td>
<td style="text-align:right;">
0.0000218
</td>
<td style="text-align:right;">
0.0016123
</td>
<td style="text-align:right;">
2.792544
</td>
<td style="text-align:right;">
0.0013629
</td>
<td style="text-align:left;">
Itga4/Col4a6/Cd44/Itgb7/Fn1/Col16a1/Icam1/Jam3/Itga1/Col13a1/Itgal/Pecam1/Col5a2/Col18a1/Itga8
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-112314
</td>
<td style="text-align:left;">
Neurotransmitter receptors and postsynaptic signal transmission
</td>
<td style="text-align:right;">
0.1691176
</td>
<td style="text-align:right;">
0.0000302
</td>
<td style="text-align:right;">
0.0020055
</td>
<td style="text-align:right;">
2.697778
</td>
<td style="text-align:right;">
0.0016952
</td>
<td style="text-align:left;">
Kcnj3/Gria4/Lrrc7/Gnai1/Grin2b/Adcy3/Gngt2/Plcb1/Rps6ka2/Glrb/Gabra4/Dlg2/Adcy4/Gng11/Adcy7/Tspan7/Gng2/Plcb2/Dlg4/Gnb5/Arhgef9/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-977444
</td>
<td style="text-align:left;">
GABA B receptor activation
</td>
<td style="text-align:right;">
0.2750000
</td>
<td style="text-align:right;">
0.0000420
</td>
<td style="text-align:right;">
0.0023266
</td>
<td style="text-align:right;">
2.633279
</td>
<td style="text-align:right;">
0.0019666
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Gng11/Adcy7/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-991365
</td>
<td style="text-align:left;">
Activation of GABAB receptors
</td>
<td style="text-align:right;">
0.2750000
</td>
<td style="text-align:right;">
0.0000420
</td>
<td style="text-align:right;">
0.0023266
</td>
<td style="text-align:right;">
2.633279
</td>
<td style="text-align:right;">
0.0019666
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Gng11/Adcy7/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-977443
</td>
<td style="text-align:left;">
GABA receptor activation
</td>
<td style="text-align:right;">
0.2321429
</td>
<td style="text-align:right;">
0.0000606
</td>
<td style="text-align:right;">
0.0031021
</td>
<td style="text-align:right;">
2.508347
</td>
<td style="text-align:right;">
0.0026221
</td>
<td style="text-align:left;">
Kcnj3/Gnai1/Adcy3/Gngt2/Gabra4/Adcy4/Gng11/Adcy7/Gng2/Gnb5/Arhgef9/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-6794362
</td>
<td style="text-align:left;">
Protein-protein interactions at synapses
</td>
<td style="text-align:right;">
0.2121212
</td>
<td style="text-align:right;">
0.0000917
</td>
<td style="text-align:right;">
0.0043570
</td>
<td style="text-align:right;">
2.360809
</td>
<td style="text-align:right;">
0.0036829
</td>
<td style="text-align:left;">
Gria4/Ptprd/Nlgn1/Epb41l3/Lrrc4b/Grin2b/Shank2/Ppfia2/Slitrk3/Dlg2/Ppfia4/Nlgn3/Lrfn2/Dlg4
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-397795
</td>
<td style="text-align:left;">
G-protein beta:gamma signalling
</td>
<td style="text-align:right;">
0.3000000
</td>
<td style="text-align:right;">
0.0001003
</td>
<td style="text-align:right;">
0.0044487
</td>
<td style="text-align:right;">
2.351765
</td>
<td style="text-align:right;">
0.0037604
</td>
<td style="text-align:left;">
Gngt2/Pik3r6/Plcb1/Gng11/Gng2/Plcb2/Akt3/Gnb5/Gnb4
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1650814
</td>
<td style="text-align:left;">
Collagen biosynthesis and modifying enzymes
</td>
<td style="text-align:right;">
0.2166667
</td>
<td style="text-align:right;">
0.0001295
</td>
<td style="text-align:right;">
0.0053807
</td>
<td style="text-align:right;">
2.269160
</td>
<td style="text-align:right;">
0.0045482
</td>
<td style="text-align:left;">
Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-418217
</td>
<td style="text-align:left;">
G beta:gamma signalling through PLC beta
</td>
<td style="text-align:right;">
0.3684211
</td>
<td style="text-align:right;">
0.0001426
</td>
<td style="text-align:right;">
0.0055776
</td>
<td style="text-align:right;">
2.253554
</td>
<td style="text-align:right;">
0.0047146
</td>
<td style="text-align:left;">
Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-416482
</td>
<td style="text-align:left;">
G alpha (12/13) signalling events
</td>
<td style="text-align:right;">
0.2000000
</td>
<td style="text-align:right;">
0.0001787
</td>
<td style="text-align:right;">
0.0063804
</td>
<td style="text-align:right;">
2.195151
</td>
<td style="text-align:right;">
0.0053932
</td>
<td style="text-align:left;">
Plxnb1/Adra1a/Arhgef25/Gngt2/Gng11/Arhgef15/Gng2/Arhgef26/Net1/Fgd1/Gnb5/Ngef/Arhgef9/Gnb4
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1296071
</td>
<td style="text-align:left;">
Potassium Channels
</td>
<td style="text-align:right;">
0.1770833
</td>
<td style="text-align:right;">
0.0001823
</td>
<td style="text-align:right;">
0.0063804
</td>
<td style="text-align:right;">
2.195151
</td>
<td style="text-align:right;">
0.0053932
</td>
<td style="text-align:left;">
Kcnk3/Kcnb2/Kcnj3/Kcna6/Kcnd2/Kcnd3/Kcnq5/Gngt2/Gng11/Kcnma1/Kcnc4/Gng2/Kcng4/Gnb5/Gnb4/Kcnq4/Gabbr1
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-500657
</td>
<td style="text-align:left;">
Presynaptic function of Kainate receptors
</td>
<td style="text-align:right;">
0.3500000
</td>
<td style="text-align:right;">
0.0002068
</td>
<td style="text-align:right;">
0.0068768
</td>
<td style="text-align:right;">
2.162615
</td>
<td style="text-align:right;">
0.0058127
</td>
<td style="text-align:left;">
Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-4085001
</td>
<td style="text-align:left;">
Sialic acid metabolism
</td>
<td style="text-align:right;">
0.2727273
</td>
<td style="text-align:right;">
0.0002258
</td>
<td style="text-align:right;">
0.0071491
</td>
<td style="text-align:right;">
2.145752
</td>
<td style="text-align:right;">
0.0060429
</td>
<td style="text-align:left;">
St6gal2/St8sia2/St8sia4/Npl/St6galnac4/St3gal2/St8sia6/St6galnac6/St3gal4
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1474290
</td>
<td style="text-align:left;">
Collagen formation
</td>
<td style="text-align:right;">
0.1917808
</td>
<td style="text-align:right;">
0.0002839
</td>
<td style="text-align:right;">
0.0085803
</td>
<td style="text-align:right;">
2.066500
</td>
<td style="text-align:right;">
0.0072526
</td>
<td style="text-align:left;">
Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Loxl3/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-76002
</td>
<td style="text-align:left;">
Platelet activation, signaling and aggregation
</td>
<td style="text-align:right;">
0.1275720
</td>
<td style="text-align:right;">
0.0003594
</td>
<td style="text-align:right;">
0.0101540
</td>
<td style="text-align:right;">
1.993364
</td>
<td style="text-align:right;">
0.0085829
</td>
<td style="text-align:left;">
Hgf/Cd109/Igf1/Pcdh7/Gnai1/Fn1/Gngt2/Pik3r6/Rasgrp2/Trpc6/Gng11/Islr/Dgkb/Vegfc/Maged2/Igf2/Pecam1/Pros1/Gng2/Arrb1/Sccpdh/Arrb2/Fyn/Tor4a/Selenop/Gnb5/Lat/Fam3c/Habp4/Apbb1ip/Gnb4
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-8948216
</td>
<td style="text-align:left;">
Collagen chain trimerization
</td>
<td style="text-align:right;">
0.2571429
</td>
<td style="text-align:right;">
0.0003665
</td>
<td style="text-align:right;">
0.0101540
</td>
<td style="text-align:right;">
1.993364
</td>
<td style="text-align:right;">
0.0085829
</td>
<td style="text-align:left;">
Col11a1/Col26a1/Col14a1/Col16a1/Col13a1/Col5a2/Col15a1/Col23a1/Col18a1
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-4086398
</td>
<td style="text-align:left;">
Ca2+ pathway
</td>
<td style="text-align:right;">
0.2325581
</td>
<td style="text-align:right;">
0.0004216
</td>
<td style="text-align:right;">
0.0112147
</td>
<td style="text-align:right;">
1.950213
</td>
<td style="text-align:right;">
0.0094794
</td>
<td style="text-align:left;">
Lef1/Wnt11/Gngt2/Plcb1/Gng11/Gng2/Plcb2/Fzd2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9013149
</td>
<td style="text-align:left;">
RAC1 GTPase cycle
</td>
<td style="text-align:right;">
0.1419753
</td>
<td style="text-align:right;">
0.0004551
</td>
<td style="text-align:right;">
0.0114220
</td>
<td style="text-align:right;">
1.942257
</td>
<td style="text-align:right;">
0.0096547
</td>
<td style="text-align:left;">
Nhs/Mcam/Fermt2/Wasf3/Arhgap45/Arhgef25/Arhgap25/Dock2/Dock10/Abi2/Sh3bp1/Arhgef15/Nckap1l/Arap3/Prex2/Arhgap10/Dlc1/Arhgap31/Ngef/Fgd5/Arhgap15/Cav1/Cyfip2
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-451326
</td>
<td style="text-align:left;">
Activation of kainate receptors upon glutamate binding
</td>
<td style="text-align:right;">
0.2758621
</td>
<td style="text-align:right;">
0.0004638
</td>
<td style="text-align:right;">
0.0114220
</td>
<td style="text-align:right;">
1.942257
</td>
<td style="text-align:right;">
0.0096547
</td>
<td style="text-align:left;">
Gngt2/Plcb1/Gng11/Gng2/Plcb2/Dlg4/Gnb5/Gnb4
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-111885
</td>
<td style="text-align:left;">
Opioid Signalling
</td>
<td style="text-align:right;">
0.1884058
</td>
<td style="text-align:right;">
0.0005559
</td>
<td style="text-align:right;">
0.0132027
</td>
<td style="text-align:right;">
1.879337
</td>
<td style="text-align:right;">
0.0111599
</td>
<td style="text-align:left;">
Gnai1/Pde1b/Adcy3/Gngt2/Plcb1/Adcy4/Gng11/Adcy7/Pde1a/Gng2/Plcb2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-392451
</td>
<td style="text-align:left;">
G beta:gamma signalling through PI3Kgamma
</td>
<td style="text-align:right;">
0.2916667
</td>
<td style="text-align:right;">
0.0007301
</td>
<td style="text-align:right;">
0.0164497
</td>
<td style="text-align:right;">
1.783841
</td>
<td style="text-align:right;">
0.0139045
</td>
<td style="text-align:left;">
Gngt2/Pik3r6/Gng11/Gng2/Akt3/Gnb5/Gnb4
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-202733
</td>
<td style="text-align:left;">
Cell surface interactions at the vascular wall
</td>
<td style="text-align:right;">
0.1685393
</td>
<td style="text-align:right;">
0.0007421
</td>
<td style="text-align:right;">
0.0164497
</td>
<td style="text-align:right;">
1.783841
</td>
<td style="text-align:right;">
0.0139045
</td>
<td style="text-align:left;">
Apob/Angpt1/Itga4/Cd44/Fn1/Tek/Jam3/Procr/Esam/Itgal/Sele/Pecam1/Pros1/Cd84/Fyn
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-446728
</td>
<td style="text-align:left;">
Cell junction organization
</td>
<td style="text-align:right;">
0.1875000
</td>
<td style="text-align:right;">
0.0009405
</td>
<td style="text-align:right;">
0.0201758
</td>
<td style="text-align:right;">
1.695169
</td>
<td style="text-align:right;">
0.0170540
</td>
<td style="text-align:left;">
Sdk2/Fermt2/Cdh2/Cdh6/Cdh13/Cdh10/Cdh5/Cdh11/Pard6g/Parvb/Flnc/Itgb4
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9013148
</td>
<td style="text-align:left;">
CDC42 GTPase cycle
</td>
<td style="text-align:right;">
0.1686747
</td>
<td style="text-align:right;">
0.0010945
</td>
<td style="text-align:right;">
0.0227450
</td>
<td style="text-align:right;">
1.643114
</td>
<td style="text-align:right;">
0.0192257
</td>
<td style="text-align:left;">
Arhgap45/Arhgef25/Dock10/Arhgef15/Arap3/Prex2/Arhgef26/Arhgap10/Dlc1/Fgd1/Arhgap31/Ngef/Arhgef9/Cav1
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-397014
</td>
<td style="text-align:left;">
Muscle contraction
</td>
<td style="text-align:right;">
0.1372549
</td>
<td style="text-align:right;">
0.0012365
</td>
<td style="text-align:right;">
0.0249183
</td>
<td style="text-align:right;">
1.603481
</td>
<td style="text-align:right;">
0.0210627
</td>
<td style="text-align:left;">
Kcnk3/Cacna1c/Kcnd2/Ryr1/Kcnd3/Slc8a1/Tmod2/Gucy1a2/Slc8a3/Itga1/Gucy1b1/Mme/Gucy1a1/Myl3/Cacng7/Kcnip3/Asph/Ryr3/Pde5a/Atp2b4/Atp1a2
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1296041
</td>
<td style="text-align:left;">
Activation of G protein gated Potassium channels
</td>
<td style="text-align:right;">
0.2592593
</td>
<td style="text-align:right;">
0.0015718
</td>
<td style="text-align:right;">
0.0290350
</td>
<td style="text-align:right;">
1.537078
</td>
<td style="text-align:right;">
0.0245425
</td>
<td style="text-align:left;">
Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1296059
</td>
<td style="text-align:left;">
G protein gated Potassium channels
</td>
<td style="text-align:right;">
0.2592593
</td>
<td style="text-align:right;">
0.0015718
</td>
<td style="text-align:right;">
0.0290350
</td>
<td style="text-align:right;">
1.537078
</td>
<td style="text-align:right;">
0.0245425
</td>
<td style="text-align:left;">
Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-997272
</td>
<td style="text-align:left;">
Inhibition of voltage gated Ca2+ channels via Gbeta/gamma subunits
</td>
<td style="text-align:right;">
0.2592593
</td>
<td style="text-align:right;">
0.0015718
</td>
<td style="text-align:right;">
0.0290350
</td>
<td style="text-align:right;">
1.537078
</td>
<td style="text-align:right;">
0.0245425
</td>
<td style="text-align:left;">
Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gnb4/Gabbr1
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1630316
</td>
<td style="text-align:left;">
Glycosaminoglycan metabolism
</td>
<td style="text-align:right;">
0.1452991
</td>
<td style="text-align:right;">
0.0018891
</td>
<td style="text-align:right;">
0.0339521
</td>
<td style="text-align:right;">
1.469134
</td>
<td style="text-align:right;">
0.0286987
</td>
<td style="text-align:left;">
Gpc6/Chst2/Cd44/Cspg4/Gpc3/B4galt2/Has2/St3gal2/Chst1/Glb1l/Bgn/Xylt1/Dse/St3gal4/Arsb/Gpc2/Abcc5
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-392170
</td>
<td style="text-align:left;">
ADP signalling through P2Y purinoceptor 12
</td>
<td style="text-align:right;">
0.2857143
</td>
<td style="text-align:right;">
0.0019821
</td>
<td style="text-align:right;">
0.0346868
</td>
<td style="text-align:right;">
1.459836
</td>
<td style="text-align:right;">
0.0293197
</td>
<td style="text-align:left;">
Gnai1/Gngt2/Gng11/Gng2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-400042
</td>
<td style="text-align:left;">
Adrenaline,noradrenaline inhibits insulin secretion
</td>
<td style="text-align:right;">
0.2727273
</td>
<td style="text-align:right;">
0.0025737
</td>
<td style="text-align:right;">
0.0437866
</td>
<td style="text-align:right;">
1.358659
</td>
<td style="text-align:right;">
0.0370115
</td>
<td style="text-align:left;">
Gnai1/Gngt2/Gng11/Gng2/Gnb5/Gnb4
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9012999
</td>
<td style="text-align:left;">
RHO GTPase cycle
</td>
<td style="text-align:right;">
0.1045918
</td>
<td style="text-align:right;">
0.0026338
</td>
<td style="text-align:right;">
0.0437866
</td>
<td style="text-align:right;">
1.358659
</td>
<td style="text-align:right;">
0.0370115
</td>
<td style="text-align:left;">
Nhs/Mcam/Plxnb1/Ccdc88a/Pcdh7/Fermt2/Wasf3/Arhgap45/Arhgef25/Akap12/Arhgap25/Dock2/Slitrk3/Plxnd1/Dock10/Abi2/Myo9a/Sh3bp1/Arhgef15/Arhgap6/Nckap1l/Arap3/Prex2/Arhgef26/Arhgap10/Arhgap28/Pde5a/Dlc1/Net1/Fgd1/Zap70/Dlg5/Arhgap31/Ngef/Fgd5/Arhgef9/Cep97/Arhgap15/Arfgap3/Cav1/Cyfip2
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-416476
</td>
<td style="text-align:left;">
G alpha (q) signalling events
</td>
<td style="text-align:right;">
0.1218274
</td>
<td style="text-align:right;">
0.0030033
</td>
<td style="text-align:right;">
0.0485967
</td>
<td style="text-align:right;">
1.313393
</td>
<td style="text-align:right;">
0.0410774
</td>
<td style="text-align:left;">
Adra1a/Rgs16/Ednrb/Lpar4/Arhgef25/Gngt2/Ptgfr/Edn3/Plcb1/Rps6ka2/Trpc6/Gng11/Dgkb/Rgs5/Agtr1a/Ednra/Bdkrb2/Xcl1/Htr2a/Gng2/Plcb2/Gnb5/Prokr1/Gnb4
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-373080
</td>
<td style="text-align:left;">
Class B/2 (Secretin family receptors)
</td>
<td style="text-align:right;">
0.1818182
</td>
<td style="text-align:right;">
0.0031179
</td>
<td style="text-align:right;">
0.0485967
</td>
<td style="text-align:right;">
1.313393
</td>
<td style="text-align:right;">
0.0410774
</td>
<td style="text-align:left;">
Vipr2/Gngt2/Calca/Pth1r/Gng11/Adcyap1r1/Gng2/Calcrl/Gnb5/Gnb4
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1500931
</td>
<td style="text-align:left;">
Cell-Cell communication
</td>
<td style="text-align:right;">
0.1566265
</td>
<td style="text-align:right;">
0.0032224
</td>
<td style="text-align:right;">
0.0485967
</td>
<td style="text-align:right;">
1.313393
</td>
<td style="text-align:right;">
0.0410774
</td>
<td style="text-align:left;">
Sdk2/Fermt2/Cdh2/Cdh6/Cdh13/Cdh10/Cdh5/Cdh11/Pard6g/Parvb/Flnc/Fyn/Itgb4
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1971475
</td>
<td style="text-align:left;">
A tetrasaccharide linker sequence is required for GAG synthesis
</td>
<td style="text-align:right;">
0.2608696
</td>
<td style="text-align:right;">
0.0032885
</td>
<td style="text-align:right;">
0.0485967
</td>
<td style="text-align:right;">
1.313393
</td>
<td style="text-align:right;">
0.0410774
</td>
<td style="text-align:left;">
Gpc6/Cspg4/Gpc3/Bgn/Xylt1/Gpc2
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-975576
</td>
<td style="text-align:left;">
N-glycan antennae elongation in the medial/trans-Golgi
</td>
<td style="text-align:right;">
0.2608696
</td>
<td style="text-align:right;">
0.0032885
</td>
<td style="text-align:right;">
0.0485967
</td>
<td style="text-align:right;">
1.313393
</td>
<td style="text-align:right;">
0.0410774
</td>
<td style="text-align:left;">
St8sia2/Man2a2/Chst10/B4galt2/St8sia6/St3gal4
</td>
<td style="text-align:right;">
6
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dot-plot-1" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>react_dot[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-1" class="section level3">
<h3>Upset</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="fc1.5" class="section level2 tabset">
<h2 class="tabset">FC=1.5</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="enriched-pathways-2" class="section level3">
<h3>Enriched pathways</h3>
<pre class="r"><code>kable(x = reactome_sig[[p]]) %&gt;%
kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;% 
scroll_box(height = &quot;600px&quot;)</code></pre>
<div
style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:600px; ">
<table class="table table-striped table-hover" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
Description
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
GeneRatio
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
pvalue
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
p.adjust
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
logFDR
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
qvalue
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
geneID
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;">
Count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
R-MMU-112316
</td>
<td style="text-align:left;">
Neuronal System
</td>
<td style="text-align:right;">
0.0623053
</td>
<td style="text-align:right;">
0.0000008
</td>
<td style="text-align:right;">
0.0002601
</td>
<td style="text-align:right;">
3.584822
</td>
<td style="text-align:right;">
0.0002228
</td>
<td style="text-align:left;">
Kcnk3/Kcnb2/Syn2/Kcnj3/Slc1a3/Kcnd2/Gria4/Kcna6/Kcnd3/Lrrc7/Nlgn1/Ptprd/Abat/Epb41l3/Lrrc4b/Gnai1/Kcnq5/Ppfia2/Grin2b/Gabra4
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-6794362
</td>
<td style="text-align:left;">
Protein-protein interactions at synapses
</td>
<td style="text-align:right;">
0.1060606
</td>
<td style="text-align:right;">
0.0001513
</td>
<td style="text-align:right;">
0.0243552
</td>
<td style="text-align:right;">
1.613409
</td>
<td style="text-align:right;">
0.0208599
</td>
<td style="text-align:left;">
Gria4/Nlgn1/Ptprd/Epb41l3/Lrrc4b/Ppfia2/Grin2b
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-5173214
</td>
<td style="text-align:left;">
O-glycosylation of TSR domain-containing proteins
</td>
<td style="text-align:right;">
0.1515152
</td>
<td style="text-align:right;">
0.0002554
</td>
<td style="text-align:right;">
0.0258090
</td>
<td style="text-align:right;">
1.588229
</td>
<td style="text-align:right;">
0.0221051
</td>
<td style="text-align:left;">
Adamts20/Sema5b/Adamts16/Thsd7b/Adamts19
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-422475
</td>
<td style="text-align:left;">
Axon guidance
</td>
<td style="text-align:right;">
0.0522088
</td>
<td style="text-align:right;">
0.0004433
</td>
<td style="text-align:right;">
0.0258090
</td>
<td style="text-align:right;">
1.588229
</td>
<td style="text-align:right;">
0.0221051
</td>
<td style="text-align:left;">
Gap43/Epha8/Efnb3/Dcc/St8sia2/Plxnb1/St8sia4/Dab1/Evl/Grin2b/Sema7a/Mmp2/Cxcl12
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-9675108
</td>
<td style="text-align:left;">
Nervous system development
</td>
<td style="text-align:right;">
0.0520000
</td>
<td style="text-align:right;">
0.0004606
</td>
<td style="text-align:right;">
0.0258090
</td>
<td style="text-align:right;">
1.588229
</td>
<td style="text-align:right;">
0.0221051
</td>
<td style="text-align:left;">
Gap43/Epha8/Efnb3/Dcc/St8sia2/Plxnb1/St8sia4/Dab1/Evl/Grin2b/Sema7a/Mmp2/Cxcl12
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1474228
</td>
<td style="text-align:left;">
Degradation of the extracellular matrix
</td>
<td style="text-align:right;">
0.0747664
</td>
<td style="text-align:right;">
0.0005833
</td>
<td style="text-align:right;">
0.0258090
</td>
<td style="text-align:right;">
1.588229
</td>
<td style="text-align:right;">
0.0221051
</td>
<td style="text-align:left;">
Col11a1/Mmp16/Col26a1/Capn6/Col4a6/Cd44/Mmp2/Nid1
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-3000157
</td>
<td style="text-align:left;">
Laminin interactions
</td>
<td style="text-align:right;">
0.3000000
</td>
<td style="text-align:right;">
0.0005908
</td>
<td style="text-align:right;">
0.0258090
</td>
<td style="text-align:right;">
1.588229
</td>
<td style="text-align:right;">
0.0221051
</td>
<td style="text-align:left;">
Col4a6/Lama4/Nid1
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1296072
</td>
<td style="text-align:left;">
Voltage gated Potassium channels
</td>
<td style="text-align:right;">
0.1250000
</td>
<td style="text-align:right;">
0.0006412
</td>
<td style="text-align:right;">
0.0258090
</td>
<td style="text-align:right;">
1.588229
</td>
<td style="text-align:right;">
0.0221051
</td>
<td style="text-align:left;">
Kcnb2/Kcnd2/Kcna6/Kcnd3/Kcnq5
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1474244
</td>
<td style="text-align:left;">
Extracellular matrix organization
</td>
<td style="text-align:right;">
0.0519481
</td>
<td style="text-align:right;">
0.0007709
</td>
<td style="text-align:right;">
0.0259472
</td>
<td style="text-align:right;">
1.585909
</td>
<td style="text-align:right;">
0.0222235
</td>
<td style="text-align:left;">
Col11a1/Mmp16/Col26a1/Col14a1/Capn6/Col4a6/Itga4/P3h3/Cd44/Mmp2/Lama4/Nid1
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-975634
</td>
<td style="text-align:left;">
Retinoid metabolism and transport
</td>
<td style="text-align:right;">
0.1190476
</td>
<td style="text-align:right;">
0.0008058
</td>
<td style="text-align:right;">
0.0259472
</td>
<td style="text-align:right;">
1.585909
</td>
<td style="text-align:right;">
0.0222235
</td>
<td style="text-align:left;">
Gpc6/Apob/Ttr/Gpc3/Lrp2
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-6806667
</td>
<td style="text-align:left;">
Metabolism of fat-soluble vitamins
</td>
<td style="text-align:right;">
0.1086957
</td>
<td style="text-align:right;">
0.0012271
</td>
<td style="text-align:right;">
0.0359194
</td>
<td style="text-align:right;">
1.444671
</td>
<td style="text-align:right;">
0.0307646
</td>
<td style="text-align:left;">
Gpc6/Apob/Ttr/Gpc3/Lrp2
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
R-MMU-1296071
</td>
<td style="text-align:left;">
Potassium Channels
</td>
<td style="text-align:right;">
0.0729167
</td>
<td style="text-align:right;">
0.0014909
</td>
<td style="text-align:right;">
0.0400069
</td>
<td style="text-align:right;">
1.397865
</td>
<td style="text-align:right;">
0.0342655
</td>
<td style="text-align:left;">
Kcnk3/Kcnb2/Kcnj3/Kcnd2/Kcna6/Kcnd3/Kcnq5
</td>
<td style="text-align:right;">
7
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dot-plot-2" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>react_dot[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="upset-2" class="section level3">
<h3>Upset</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="export-data" class="section level1">
<h1>Export Data</h1>
<pre class="r"><code># save to csv
writexl::write_xlsx(x = reactome_all, here::here(&quot;3_output/reactome_all.xlsx&quot;))
writexl::write_xlsx(x = reactome_sig, here::here(&quot;3_output/reactome_sig.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.2.1 (2022-06-23 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.utf8  LC_CTYPE=English_Australia.utf8   
[3] LC_MONETARY=English_Australia.utf8 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.utf8    

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ReactomePA_1.40.0     enrichplot_1.16.2     org.Mm.eg.db_3.15.0  
 [4] AnnotationDbi_1.58.0  IRanges_2.30.1        S4Vectors_0.34.0     
 [7] Biobase_2.56.0        BiocGenerics_0.42.0   clusterProfiler_4.4.4
[10] Glimma_2.6.0          edgeR_3.38.4          limma_3.52.4         
[13] ggrepel_0.9.1         ggbiplot_0.55         scales_1.2.1         
[16] plyr_1.8.7            pheatmap_1.0.12       cowplot_1.1.1        
[19] pander_0.6.5          kableExtra_1.3.4      forcats_0.5.2        
[22] stringr_1.4.1         purrr_0.3.5           tidyr_1.2.1          
[25] ggplot2_3.3.6         tidyverse_1.3.2       reshape2_1.4.4       
[28] tibble_3.1.8          readr_2.1.3           magrittr_2.0.3       
[31] dplyr_1.0.10         

loaded via a namespace (and not attached):
  [1] utf8_1.2.2                  tidyselect_1.2.0           
  [3] RSQLite_2.2.18              htmlwidgets_1.5.4          
  [5] BiocParallel_1.30.3         scatterpie_0.1.8           
  [7] munsell_0.5.0               ragg_1.2.3                 
  [9] codetools_0.2-18            withr_2.5.0                
 [11] colorspace_2.0-3            GOSemSim_2.22.0            
 [13] highr_0.9                   knitr_1.40                 
 [15] rstudioapi_0.14             DOSE_3.22.1                
 [17] labeling_0.4.2              MatrixGenerics_1.8.1       
 [19] git2r_0.30.1                GenomeInfoDbData_1.2.8     
 [21] polyclip_1.10-0             bit64_4.0.5                
 [23] farver_2.1.1                rprojroot_2.0.3            
 [25] downloader_0.4              treeio_1.20.2              
 [27] vctrs_0.4.2                 generics_0.1.3             
 [29] xfun_0.33                   R6_2.5.1                   
 [31] GenomeInfoDb_1.32.4         graphlayouts_0.8.2         
 [33] locfit_1.5-9.6              bitops_1.0-7               
 [35] cachem_1.0.6                fgsea_1.22.0               
 [37] gridGraphics_0.5-1          DelayedArray_0.22.0        
 [39] assertthat_0.2.1            promises_1.2.0.1           
 [41] ggraph_2.1.0                googlesheets4_1.0.1        
 [43] gtable_0.3.1                tidygraph_1.2.2            
 [45] workflowr_1.7.0             rlang_1.0.6                
 [47] genefilter_1.78.0           systemfonts_1.0.4          
 [49] splines_4.2.1               lazyeval_0.2.2             
 [51] gargle_1.2.1                broom_1.0.1                
 [53] yaml_2.3.5                  modelr_0.1.9               
 [55] backports_1.4.1             httpuv_1.6.6               
 [57] qvalue_2.28.0               tools_4.2.1                
 [59] ggplotify_0.1.0             ellipsis_0.3.2             
 [61] jquerylib_0.1.4             RColorBrewer_1.1-3         
 [63] Rcpp_1.0.9                  zlibbioc_1.42.0            
 [65] RCurl_1.98-1.9              viridis_0.6.2              
 [67] SummarizedExperiment_1.26.1 haven_2.5.1                
 [69] fs_1.5.2                    here_1.0.1                 
 [71] data.table_1.14.2           DO.db_2.9                  
 [73] reactome.db_1.81.0          reprex_2.0.2               
 [75] googledrive_2.0.0           whisker_0.4                
 [77] matrixStats_0.62.0          hms_1.1.2                  
 [79] patchwork_1.1.2             evaluate_0.17              
 [81] xtable_1.8-4                XML_3.99-0.11              
 [83] readxl_1.4.1                gridExtra_2.3              
 [85] ggupset_0.3.0               compiler_4.2.1             
 [87] writexl_1.4.0               shadowtext_0.1.2           
 [89] crayon_1.5.2                htmltools_0.5.3            
 [91] ggfun_0.0.7                 later_1.3.0                
 [93] tzdb_0.3.0                  geneplotter_1.74.0         
 [95] aplot_0.1.8                 lubridate_1.8.0            
 [97] DBI_1.1.3                   tweenr_2.0.2               
 [99] dbplyr_2.2.1                rappdirs_0.3.3             
[101] MASS_7.3-57                 Matrix_1.5-1               
[103] cli_3.4.1                   parallel_4.2.1             
[105] igraph_1.3.5                GenomicRanges_1.48.0       
[107] pkgconfig_2.0.3             xml2_1.3.3                 
[109] ggtree_3.4.4                svglite_2.1.0              
[111] annotate_1.74.0             bslib_0.4.0                
[113] webshot_0.5.4               XVector_0.36.0             
[115] rvest_1.0.3                 yulab.utils_0.0.5          
[117] digest_0.6.29               graph_1.74.0               
[119] Biostrings_2.64.1           rmarkdown_2.17             
[121] cellranger_1.1.0            fastmatch_1.1-3            
[123] tidytree_0.4.1              graphite_1.42.0            
[125] lifecycle_1.0.3             nlme_3.1-157               
[127] jsonlite_1.8.2              viridisLite_0.4.1          
[129] fansi_1.0.3                 pillar_1.8.1               
[131] lattice_0.20-45             KEGGREST_1.36.3            
[133] fastmap_1.1.0               httr_1.4.4                 
[135] survival_3.3-1              GO.db_3.15.0               
[137] glue_1.6.2                  png_0.1-7                  
[139] bit_4.0.4                   ggforce_0.4.1              
[141] stringi_1.7.8               sass_0.4.2                 
[143] blob_1.2.3                  textshaping_0.3.6          
[145] DESeq2_1.36.0               memoise_2.0.1              
[147] ape_5.6-2                  </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
