<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Ha M. Tran" />


<title>Reactome Analysis</title>

<script src="site_libs/header-attrs-2.24/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.6.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.29/datatables.js"></script>
<link href="site_libs/dt-core-1.13.4/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.13.4/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.13.4/js/jquery.dataTables.min.js"></script>
<script src="site_libs/jszip-1.13.4/jszip.min.js"></script>
<script src="site_libs/pdfmake-1.13.4/pdfmake.js"></script>
<script src="site_libs/pdfmake-1.13.4/vfs_fonts.js"></script>
<link href="site_libs/dt-ext-buttons-1.13.4/css/buttons.dataTables.min.css" rel="stylesheet" />
<script src="site_libs/dt-ext-buttons-1.13.4/js/dataTables.buttons.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.13.4/js/buttons.html5.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.13.4/js/buttons.colVis.min.js"></script>
<script src="site_libs/dt-ext-buttons-1.13.4/js/buttons.print.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Mouse endometial transcriptome 2023</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="setUp.html">Pre-processing</a>
</li>
<li>
  <a href="deAnalysis.html">DGE Analysis</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Functional Enrichment Analysis
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="go.html">Gene Ontology</a>
    </li>
    <li>
      <a href="kegg.html">KEGG</a>
    </li>
    <li>
      <a href="reactome.html">Reactome</a>
    </li>
    <li>
      <a href="extraFigures.html">IPA</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Reactome Analysis</h1>
<h4 class="author">Ha M. Tran</h4>
<h4 class="date">22/08/2021</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-09-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong>
<code>Mouse_endometrial_transcriptome_2023/1_analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed12345code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(12345)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed12345code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(12345)</code> was run prior to running the
code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135Mouseendometrialtranscriptome2023tree18c64635cc1b08f42233ef4416cc5f1869e6e3e6targetblank18c6463a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/tree/18c64635cc1b08f42233ef4416cc5f1869e6e3e6" target="_blank">18c6463</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomtranmanhha135Mouseendometrialtranscriptome2023tree18c64635cc1b08f42233ef4416cc5f1869e6e3e6targetblank18c6463a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/tree/18c64635cc1b08f42233ef4416cc5f1869e6e3e6" target="_blank">18c6463</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  .gitignore
    Untracked:  0_data/RDS_objects/DT.rds

Unstaged changes:
    Modified:   0_data/RDS_objects/dge.rds
    Modified:   0_data/RDS_objects/enrichGO.rds
    Modified:   0_data/RDS_objects/enrichGO_sig.rds
    Modified:   0_data/RDS_objects/fc.rds
    Modified:   0_data/RDS_objects/lfc.rds
    Modified:   0_data/RDS_objects/lmTreat.rds
    Modified:   0_data/RDS_objects/lmTreat_all.rds
    Modified:   0_data/RDS_objects/lmTreat_sig.rds
    Modified:   0_data/RDS_objects/pub.rds
    Modified:   2_plots/de/pval_1.05.svg
    Modified:   2_plots/de/pval_1.1.svg
    Modified:   2_plots/de/pval_1.5.svg
    Modified:   2_plots/go/bp_dot_1.05.svg
    Modified:   2_plots/go/bp_dot_1.5.svg
    Modified:   2_plots/go/cc_dot_1.05.svg
    Modified:   2_plots/go/cc_dot_1.5.svg
    Modified:   2_plots/go/mf_dot_1.5.svg
    Modified:   2_plots/go/upset_1.05.svg
    Modified:   2_plots/go/upset_1.1.svg
    Modified:   2_plots/go/upset_1.5.svg
    Modified:   2_plots/ipa/Cell-To-Cell Signaling.svg
    Modified:   2_plots/ipa/diseaseAndFunction.svg
    Modified:   2_plots/ipa/pathways.svg
    Modified:   2_plots/kegg/kegg_dot_1.05.svg
    Modified:   2_plots/kegg/kegg_dot_1.1.svg
    Modified:   2_plots/kegg/kegg_dot_1.5.svg
    Modified:   2_plots/kegg/upset_kegg_1.05.svg
    Modified:   2_plots/kegg/upset_kegg_1.1.svg
    Modified:   2_plots/kegg/upset_kegg_1.5.svg
    Modified:   2_plots/qc/PCA_IntvsCont.svg
    Modified:   2_plots/qc/counts_after_filtering_3_3.svg
    Modified:   2_plots/qc/counts_before_after_filtering_3_3.svg
    Modified:   2_plots/qc/counts_before_filtering.svg
    Modified:   2_plots/qc/library_size.svg
    Modified:   2_plots/reactome/react_dot_1.05.svg
    Modified:   2_plots/reactome/react_dot_1.1.svg
    Modified:   2_plots/reactome/react_dot_1.5.svg
    Modified:   2_plots/reactome/upset_react_1.05.svg
    Modified:   2_plots/reactome/upset_react_1.1.svg
    Modified:   2_plots/reactome/upset_react_1.5.svg
    Modified:   3_output/enrichGO_sig.xlsx
    Modified:   3_output/enrichKEGG_all.xlsx
    Modified:   3_output/enrichKEGG_sig.xlsx
    Modified:   3_output/lmTreat_all.xlsx
    Modified:   3_output/lmTreat_fc1.5_voom2_all_fdr.xlsx
    Modified:   3_output/lmTreat_sig.xlsx
    Modified:   3_output/reactome_all.xlsx
    Modified:   3_output/reactome_sig.xlsx

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>1_analysis/reactome.Rmd</code>) and HTML
(<code>docs/reactome.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/18c64635cc1b08f42233ef4416cc5f1869e6e3e6/1_analysis/reactome.Rmd" target="_blank">18c6463</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-09-16
</td>
<td>
Added KEGG copyright permission and changed kable
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/b44640abac9ec4e34694489147942349d93fa0c2/docs/reactome.html" target="_blank">b44640a</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/eb10bef914f0d0b9b1743ea9b235859ce87feba4/1_analysis/reactome.Rmd" target="_blank">eb10bef</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-22
</td>
<td>
workflowr::wflow_publish(here::here("1_analysis/*.Rmd"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/8c9178d0ddead7d974f55328edb95ebe40c0a3dd/docs/reactome.html" target="_blank">8c9178d</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
added png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/d578f46c648558988d8eac977c170c39e29213ab/docs/reactome.html" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/159f3529790db3ae75472a460696c82121d932a3/docs/reactome.html" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
<td>
adding for pathview
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/1_analysis/reactome.Rmd" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
<td>
test png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/reactome.html" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
<td>
test png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/691cf3499cc98849bde703fde17b25047c072b71/docs/reactome.html" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/7f6bab239f8f9682dab208d1c314dd72dbbd21bc/1_analysis/reactome.Rmd" target="_blank">7f6bab2</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
<td>
workflowr::wflow_publish(here::here("1_analysis/*.Rmd"))
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/b6cf1906d2fd775f953bd1dd6b835234e260209c/1_analysis/reactome.Rmd" target="_blank">b6cf190</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-19
</td>
<td>
quick commit
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/1_analysis/reactome.Rmd" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/reactome.html" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
<td>
build website
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="data-setup" class="section level1">
<h1>Data Setup</h1>
<pre class="r"><code># working with data
library(dplyr)
library(magrittr)
library(readr)
library(tibble)
library(reshape2)
library(tidyverse)

# Visualisation:
library(kableExtra)
library(ggplot2)
library(grid)
library(pander)
library(cowplot)
library(pheatmap)
library(DT)

# Custom ggplot
library(ggbiplot)
library(ggrepel)
theme_set(theme_light())

pub &lt;- readRDS(here::here(&quot;0_data/RDS_objects/pub.rds&quot;))
DT &lt;- readRDS(here::here(&quot;0_data/RDS_objects/DT.rds&quot;))


# Bioconductor packages:
library(edgeR)
library(limma)
library(Glimma)
library(clusterProfiler)
library(org.Mm.eg.db)
library(enrichplot)
library(ReactomePA)</code></pre>
<div id="import-dgelist-data" class="section level3">
<h3>Import DGElist Data</h3>
<p>DGElist object containing the raw feature count, sample metadata, and
gene metadata, created in the Set Up stage.</p>
<pre class="r"><code># load DGElist previously created in the set up
dge &lt;- readRDS(here::here(&quot;0_data/RDS_objects/dge.rds&quot;))
fc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/fc.rds&quot;))
lfc &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lfc.rds&quot;))
lmTreat &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat.rds&quot;))
lmTreat_sig &lt;- readRDS(here::here(&quot;0_data/RDS_objects/lmTreat_sig.rds&quot;))</code></pre>
</div>
</div>
<div id="reactome" class="section level1 tabset">
<h1 class="tabset">Reactome</h1>
<div id="fc1.05" class="section level2 tabset">
<h2 class="tabset">FC=1.05</h2>
<pre class="r"><code>p=1</code></pre>
<div id="enriched-pathways" class="section level3">
<h3>Enriched pathways</h3>
<pre class="r"><code>reactome=list()
reactome_all=list()
reactome_sig=list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  
  reactome[[x]] &lt;- enrichPathway(gene = lmTreat_sig[[x]]$entrezid, organism = &quot;mouse&quot;, pvalueCutoff = 0.05, pAdjustMethod = &quot;fdr&quot;, readable = T)

reactome_all[[x]] &lt;- reactome[[x]]@result
reactome_sig[[x]] &lt;- reactome_all[[x]] %&gt;% dplyr::filter(p.adjust &lt;= 0.05) %&gt;% 
  separate(col = BgRatio, sep = &quot;/&quot;, into = c(&quot;Total&quot;, &quot;Universe&quot;)) %&gt;%
  dplyr::mutate(
    logFDR = -log(p.adjust, 10),
    GeneRatio = Count / as.numeric(Total))%&gt;%
    dplyr::select(c(&quot;Description&quot;, &quot;GeneRatio&quot;, &quot;pvalue&quot;, &quot;p.adjust&quot;, &quot;logFDR&quot;, &quot;qvalue&quot;, &quot;geneID&quot;, &quot;Count&quot;))

 # at the beginnning of a word (after 35 characters), add a newline. shorten the y axis for dot plot 
  reactome_sig[[x]]$Description &lt;- sub(pattern = &quot;(.{1,55})(?:$| )&quot;, 
                                       replacement = &quot;\\1\n&quot;, 
                                       x = reactome_sig[[x]]$Description)
  
  # remove the additional newline at the end of the string
  reactome_sig[[x]]$Description &lt;- sub(pattern = &quot;\n$&quot;, 
                                       replacement = &quot;&quot;, 
                                       x = reactome_sig[[x]]$Description)
}
reactome_sig[[p]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  DT(.,&quot;Enriched pathways&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-8d32cfd9db97395c4a5f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8d32cfd9db97395c4a5f">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Enriched pathways<\/caption>","data":[["R-MMU-112316","R-MMU-1474244","R-MMU-109582","R-MMU-418346","R-MMU-112315","R-MMU-216083","R-MMU-5173214","R-MMU-5173105","R-MMU-112314","R-MMU-418597","R-MMU-416482","R-MMU-977444","R-MMU-991365","R-MMU-418217","R-MMU-397795","R-MMU-977443","R-MMU-500657","R-MMU-76002","R-MMU-6794362","R-MMU-1296071","R-MMU-1650814","R-MMU-111885","R-MMU-451326","R-MMU-202433","R-MMU-1474290","R-MMU-202733","R-MMU-4086398","R-MMU-392451","R-MMU-3000157","R-MMU-456926","R-MMU-4085001","R-MMU-1296041","R-MMU-1296059","R-MMU-9634597","R-MMU-997272","R-MMU-392170","R-MMU-8948216","R-MMU-400042","R-MMU-1630316","R-MMU-9013149","R-MMU-9012999","R-MMU-9013148","R-MMU-8964315","R-MMU-9009391","R-MMU-392851","R-MMU-8964616","R-MMU-112040","R-MMU-1296072","R-MMU-420092","R-MMU-397014","R-MMU-114608","R-MMU-446728","R-MMU-373080","R-MMU-1296065"],["Neuronal System","Extracellular matrix organization","Hemostasis","Platelet homeostasis","Transmission across Chemical Synapses","Integrin cell surface interactions","O-glycosylation of TSR domain-containing proteins","O-linked glycosylation","Neurotransmitter receptors and postsynaptic signal\ntransmission","G alpha (z) signalling events","G alpha (12/13) signalling events","GABA B receptor activation","Activation of GABAB receptors","G beta:gamma signalling through PLC beta","G-protein beta:gamma signalling","GABA receptor activation","Presynaptic function of Kainate receptors","Platelet activation, signaling and aggregation","Protein-protein interactions at synapses","Potassium Channels","Collagen biosynthesis and modifying enzymes","Opioid Signalling","Activation of kainate receptors upon glutamate binding","Generation of second messenger molecules","Collagen formation","Cell surface interactions at the vascular wall","Ca2+ pathway","G beta:gamma signalling through PI3Kgamma","Laminin interactions","Thrombin signalling through proteinase activated\nreceptors (PARs)","Sialic acid metabolism","Activation of G protein gated Potassium channels","G protein gated Potassium channels","GPER1 signaling","Inhibition  of voltage gated Ca2+ channels via\nGbeta/gamma subunits","ADP signalling through P2Y purinoceptor 12","Collagen chain trimerization","Adrenaline,noradrenaline inhibits insulin secretion","Glycosaminoglycan metabolism","RAC1 GTPase cycle","RHO GTPase cycle","CDC42 GTPase cycle","G beta:gamma signalling through BTK","Extra-nuclear estrogen signaling","Prostacyclin signalling through prostacyclin receptor","G beta:gamma signalling through CDC42","G-protein mediated events","Voltage gated Potassium channels","Glucagon-type ligand receptors","Muscle contraction","Platelet degranulation ","Cell junction organization","Class B/2 (Secretin family receptors)","Inwardly rectifying K+ channels"],["0.176","0.19","0.147","0.274","0.179","0.273","0.364","0.219","0.188","0.344","0.239","0.293","0.293","0.4","0.323","0.246","0.381","0.147","0.227","0.196","0.233","0.214","0.3","0.368","0.205","0.191","0.25","0.32","0.5","0.281","0.273","0.286","0.286","0.286","0.286","0.318","0.257","0.304","0.162","0.147","0.119","0.179","0.333","0.188","0.316","0.316","0.225","0.225","0.242","0.144","0.155","0.188","0.196","0.235"],["1.23e-09","9.77e-09","3.27e-08","3.08e-07","1.06e-06","1.29e-06","2.82e-06","8.32e-06","1.32e-05","1.38e-05","1.72e-05","3.56e-05","3.56e-05","6.1e-05","6.37e-05","7e-05","9.2e-05","9.78e-05","0.000102","0.000111","0.000127","0.000205","0.000277","0.000327","0.000334","0.000344","0.000352","0.000372","0.000466","0.000473","0.000607","0.00087","0.00087","0.00087","0.00087","0.000904","0.000968","0.00122","0.0013","0.00134","0.0014","0.00155","0.00162","0.0019","0.00222","0.00222","0.00266","0.00266","0.00277","0.00283","0.00288","0.00292","0.00295","0.00338"],["8.69e-07","3.45e-06","7.71e-06","5.45e-05","0.00015","0.000153","0.000285","0.000735","0.000975","0.000975","0.0011","0.00193","0.00193","0.003","0.003","0.00309","0.00378","0.00378","0.00378","0.00391","0.00429","0.00659","0.00851","0.00922","0.00922","0.00922","0.00922","0.00938","0.0111","0.0111","0.0138","0.0176","0.0176","0.0176","0.0176","0.0178","0.0185","0.0226","0.0235","0.0236","0.0242","0.0261","0.0266","0.0305","0.0341","0.0341","0.0392","0.0392","0.0394","0.0394","0.0394","0.0394","0.0394","0.0443"],["6.06","5.46","5.11","4.26","3.82","3.82","3.55","3.13","3.01","3.01","2.96","2.71","2.71","2.52","2.52","2.51","2.42","2.42","2.42","2.41","2.37","2.18","2.07","2.04","2.04","2.04","2.04","2.03","1.95","1.95","1.86","1.76","1.76","1.76","1.76","1.75","1.73","1.65","1.63","1.63","1.62","1.58","1.57","1.52","1.47","1.47","1.41","1.41","1.4","1.4","1.4","1.4","1.4","1.35"],["7.38e-07","2.93e-06","6.54e-06","4.62e-05","0.000127","0.000129","0.000242","0.000624","0.000827","0.000827","0.000937","0.00164","0.00164","0.00255","0.00255","0.00262","0.00321","0.00321","0.00321","0.00332","0.00364","0.00559","0.00723","0.00782","0.00782","0.00782","0.00782","0.00796","0.00946","0.00946","0.0117","0.0149","0.0149","0.0149","0.0149","0.0151","0.0157","0.0192","0.0199","0.0201","0.0205","0.0221","0.0226","0.0259","0.0289","0.0289","0.0332","0.0332","0.0334","0.0334","0.0334","0.0334","0.0334","0.0376"],["Kcnk3/Kcnb2/Syn2/Slc1a3/Kcnj3/Gria4/Kcna6/Kcnd2/Ptprd/Kcnd3/Nlgn1/Epb41l3/Abat/Lrrc7/Gnai1/Lrrc4b/Kcnq5/Grin2b/Adcy3/Shank2/Maoa/Gngt2/Plcb1/Ppfia2/Slitrk3/Rps6ka2/Glrb/Ppfia4/Adcy4/Adcy7/Gabra4/Dlg2/Gng11/Lin7a/Nlgn3/Tspan7/Kcnma1/Gjc1/Gng2/Kcnc4/Plcb2/Slc38a1/Lrfn2/Dlg4/Kcng4/Gnb5/Gng7/Arhgef9/Gnb4/Gabbr1/Kcnq4/Cacna1a/Cacnb4/Prkar1b/Kcnh1/Camk1/Nlgn2","Col11a1/Mmp16/Col26a1/Itga4/P3h3/Col14a1/Capn6/Ddr2/Col4a6/Cd44/Lama4/Col16a1/Itgb7/Fn1/Mmp2/Icam1/Nid1/Jam3/Itga1/Col13a1/Loxl3/Itgal/Emilin1/Pecam1/Col5a2/Ltbp3/Mmp19/Bgn/Col15a1/P3h1/Adam19/Efemp2/Crtap/Col23a1/Col18a1/Itga8/Itgb4/Tmprss6/Lum/Nid2/Bmp7/Jam2/Fbln5/Col4a5","Apob/Hgf/Angpt1/Itga4/Cd109/Igf1/Tfpi/Slc8a1/Nos3/Cd44/Pcdh7/P2rx2/Gnai1/Fn1/Pde1b/Serpine2/Jam3/Tek/Gngt2/Pik3r6/Dock2/Rasgrp2/Slc8a3/Trpc6/Sh2b2/Kif26a/Dock10/Maged2/Islr/Procr/Gng11/Itgal/Esam/Dgkb/Tubb4a/Vegfc/Pde1a/Sele/Pecam1/Kif6/Igf2/Gng2/Pros1/Arrb1/Sccpdh/Arrb2/Fyn/Pde5a/Cd84/Tor4a/Fgr/Habp4/Atp2b4/Gnb5/Fam3c/Gng7/Selenop/Gnb4/Apbb1ip/Sri/Lat/Vps45/Kif16b/Spn/Lcp2/Pafah2/Lhfpl2/F2r/Prkar1b/Kif5a/Plek/Jam2","Apob/Slc8a1/Nos3/P2rx2/Pde1b/Gngt2/Slc8a3/Trpc6/Gng11/Pde1a/Pecam1/Gng2/Pde5a/Fgr/Atp2b4/Gnb5/Gng7/Gnb4/Sri/Pafah2","Syn2/Slc1a3/Kcnj3/Gria4/Abat/Lrrc7/Gnai1/Grin2b/Adcy3/Maoa/Gngt2/Plcb1/Ppfia2/Rps6ka2/Glrb/Ppfia4/Adcy4/Adcy7/Gabra4/Dlg2/Gng11/Lin7a/Tspan7/Gng2/Plcb2/Slc38a1/Dlg4/Gnb5/Gng7/Arhgef9/Gnb4/Gabbr1/Cacna1a/Cacnb4/Prkar1b/Camk1","Itga4/Col4a6/Cd44/Col16a1/Itgb7/Fn1/Icam1/Jam3/Itga1/Col13a1/Itgal/Pecam1/Col5a2/Col18a1/Itga8/Lum/Jam2/Col4a5","Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Adamts10/Adamts7/Adamtsl1/Sbspon/Adamts17","Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Galnt16/St3gal2/St6galnac4/Adamts10/Adamts7/Galntl6/Adamtsl1/Pomt2/Large1/B3gnt5/St3gal4/Sbspon/Adamts17/Galnt18","Kcnj3/Gria4/Lrrc7/Gnai1/Grin2b/Adcy3/Gngt2/Plcb1/Rps6ka2/Glrb/Adcy4/Adcy7/Gabra4/Dlg2/Gng11/Tspan7/Gng2/Plcb2/Dlg4/Gnb5/Gng7/Arhgef9/Gnb4/Gabbr1/Prkar1b/Camk1","Rgs16/Adcy3/Gngt2/Adcy4/Adcy7/Gng11/Gng2/Gnaz/Gnb5/Gng7/Gnb4","Plxnb1/Adra1a/Arhgef25/Gngt2/Gng11/Arhgef15/Gng2/Arhgef26/Net1/Fgd1/Gnb5/Gng7/Ngef/Arhgef9/Gnb4/Fgd2/Plekhg5","Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gng11/Gng2/Gnb5/Gng7/Gnb4/Gabbr1","Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gng11/Gng2/Gnb5/Gng7/Gnb4/Gabbr1","Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gnb5/Gng7/Gnb4","Gngt2/Pik3r6/Plcb1/Gng11/Gng2/Plcb2/Akt3/Gnb5/Gng7/Gnb4","Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Adcy7/Gabra4/Gng11/Gng2/Gnb5/Gng7/Arhgef9/Gnb4/Gabbr1","Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gnb5/Gng7/Gnb4","Hgf/Cd109/Igf1/Pcdh7/Gnai1/Fn1/Gngt2/Pik3r6/Rasgrp2/Trpc6/Maged2/Islr/Gng11/Dgkb/Vegfc/Pecam1/Igf2/Gng2/Pros1/Arrb1/Sccpdh/Arrb2/Fyn/Tor4a/Habp4/Gnb5/Fam3c/Gng7/Selenop/Gnb4/Apbb1ip/Lat/Lcp2/Lhfpl2/F2r/Plek","Gria4/Ptprd/Nlgn1/Epb41l3/Lrrc4b/Grin2b/Shank2/Ppfia2/Slitrk3/Ppfia4/Dlg2/Nlgn3/Lrfn2/Dlg4/Nlgn2","Kcnk3/Kcnb2/Kcnj3/Kcna6/Kcnd2/Kcnd3/Kcnq5/Gngt2/Gng11/Kcnma1/Gng2/Kcnc4/Kcng4/Gnb5/Gng7/Gnb4/Gabbr1/Kcnq4/Kcnh1","Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1/Col4a5","Gnai1/Pde1b/Adcy3/Gngt2/Plcb1/Adcy4/Adcy7/Gng11/Pde1a/Gng2/Plcb2/Gnb5/Gng7/Gnb4/Prkar1b","Gngt2/Plcb1/Gng11/Gng2/Plcb2/Dlg4/Gnb5/Gng7/Gnb4","Itk/Cd3g/Zap70/Lat/Lcp2/Grap2/Cd4","Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Loxl3/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1/Col4a5","Apob/Angpt1/Itga4/Cd44/Fn1/Jam3/Tek/Procr/Itgal/Esam/Sele/Pecam1/Pros1/Fyn/Cd84/Spn/Jam2","Lef1/Wnt11/Gngt2/Plcb1/Gng11/Gng2/Plcb2/Fzd2/Gnb5/Gng7/Gnb4","Gngt2/Pik3r6/Gng11/Gng2/Akt3/Gnb5/Gng7/Gnb4","Col4a6/Lama4/Nid1/Nid2/Col4a5","Gngt2/Gng11/Gng2/Arrb1/Arrb2/Gnb5/Gng7/Gnb4/F2r","St6gal2/St8sia2/St8sia4/Npl/St3gal2/St6galnac4/St8sia6/St6galnac6/St3gal4","Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4/Gabbr1","Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4/Gabbr1","Fn1/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4/Prkar1b","Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4/Gabbr1","Gnai1/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4","Col11a1/Col26a1/Col14a1/Col16a1/Col13a1/Col5a2/Col15a1/Col23a1/Col18a1","Gnai1/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4","Gpc6/Chst2/Cd44/Cspg4/Gpc3/B4galt2/Has2/St3gal2/Chst1/Glb1l/Bgn/Dse/Xylt1/St3gal4/Arsb/Abcc5/Gpc2/Lum/Papss2","Nhs/Mcam/Fermt2/Wasf3/Arhgap45/Arhgef25/Arhgap25/Dock2/Dock10/Abi2/Sh3bp1/Arhgef15/Nckap1l/Arap3/Prex2/Arhgap10/Dlc1/Arhgap31/Ngef/Fgd5/Arhgap15/Cav1/Cyfip2/Pld2","Nhs/Mcam/Plxnb1/Ccdc88a/Pcdh7/Fermt2/Wasf3/Arhgap45/Arhgef25/Akap12/Arhgap25/Dock2/Slitrk3/Myo9a/Plxnd1/Dock10/Abi2/Sh3bp1/Arhgap6/Arhgef15/Nckap1l/Arap3/Prex2/Arhgap10/Dlc1/Arhgef26/Arhgap28/Net1/Pde5a/Fgd1/Dlg5/Arhgap31/Arfgap3/Zap70/Cep97/Ngef/Arhgef9/Fgd5/Arhgap15/Elmo2/Cav1/Cyfip2/Pld2/Wdr6/Fmnl3/Fgd2/Plekhg5","Arhgap45/Arhgef25/Dock10/Arhgef15/Arap3/Prex2/Arhgap10/Dlc1/Arhgef26/Fgd1/Arhgap31/Ngef/Arhgef9/Cav1/Fgd2","Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4","Nos3/Gnai1/Areg/Mmp2/Gngt2/Gng11/Gng2/Akt3/Gnb5/Gng7/Gnb4/Cav1/S1pr3","Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4","Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4","Gnai1/Pde1b/Adcy3/Plcb1/Adcy4/Adcy7/Pde1a/Plcb2/Prkar1b","Kcnb2/Kcna6/Kcnd2/Kcnd3/Kcnq5/Kcnc4/Kcng4/Kcnq4/Kcnh1","Vipr2/Gngt2/Gng11/Gng2/Adcyap1r1/Gnb5/Gng7/Gnb4","Kcnk3/Cacna1c/Kcnd2/Ryr1/Kcnd3/Slc8a1/Tmod2/Gucy1a2/Itga1/Slc8a3/Gucy1b1/Gucy1a1/Mme/Myl3/Cacng7/Kcnip3/Asph/Pde5a/Ryr3/Atp2b4/Atp1a2/Sri","Hgf/Cd109/Igf1/Pcdh7/Fn1/Maged2/Islr/Vegfc/Pecam1/Igf2/Pros1/Sccpdh/Tor4a/Habp4/Fam3c/Selenop/Lhfpl2/Plek","Sdk2/Fermt2/Cdh2/Cdh6/Cdh13/Cdh10/Cdh5/Cdh11/Pard6g/Parvb/Flnc/Itgb4","Vipr2/Gngt2/Calca/Pth1r/Gng11/Gng2/Adcyap1r1/Calcrl/Gnb5/Gng7/Gnb4","Kcnj3/Gngt2/Gng11/Gng2/Gnb5/Gng7/Gnb4/Gabbr1"],["57","44","72","20","36","18","12","21","26","11","17","12","12","8","10","14","8","36","15","19","14","15","9","7","15","17","11","8","5","9","9","8","8","8","8","7","9","7","19","24","47","15","6","13","6","6","9","9","8","22","18","12","11","8"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>logFDR<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="dot-plot" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>react_dot=list()
upset=list()
for (i in 1:length(fc)) {
  x &lt;- fc[i] %&gt;% as.character()
  react_dot[[x]] &lt;- ggplot(reactome_sig[[x]][1:12, ]) +
  geom_point(aes(x = GeneRatio, y = reorder(Description, GeneRatio), colour = logFDR, size = Count)) +
  scale_color_gradient(low = &quot;dodgerblue3&quot;, high = &quot;firebrick3&quot;, limits = c(0, NA)) +
  scale_size(range = c(1.5,5)) +
  ggtitle(&quot;Reactome Pathways&quot;) +
  ylab(label = &quot;&quot;) +
  xlab(label = &quot;Gene Ratio&quot;) +
  labs(color = expression(&quot;-log&quot;[10] * &quot;FDR&quot;), size = &quot;Gene Counts&quot;)
  ggsave(filename = paste0(&quot;react_dot_&quot;, x, &quot;.svg&quot;), plot = react_dot[[x]] + pub, path = here::here(&quot;2_plots/reactome/&quot;), 
       width = 250, height = 130, units = &quot;mm&quot;)
  
  upset[[x]] &lt;- upsetplot(x = reactome[[x]], 9)
  ggsave(filename = paste0(&quot;upset_react_&quot;, fc[i], &quot;.svg&quot;), plot = upset[[x]], path = here::here(&quot;2_plots/reactome/&quot;))
}

react_dot[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/dotplot-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-dotplot-1">
Past versions of dotplot-1.png
</button>
</p>
<div id="fig-dotplot-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/reactome.Rmd/dotplot-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/reactome.Rmd/dotplot-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/figure/reactome.Rmd/dotplot-1.png" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/reactome.Rmd/dotplot-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/reactome.Rmd/dotplot-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="upset" class="section level3">
<h3>Upset</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/reactome.Rmd/unnamed-chunk-2-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/reactome.Rmd/unnamed-chunk-2-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/reactome.Rmd/unnamed-chunk-2-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/reactome.Rmd/unnamed-chunk-2-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="fc1.1" class="section level2 tabset">
<h2 class="tabset">FC=1.1</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="enriched-pathways-1" class="section level3">
<h3>Enriched pathways</h3>
<pre class="r"><code>reactome_sig[[p]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  DT(.,&quot;Enriched pathways&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-d701554a25665d09d09f" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d701554a25665d09d09f">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Enriched pathways<\/caption>","data":[["R-MMU-112316","R-MMU-1474244","R-MMU-112315","R-MMU-5173214","R-MMU-418346","R-MMU-109582","R-MMU-418597","R-MMU-977444","R-MMU-991365","R-MMU-112314","R-MMU-5173105","R-MMU-977443","R-MMU-397795","R-MMU-216083","R-MMU-418217","R-MMU-500657","R-MMU-416482","R-MMU-1296071","R-MMU-6794362","R-MMU-451326","R-MMU-4086398","R-MMU-1650814","R-MMU-392451","R-MMU-111885","R-MMU-76002","R-MMU-4085001","R-MMU-1474290","R-MMU-1296041","R-MMU-1296059","R-MMU-997272","R-MMU-8948216","R-MMU-392170","R-MMU-9013149","R-MMU-400042","R-MMU-202733","R-MMU-8964315","R-MMU-446728","R-MMU-456926","R-MMU-373080","R-MMU-392851","R-MMU-8964616","R-MMU-420092","R-MMU-397014","R-MMU-9013148","R-MMU-1296065","R-MMU-416476","R-MMU-9009391","R-MMU-1630316","R-MMU-9634597","R-MMU-422356","R-MMU-9012999","R-MMU-1500931","R-MMU-1971475","R-MMU-202040","R-MMU-975576","R-MMU-428930","R-MMU-114608","R-MMU-112040","R-MMU-1296072","R-MMU-421270","R-MMU-5358351","R-MMU-163685","R-MMU-1474228"],["Neuronal System","Extracellular matrix organization","Transmission across Chemical Synapses","O-glycosylation of TSR domain-containing proteins","Platelet homeostasis","Hemostasis","G alpha (z) signalling events","GABA B receptor activation","Activation of GABAB receptors","Neurotransmitter receptors and postsynaptic signal\ntransmission","O-linked glycosylation","GABA receptor activation","G-protein beta:gamma signalling","Integrin cell surface interactions","G beta:gamma signalling through PLC beta","Presynaptic function of Kainate receptors","G alpha (12/13) signalling events","Potassium Channels","Protein-protein interactions at synapses","Activation of kainate receptors upon glutamate binding","Ca2+ pathway","Collagen biosynthesis and modifying enzymes","G beta:gamma signalling through PI3Kgamma","Opioid Signalling","Platelet activation, signaling and aggregation","Sialic acid metabolism","Collagen formation","Activation of G protein gated Potassium channels","G protein gated Potassium channels","Inhibition  of voltage gated Ca2+ channels via\nGbeta/gamma subunits","Collagen chain trimerization","ADP signalling through P2Y purinoceptor 12","RAC1 GTPase cycle","Adrenaline,noradrenaline inhibits insulin secretion","Cell surface interactions at the vascular wall","G beta:gamma signalling through BTK","Cell junction organization","Thrombin signalling through proteinase activated\nreceptors (PARs)","Class B/2 (Secretin family receptors)","Prostacyclin signalling through prostacyclin receptor","G beta:gamma signalling through CDC42","Glucagon-type ligand receptors","Muscle contraction","CDC42 GTPase cycle","Inwardly rectifying K+ channels","G alpha (q) signalling events","Extra-nuclear estrogen signaling","Glycosaminoglycan metabolism","GPER1 signaling","Regulation of insulin secretion","RHO GTPase cycle","Cell-Cell communication","A tetrasaccharide linker sequence is required for GAG\nsynthesis","G-protein activation","N-glycan antennae elongation in the medial/trans-Golgi","Thromboxane signalling through TP receptor","Platelet degranulation ","G-protein mediated events","Voltage gated Potassium channels","Cell-cell junction organization","Signaling by Hedgehog","Integration of energy metabolism","Degradation of the extracellular matrix"],["0.164","0.165","0.169","0.364","0.247","0.124","0.344","0.293","0.293","0.174","0.198","0.246","0.323","0.227","0.4","0.381","0.211","0.186","0.212","0.3","0.25","0.217","0.32","0.2","0.131","0.273","0.192","0.286","0.286","0.286","0.257","0.318","0.141","0.304","0.169","0.333","0.188","0.25","0.196","0.316","0.316","0.242","0.137","0.167","0.235","0.125","0.174","0.145","0.25","0.18","0.104","0.157","0.261","0.261","0.261","0.25","0.138","0.2","0.2","0.2","0.13","0.156","0.14"],["5.13e-10","1.5e-07","3.51e-07","6.53e-07","8.75e-07","1.12e-06","3.67e-06","8.89e-06","8.89e-06","1.16e-05","1.44e-05","1.52e-05","1.94e-05","2.07e-05","2.25e-05","3.43e-05","5.17e-05","5.94e-05","8.74e-05","9.69e-05","0.000105","0.000124","0.000144","0.00017","0.000173","0.000218","0.000271","0.000346","0.000346","0.000346","0.000354","0.000393","0.000467","0.000533","0.000708","0.000786","0.000904","0.000923","0.000981","0.00109","0.00109","0.00115","0.00117","0.00118","0.00142","0.00164","0.00179","0.0018","0.00193","0.00204","0.00279","0.0031","0.00321","0.00321","0.00321","0.00405","0.00413","0.00422","0.00422","0.00422","0.00447","0.0046","0.00463"],["3.43e-07","5.03e-05","7.83e-05","0.000109","0.000117","0.000124","0.00035","0.000661","0.000661","0.000777","0.000845","0.000845","0.000989","0.000989","0.001","0.00143","0.00203","0.00221","0.00308","0.00324","0.00333","0.00376","0.00418","0.00463","0.00463","0.00561","0.00671","0.00765","0.00765","0.00765","0.00765","0.00823","0.00947","0.0105","0.0135","0.0146","0.0163","0.0163","0.0168","0.0177","0.0177","0.018","0.018","0.018","0.0211","0.0238","0.0251","0.0251","0.0263","0.0272","0.0366","0.0391","0.0391","0.0391","0.0391","0.047","0.047","0.047","0.047","0.047","0.049","0.0491","0.0491"],["6.46","4.3","4.11","3.96","3.93","3.9","3.46","3.18","3.18","3.11","3.07","3.07","3","3","3","2.84","2.69","2.66","2.51","2.49","2.48","2.42","2.38","2.33","2.33","2.25","2.17","2.12","2.12","2.12","2.12","2.08","2.02","1.98","1.87","1.84","1.79","1.79","1.77","1.75","1.75","1.75","1.75","1.75","1.68","1.62","1.6","1.6","1.58","1.56","1.44","1.41","1.41","1.41","1.41","1.33","1.33","1.33","1.33","1.33","1.31","1.31","1.31"],["2.9e-07","4.24e-05","6.6e-05","9.22e-05","9.87e-05","0.000105","0.000295","0.000557","0.000557","0.000655","0.000713","0.000713","0.000834","0.000834","0.000848","0.00121","0.00172","0.00186","0.00259","0.00273","0.00281","0.00317","0.00352","0.00391","0.00391","0.00473","0.00566","0.00645","0.00645","0.00645","0.00645","0.00694","0.00798","0.00885","0.0114","0.0123","0.0137","0.0137","0.0142","0.0149","0.0149","0.0152","0.0152","0.0152","0.0178","0.0201","0.0212","0.0212","0.0222","0.023","0.0309","0.033","0.033","0.033","0.033","0.0397","0.0397","0.0397","0.0397","0.0397","0.0413","0.0414","0.0414"],["Kcnk3/Kcnb2/Syn2/Kcnj3/Slc1a3/Gria4/Kcna6/Kcnd2/Ptprd/Kcnd3/Nlgn1/Epb41l3/Lrrc7/Abat/Gnai1/Lrrc4b/Kcnq5/Grin2b/Adcy3/Maoa/Shank2/Gngt2/Ppfia2/Plcb1/Slitrk3/Rps6ka2/Glrb/Gabra4/Dlg2/Adcy4/Ppfia4/Gng11/Nlgn3/Lin7a/Adcy7/Tspan7/Kcnma1/Kcnc4/Gjc1/Gng2/Plcb2/Lrfn2/Kcng4/Slc38a1/Dlg4/Gng7/Gnb5/Arhgef9/Gnb4/Kcnq4/Gabbr1/Cacna1a/Cacnb4","Col11a1/Mmp16/Col26a1/Itga4/P3h3/Col14a1/Capn6/Col4a6/Ddr2/Cd44/Lama4/Itgb7/Fn1/Mmp2/Col16a1/Icam1/Nid1/Jam3/Itga1/Col13a1/Loxl3/Itgal/Emilin1/Pecam1/Col5a2/Col15a1/Ltbp3/Mmp19/Bgn/P3h1/Adam19/Crtap/Efemp2/Col23a1/Col18a1/Itga8/Tmprss6/Itgb4","Syn2/Kcnj3/Slc1a3/Gria4/Lrrc7/Abat/Gnai1/Grin2b/Adcy3/Maoa/Gngt2/Ppfia2/Plcb1/Rps6ka2/Glrb/Gabra4/Dlg2/Adcy4/Ppfia4/Gng11/Lin7a/Adcy7/Tspan7/Gng2/Plcb2/Slc38a1/Dlg4/Gng7/Gnb5/Arhgef9/Gnb4/Gabbr1/Cacna1a/Cacnb4","Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Adamts10/Adamts7/Adamtsl1/Sbspon/Adamts17","Apob/Nos3/Slc8a1/P2rx2/Pde1b/Gngt2/Slc8a3/Trpc6/Gng11/Pde1a/Pecam1/Gng2/Pde5a/Gng7/Fgr/Atp2b4/Gnb5/Gnb4","Apob/Hgf/Angpt1/Itga4/Cd109/Igf1/Tfpi/Nos3/Slc8a1/Cd44/Pcdh7/P2rx2/Gnai1/Fn1/Serpine2/Tek/Pde1b/Jam3/Gngt2/Pik3r6/Slc8a3/Rasgrp2/Dock2/Sh2b2/Kif26a/Dock10/Trpc6/Procr/Gng11/Islr/Esam/Itgal/Dgkb/Vegfc/Tubb4a/Pde1a/Maged2/Sele/Kif6/Igf2/Pecam1/Pros1/Gng2/Arrb1/Sccpdh/Pde5a/Cd84/Arrb2/Fyn/Gng7/Fgr/Atp2b4/Tor4a/Selenop/Gnb5/Lat/Fam3c/Habp4/Apbb1ip/Gnb4/Kif16b","Rgs16/Adcy3/Gngt2/Adcy4/Gng11/Adcy7/Gng2/Gnaz/Gng7/Gnb5/Gnb4","Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Gng11/Adcy7/Gng2/Gng7/Gnb5/Gnb4/Gabbr1","Kcnj3/Gnai1/Adcy3/Gngt2/Adcy4/Gng11/Adcy7/Gng2/Gng7/Gnb5/Gnb4/Gabbr1","Kcnj3/Gria4/Lrrc7/Gnai1/Grin2b/Adcy3/Gngt2/Plcb1/Rps6ka2/Glrb/Gabra4/Dlg2/Adcy4/Gng11/Adcy7/Tspan7/Gng2/Plcb2/Dlg4/Gng7/Gnb5/Arhgef9/Gnb4/Gabbr1","Adamts20/Sema5b/Adamts16/Adamts19/Thsd7b/Thsd1/Adamts1/Galnt16/St6galnac4/St3gal2/Adamts10/Adamts7/Galntl6/Adamtsl1/B3gnt5/Sbspon/St3gal4/Large1/Adamts17","Kcnj3/Gnai1/Adcy3/Gngt2/Gabra4/Adcy4/Gng11/Adcy7/Gng2/Gng7/Gnb5/Arhgef9/Gnb4/Gabbr1","Gngt2/Pik3r6/Plcb1/Gng11/Gng2/Plcb2/Gng7/Akt3/Gnb5/Gnb4","Itga4/Col4a6/Cd44/Itgb7/Fn1/Col16a1/Icam1/Jam3/Itga1/Col13a1/Itgal/Pecam1/Col5a2/Col18a1/Itga8","Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gng7/Gnb5/Gnb4","Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gng7/Gnb5/Gnb4","Plxnb1/Adra1a/Arhgef25/Gngt2/Gng11/Arhgef15/Gng2/Arhgef26/Net1/Fgd1/Gng7/Gnb5/Ngef/Arhgef9/Gnb4","Kcnk3/Kcnb2/Kcnj3/Kcna6/Kcnd2/Kcnd3/Kcnq5/Gngt2/Gng11/Kcnma1/Kcnc4/Gng2/Kcng4/Gng7/Gnb5/Gnb4/Kcnq4/Gabbr1","Gria4/Ptprd/Nlgn1/Epb41l3/Lrrc4b/Grin2b/Shank2/Ppfia2/Slitrk3/Dlg2/Ppfia4/Nlgn3/Lrfn2/Dlg4","Gngt2/Plcb1/Gng11/Gng2/Plcb2/Dlg4/Gng7/Gnb5/Gnb4","Lef1/Wnt11/Gngt2/Plcb1/Gng11/Gng2/Plcb2/Fzd2/Gng7/Gnb5/Gnb4","Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1","Gngt2/Pik3r6/Gng11/Gng2/Gng7/Akt3/Gnb5/Gnb4","Gnai1/Pde1b/Adcy3/Gngt2/Plcb1/Adcy4/Gng11/Adcy7/Pde1a/Gng2/Plcb2/Gng7/Gnb5/Gnb4","Hgf/Cd109/Igf1/Pcdh7/Gnai1/Fn1/Gngt2/Pik3r6/Rasgrp2/Trpc6/Gng11/Islr/Dgkb/Vegfc/Maged2/Igf2/Pecam1/Pros1/Gng2/Arrb1/Sccpdh/Arrb2/Fyn/Gng7/Tor4a/Selenop/Gnb5/Lat/Fam3c/Habp4/Apbb1ip/Gnb4","St6gal2/St8sia2/St8sia4/Npl/St6galnac4/St3gal2/St8sia6/St6galnac6/St3gal4","Col11a1/Col26a1/P3h3/Col14a1/Col4a6/Col16a1/Col13a1/Loxl3/Col5a2/Col15a1/P3h1/Crtap/Col23a1/Col18a1","Kcnj3/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4/Gabbr1","Kcnj3/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4/Gabbr1","Kcnj3/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4/Gabbr1","Col11a1/Col26a1/Col14a1/Col16a1/Col13a1/Col5a2/Col15a1/Col23a1/Col18a1","Gnai1/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Nhs/Mcam/Fermt2/Wasf3/Arhgap45/Arhgef25/Arhgap25/Dock2/Dock10/Abi2/Sh3bp1/Arhgef15/Nckap1l/Arap3/Prex2/Arhgap10/Dlc1/Arhgap31/Ngef/Fgd5/Arhgap15/Cav1/Cyfip2","Gnai1/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Apob/Angpt1/Itga4/Cd44/Fn1/Tek/Jam3/Procr/Esam/Itgal/Sele/Pecam1/Pros1/Cd84/Fyn","Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Sdk2/Fermt2/Cdh2/Cdh6/Cdh13/Cdh10/Cdh5/Cdh11/Pard6g/Parvb/Flnc/Itgb4","Gngt2/Gng11/Gng2/Arrb1/Arrb2/Gng7/Gnb5/Gnb4","Vipr2/Gngt2/Calca/Pth1r/Gng11/Adcyap1r1/Gng2/Calcrl/Gng7/Gnb5/Gnb4","Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Vipr2/Gngt2/Gng11/Adcyap1r1/Gng2/Gng7/Gnb5/Gnb4","Kcnk3/Cacna1c/Kcnd2/Ryr1/Kcnd3/Slc8a1/Tmod2/Gucy1a2/Slc8a3/Itga1/Gucy1b1/Mme/Gucy1a1/Myl3/Cacng7/Kcnip3/Asph/Ryr3/Pde5a/Atp2b4/Atp1a2","Arhgap45/Arhgef25/Dock10/Arhgef15/Arap3/Prex2/Arhgef26/Arhgap10/Dlc1/Fgd1/Arhgap31/Ngef/Arhgef9/Cav1","Kcnj3/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4/Gabbr1","Adra1a/Rgs16/Ednrb/Lpar4/Arhgef25/Gngt2/Ptgfr/Edn3/Plcb1/Rps6ka2/Trpc6/Gng11/Dgkb/Rgs5/Agtr1a/Ednra/Bdkrb2/Xcl1/Htr2a/Gng2/Plcb2/Gng7/Gnb5/Prokr1/Gnb4","Nos3/Gnai1/Areg/Mmp2/Gngt2/Gng11/Gng2/Gng7/Akt3/Gnb5/Gnb4/Cav1","Gpc6/Chst2/Cd44/Cspg4/Gpc3/B4galt2/Has2/St3gal2/Chst1/Glb1l/Bgn/Xylt1/Dse/St3gal4/Arsb/Gpc2/Abcc5","Fn1/Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Cacna1c/Gnai1/Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gng7/Gnb5/Gnb4/Cacna1a","Nhs/Mcam/Plxnb1/Ccdc88a/Pcdh7/Fermt2/Wasf3/Arhgap45/Arhgef25/Akap12/Arhgap25/Dock2/Slitrk3/Plxnd1/Dock10/Abi2/Myo9a/Sh3bp1/Arhgef15/Arhgap6/Nckap1l/Arap3/Prex2/Arhgef26/Arhgap10/Arhgap28/Pde5a/Dlc1/Net1/Fgd1/Zap70/Dlg5/Arhgap31/Ngef/Fgd5/Arhgef9/Cep97/Arhgap15/Arfgap3/Cav1/Cyfip2","Sdk2/Fermt2/Cdh2/Cdh6/Cdh13/Cdh10/Cdh5/Cdh11/Pard6g/Parvb/Flnc/Fyn/Itgb4","Gpc6/Cspg4/Gpc3/Bgn/Xylt1/Gpc2","Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","St8sia2/Man2a2/Chst10/B4galt2/St8sia6/St3gal4","Gngt2/Gng11/Gng2/Gng7/Gnb5/Gnb4","Hgf/Cd109/Igf1/Pcdh7/Fn1/Islr/Vegfc/Maged2/Igf2/Pecam1/Pros1/Sccpdh/Tor4a/Selenop/Fam3c/Habp4","Gnai1/Pde1b/Adcy3/Plcb1/Adcy4/Adcy7/Pde1a/Plcb2","Kcnb2/Kcna6/Kcnd2/Kcnd3/Kcnq5/Kcnc4/Kcng4/Kcnq4","Sdk2/Cdh2/Cdh6/Cdh13/Cdh10/Cdh5/Cdh11/Pard6g","Gli1/Ihh/Hhip/Evc2/Evc/Wdr35/Adcy3/Gli3/Ift57/Adcy4/Gpr161/Adcy7/Tubb4a/Dync2h1/Arrb1/Ptch1/Arrb2/Dzip1","Cacna1c/Acacb/Gnai1/Gngt2/Plcb1/Gng11/Gng2/Plcb2/Gng7/Gnb5/Gnb4/Cacna1a","Col11a1/Mmp16/Col26a1/Capn6/Col4a6/Cd44/Fn1/Mmp2/Nid1/Col13a1/Col5a2/Col15a1/Mmp19/Col18a1/Tmprss6"],["53","38","34","12","18","61","11","12","12","24","19","14","10","15","8","8","15","18","14","9","11","13","8","14","32","9","14","8","8","8","9","7","23","7","15","6","12","8","11","6","6","8","21","14","8","25","12","17","7","11","41","13","6","6","6","6","16","8","8","8","18","12","15"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>logFDR<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="dot-plot-1" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>react_dot[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/reactome.Rmd/unnamed-chunk-5-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/reactome.Rmd/unnamed-chunk-5-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/figure/reactome.Rmd/unnamed-chunk-5-1.png" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/reactome.Rmd/unnamed-chunk-5-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/reactome.Rmd/unnamed-chunk-5-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="upset-1" class="section level3">
<h3>Upset</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-6-1">
Past versions of unnamed-chunk-6-1.png
</button>
</p>
<div id="fig-unnamed-chunk-6-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/reactome.Rmd/unnamed-chunk-6-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/reactome.Rmd/unnamed-chunk-6-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/reactome.Rmd/unnamed-chunk-6-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/reactome.Rmd/unnamed-chunk-6-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="fc1.5" class="section level2 tabset">
<h2 class="tabset">FC=1.5</h2>
<pre class="r"><code>p=p+1</code></pre>
<div id="enriched-pathways-2" class="section level3">
<h3>Enriched pathways</h3>
<pre class="r"><code>reactome_sig[[p]] %&gt;%
  dplyr::mutate_if(is.numeric, funs(as.character(signif(., 3)))) %&gt;%
  DT(.,&quot;Enriched pathways&quot;)</code></pre>
<div class="datatables html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-27a6251d00cbd347f90b" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-27a6251d00cbd347f90b">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Enriched pathways<\/caption>","data":[["R-MMU-112316","R-MMU-6794362","R-MMU-5173214","R-MMU-422475","R-MMU-9675108","R-MMU-1474228","R-MMU-3000157","R-MMU-1296072","R-MMU-1474244","R-MMU-975634","R-MMU-6806667","R-MMU-1296071"],["Neuronal System","Protein-protein interactions at synapses","O-glycosylation of TSR domain-containing proteins","Axon guidance","Nervous system development","Degradation of the extracellular matrix","Laminin interactions","Voltage gated Potassium channels","Extracellular matrix organization","Retinoid metabolism and transport","Metabolism of fat-soluble vitamins","Potassium Channels"],["0.0619","0.106","0.152","0.0522","0.052","0.0748","0.3","0.125","0.0519","0.119","0.109","0.0722"],["7.86e-07","0.000144","0.000246","0.000411","0.000427","0.000553","0.000577","0.000618","0.000719","0.000777","0.00118","0.00151"],["0.000253","0.0232","0.0249","0.0249","0.0249","0.0249","0.0249","0.0249","0.025","0.025","0.0347","0.0406"],["3.6","1.63","1.6","1.6","1.6","1.6","1.6","1.6","1.6","1.6","1.46","1.39"],["0.000217","0.0199","0.0213","0.0213","0.0213","0.0213","0.0213","0.0213","0.0214","0.0214","0.0297","0.0348"],["Kcnk3/Kcnb2/Syn2/Kcnj3/Slc1a3/Kcnd2/Gria4/Kcna6/Kcnd3/Lrrc7/Nlgn1/Ptprd/Abat/Epb41l3/Lrrc4b/Gnai1/Kcnq5/Ppfia2/Grin2b/Gabra4","Gria4/Nlgn1/Ptprd/Epb41l3/Lrrc4b/Ppfia2/Grin2b","Adamts20/Sema5b/Adamts16/Thsd7b/Adamts19","Gap43/Epha8/Efnb3/Dcc/St8sia2/Plxnb1/St8sia4/Dab1/Evl/Grin2b/Sema7a/Mmp2/Cxcl12","Gap43/Epha8/Efnb3/Dcc/St8sia2/Plxnb1/St8sia4/Dab1/Evl/Grin2b/Sema7a/Mmp2/Cxcl12","Col11a1/Mmp16/Col26a1/Capn6/Col4a6/Cd44/Mmp2/Nid1","Col4a6/Lama4/Nid1","Kcnb2/Kcnd2/Kcna6/Kcnd3/Kcnq5","Col11a1/Mmp16/Col26a1/Col14a1/Capn6/Col4a6/Itga4/P3h3/Cd44/Mmp2/Lama4/Nid1","Gpc6/Apob/Ttr/Gpc3/Lrp2","Gpc6/Apob/Ttr/Gpc3/Lrp2","Kcnk3/Kcnb2/Kcnj3/Kcnd2/Kcna6/Kcnd3/Kcnq5"],["20","7","5","13","13","8","3","5","12","5","5","7"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>logFDR<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"Blfrtip","scrollX":true,"buttons":["copy","csv","excel","pdf","print"],"lengthMenu":[[10,25,50,-1],["10","25","50","All"]],"columnDefs":[{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code>  # kable() %&gt;%
  # kable_styling(bootstrap_options = c(&quot;striped&quot;, &quot;hover&quot;)) %&gt;%
  # scroll_box(height = &quot;600px&quot;)</code></pre>
</div>
<div id="dot-plot-2" class="section level3">
<h3>Dot plot</h3>
<pre class="r"><code>react_dot[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/reactome.Rmd/unnamed-chunk-9-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/reactome.Rmd/unnamed-chunk-9-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/4d51a4ed437ceeb2919e7688538d43b96bd8ab4c/docs/figure/reactome.Rmd/unnamed-chunk-9-1.png" target="_blank">4d51a4e</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/reactome.Rmd/unnamed-chunk-9-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/reactome.Rmd/unnamed-chunk-9-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="upset-2" class="section level3">
<h3>Upset</h3>
<pre class="r"><code>upset[[p]]</code></pre>
<p><img src="figure/reactome.Rmd/unnamed-chunk-10-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/d578f46c648558988d8eac977c170c39e29213ab/docs/figure/reactome.Rmd/unnamed-chunk-10-1.png" target="_blank">d578f46</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/159f3529790db3ae75472a460696c82121d932a3/docs/figure/reactome.Rmd/unnamed-chunk-10-1.png" target="_blank">159f352</a>
</td>
<td>
tranmanhha135
</td>
<td>
2023-01-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/691cf3499cc98849bde703fde17b25047c072b71/docs/figure/reactome.Rmd/unnamed-chunk-10-1.png" target="_blank">691cf34</a>
</td>
<td>
Ha Manh Tran
</td>
<td>
2023-01-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/tranmanhha135/Mouse_endometrial_transcriptome_2023/blob/3119fadcedcb2bd928908d12f40b0b10f64961e0/docs/figure/reactome.Rmd/unnamed-chunk-10-1.png" target="_blank">3119fad</a>
</td>
<td>
tranmanhha135
</td>
<td>
2022-11-05
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="export-data" class="section level1">
<h1>Export Data</h1>
<pre class="r"><code># save to csv
writexl::write_xlsx(x = reactome_all, here::here(&quot;3_output/reactome_all.xlsx&quot;))
writexl::write_xlsx(x = reactome_sig, here::here(&quot;3_output/reactome_sig.xlsx&quot;))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)

Matrix products: default


locale:
[1] LC_COLLATE=English_Australia.utf8  LC_CTYPE=English_Australia.utf8   
[3] LC_MONETARY=English_Australia.utf8 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.utf8    

time zone: Australia/Adelaide
tzcode source: internal

attached base packages:
[1] stats4    grid      stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] ReactomePA_1.44.0     enrichplot_1.20.1     org.Mm.eg.db_3.17.0  
 [4] AnnotationDbi_1.62.2  IRanges_2.34.1        S4Vectors_0.38.1     
 [7] Biobase_2.60.0        BiocGenerics_0.46.0   clusterProfiler_4.8.3
[10] Glimma_2.10.0         edgeR_3.42.4          limma_3.56.2         
[13] ggrepel_0.9.3         ggbiplot_0.55         scales_1.2.1         
[16] plyr_1.8.8            DT_0.29               pheatmap_1.0.12      
[19] cowplot_1.1.1         pander_0.6.5          kableExtra_1.3.4     
[22] lubridate_1.9.2       forcats_1.0.0         stringr_1.5.0        
[25] purrr_1.0.1           tidyr_1.3.0           ggplot2_3.4.3        
[28] tidyverse_2.0.0       reshape2_1.4.4        tibble_3.2.1         
[31] readr_2.1.4           magrittr_2.0.3        dplyr_1.1.2          

loaded via a namespace (and not attached):
  [1] splines_4.3.1               later_1.3.1                
  [3] bitops_1.0-7                ggplotify_0.1.2            
  [5] polyclip_1.10-4             graph_1.78.0               
  [7] lifecycle_1.0.3             rprojroot_2.0.3            
  [9] lattice_0.21-8              MASS_7.3-60                
 [11] crosstalk_1.2.0             sass_0.4.7                 
 [13] rmarkdown_2.24              jquerylib_0.1.4            
 [15] yaml_2.3.7                  httpuv_1.6.11              
 [17] DBI_1.1.3                   RColorBrewer_1.1-3         
 [19] abind_1.4-5                 zlibbioc_1.46.0            
 [21] rvest_1.0.3                 GenomicRanges_1.52.0       
 [23] ggraph_2.1.0                RCurl_1.98-1.12            
 [25] yulab.utils_0.0.9           tweenr_2.0.2               
 [27] rappdirs_0.3.3              git2r_0.32.0               
 [29] GenomeInfoDbData_1.2.10     tidytree_0.4.5             
 [31] reactome.db_1.84.0          svglite_2.1.1              
 [33] codetools_0.2-19            DelayedArray_0.26.7        
 [35] DOSE_3.26.1                 xml2_1.3.5                 
 [37] ggforce_0.4.1               tidyselect_1.2.0           
 [39] aplot_0.2.1                 farver_2.1.1               
 [41] viridis_0.6.4               matrixStats_1.0.0          
 [43] webshot_0.5.5               jsonlite_1.8.7             
 [45] ellipsis_0.3.2              tidygraph_1.2.3            
 [47] systemfonts_1.0.4           tools_4.3.1                
 [49] ragg_1.2.5                  treeio_1.24.3              
 [51] Rcpp_1.0.11                 glue_1.6.2                 
 [53] gridExtra_2.3               xfun_0.39                  
 [55] here_1.0.1                  DESeq2_1.40.2              
 [57] qvalue_2.32.0               MatrixGenerics_1.12.3      
 [59] GenomeInfoDb_1.36.3         withr_2.5.0                
 [61] fastmap_1.1.1               fansi_1.0.4                
 [63] digest_0.6.33               timechange_0.2.0           
 [65] R6_2.5.1                    gridGraphics_0.5-1         
 [67] textshaping_0.3.6           colorspace_2.1-0           
 [69] GO.db_3.17.0                RSQLite_2.3.1              
 [71] utf8_1.2.3                  generics_0.1.3             
 [73] data.table_1.14.8           graphlayouts_1.0.0         
 [75] httr_1.4.7                  htmlwidgets_1.6.2          
 [77] S4Arrays_1.0.6              scatterpie_0.2.1           
 [79] graphite_1.46.0             whisker_0.4.1              
 [81] pkgconfig_2.0.3             gtable_0.3.4               
 [83] blob_1.2.4                  workflowr_1.7.1            
 [85] XVector_0.40.0              shadowtext_0.1.2           
 [87] htmltools_0.5.5             fgsea_1.26.0               
 [89] ggupset_0.3.0               png_0.1-8                  
 [91] ggfun_0.1.3                 knitr_1.44                 
 [93] rstudioapi_0.15.0           tzdb_0.4.0                 
 [95] nlme_3.1-163                cachem_1.0.8               
 [97] parallel_4.3.1              HDO.db_0.99.1              
 [99] pillar_1.9.0                vctrs_0.6.3                
[101] promises_1.2.0.1            evaluate_0.21              
[103] cli_3.6.1                   locfit_1.5-9.8             
[105] compiler_4.3.1              rlang_1.1.1                
[107] crayon_1.5.2                labeling_0.4.3             
[109] fs_1.6.3                    writexl_1.4.2              
[111] stringi_1.7.12              viridisLite_0.4.2          
[113] BiocParallel_1.34.2         munsell_0.5.0              
[115] Biostrings_2.68.1           lazyeval_0.2.2             
[117] GOSemSim_2.26.1             Matrix_1.6-1               
[119] hms_1.1.3                   patchwork_1.1.3            
[121] bit64_4.0.5                 KEGGREST_1.40.0            
[123] SummarizedExperiment_1.30.2 igraph_1.5.1               
[125] memoise_2.0.1               bslib_0.5.1                
[127] ggtree_3.8.2                fastmatch_1.1-4            
[129] bit_4.0.5                   downloader_0.4             
[131] ape_5.7-1                   gson_0.1.0                 </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
